<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Bioc2020</title>

<script src="site_libs/header-attrs-2.3/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lazappi's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/lazappi/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Bioc2020</h1>
<h3 class="subtitle">Bioconductor conference 2020</h3>
<h4 class="date">2020-07-28 15:38:48</h4>

</div>


<p><strong>Parameters</strong></p>
<table>
<thead>
<tr class="header">
<th align="left">Parameter</th>
<th align="left">Value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">hashtag</td>
<td align="left">#bioc2020</td>
</tr>
<tr class="even">
<td align="left">start_day</td>
<td align="left">2020-07-27</td>
</tr>
<tr class="odd">
<td align="left">end_day</td>
<td align="left">2020-07-31</td>
</tr>
<tr class="even">
<td align="left">timezone</td>
<td align="left">America/New_York</td>
</tr>
<tr class="odd">
<td align="left">theme</td>
<td align="left">theme_light</td>
</tr>
<tr class="even">
<td align="left">accent</td>
<td align="left">#0092ac</td>
</tr>
<tr class="odd">
<td align="left">accent2</td>
<td align="left">#87b13f</td>
</tr>
<tr class="even">
<td align="left">kcore</td>
<td align="left">2</td>
</tr>
<tr class="odd">
<td align="left">topics_k</td>
<td align="left">6</td>
</tr>
<tr class="even">
<td align="left">bigram_filter</td>
<td align="left">3</td>
</tr>
<tr class="odd">
<td align="left">fixed</td>
<td align="left">FALSE</td>
</tr>
<tr class="even">
<td align="left">seed</td>
<td align="left">1</td>
</tr>
</tbody>
</table>
<div id="introduction" class="section level1" number="1">
<h1 number="1"><span class="header-section-number">1</span> Introduction</h1>
<p>An analysis of tweets from the #bioc2020 hashtag for the <a href="http://bioc2020.bioconductor.org/" title="Bioconductor 2020 conference website">BioC 2020</a> conference.</p>
<p>A total of 725 tweets from 213 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1" number="2">
<h1 number="2"><span class="header-section-number">2</span> Timeline</h1>
<div id="tweets-by-day" class="section level2" number="2.1">
<h2 number="2.1"><span class="header-section-number">2.1</span> Tweets by day</h2>
<p><img src="figures/Bioc2020/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2" number="2.2">
<h2 number="2.2"><span class="header-section-number">2.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2020-07-27 - 2020-07-31 in the America/New_York timezone.</p>
<p><img src="figures/Bioc2020/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1" number="3">
<h1 number="3"><span class="header-section-number">3</span> Users</h1>
<div id="top-tweeters" class="section level2 tabset tabset-pills" number="3.1">
<h2 class="tabset tabset-pills" number="3.1"><span class="header-section-number">3.1</span> Top tweeters</h2>
<div id="overall" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Overall</h3>
<p><img src="figures/Bioc2020/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="original" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Original</h3>
<p><img src="figures/Bioc2020/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweets" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Retweets</h3>
<p><img src="figures/Bioc2020/tweets-top-users-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweet-proportion" class="section level2" number="3.2">
<h2 number="3.2"><span class="header-section-number">3.2</span> Retweet proportion</h2>
<p><img src="figures/Bioc2020/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-timeline" class="section level2" number="3.3">
<h2 number="3.3"><span class="header-section-number">3.3</span> Top tweeters timeline</h2>
<p><img src="figures/Bioc2020/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2 tabset tabset-pills" number="3.4">
<h2 class="tabset tabset-pills" number="3.4"><span class="header-section-number">3.4</span> Top tweeters by day</h2>
<div id="overall-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Overall</h3>
<div id="day-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/Bioc2020/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/Bioc2020/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 3</h4>
<p><img src="figures/Bioc2020/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 4</h4>
<p><img src="figures/Bioc2020/tweets-top-users-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 5</h4>
<p><img src="figures/Bioc2020/tweets-top-users-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Original</h3>
<div id="day-1-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/Bioc2020/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/Bioc2020/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 3</h4>
<p><img src="figures/Bioc2020/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 4</h4>
<p><img src="figures/Bioc2020/tweets-top-users-orig-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 5</h4>
<p><img src="figures/Bioc2020/tweets-top-users-orig-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Retweets</h3>
<div id="day-1-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/Bioc2020/tweets-top-users-retweets-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/Bioc2020/tweets-top-users-retweets-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 3</h4>
<p><img src="figures/Bioc2020/tweets-top-users-retweets-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 4</h4>
<p><img src="figures/Bioc2020/tweets-top-users-retweets-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 5</h4>
<p><img src="figures/Bioc2020/tweets-top-users-retweets-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
</div>
<div id="sources" class="section level1" number="4">
<h1 number="4"><span class="header-section-number">4</span> Sources</h1>
<p><img src="figures/Bioc2020/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="networks" class="section level1" number="5">
<h1 number="5"><span class="header-section-number">5</span> Networks</h1>
<div id="replies" class="section level2" number="5.1">
<h2 number="5.1"><span class="header-section-number">5.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by PageRank.</p>
<p><img src="figures/Bioc2020/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2" number="5.2">
<h2 number="5.2"><span class="header-section-number">5.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Filtered for a k-core of 2. Node colour and size adjusted according to PageRank score.</p>
<p><img src="figures/Bioc2020/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="tweet-types" class="section level1" number="6">
<h1 number="6"><span class="header-section-number">6</span> Tweet types</h1>
<div id="retweets-2" class="section level2 tabset tabset-pills" number="6.1">
<h2 class="tabset tabset-pills" number="6.1"><span class="header-section-number">6.1</span> Retweets</h2>
<div id="proportion" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/Bioc2020/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/Bioc2020/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
Here is the full collection of posters for <span class="citation">@Bioconductor</span> #bioc2020. Slides for at least some talks will be posted there too, and a couple are up. <a href="https://t.co/QOOuTlvL3Z" class="uri">https://t.co/QOOuTlvL3Z</a>. This thread lists what’s there now - comments welcome.
</td>
<td style="text-align:right;">
34
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Tomorrow I’ll be teaching 🎓 a workshop on #recount2 for #BioC2020</p>
<p>It’ll likely be my last #recount2 workshop 😢😇</p>
<p>🌐 <a href="https://t.co/navj34rcq4" class="uri">https://t.co/navj34rcq4</a> 💻 <a href="https://t.co/sDlZfkKMTo" class="uri">https://t.co/sDlZfkKMTo</a> 🎚️🌈 <a href="https://t.co/Rwy2pPrNJk" class="uri">https://t.co/Rwy2pPrNJk</a></p>
<p>🎅🏾🎁 is coming later in 2020 at a <span class="citation">@Bioconductor</span> 🪞 near you!☺️</p>
#rstats #RNAseq <a href="https://t.co/EhIIwXPvsq" class="uri">https://t.co/EhIIwXPvsq</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
<p>#Bioc2020 are extremely grateful to phenomenal <span class="citation">@seandavis12</span> who is co-ordinating everything to run 22 live workshops</p>
<p><a href="https://t.co/RAv2Kmhgcu" class="uri">https://t.co/RAv2Kmhgcu</a></p>
He created a <span class="citation">@github</span> template for workshop presenters. It uses GitHub Actions to pkgdown websites and creates a docker image of each pkg <a href="https://t.co/wo9x7ccTxo" class="uri">https://t.co/wo9x7ccTxo</a>
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Chloe <span class="citation">@cmirzayi</span> shows how easily you can make causal diagrams using #daggity 📦</p>
#rstats #BioC2020 <a href="https://t.co/8cEElGwANs" class="uri">https://t.co/8cEElGwANs</a>
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
stephaniehicks
</td>
<td style="text-align:left;">
One of my favorite talks that I look forward to every year. <span class="citation">@mt_morgan</span> is kicking off #bioc2020 where he always so wonderfully highlights updates from the <span class="citation">@Bioconductor</span> community as a whole over the past year. Thank you <span class="citation">@mt_morgan</span>! #rstats <a href="https://t.co/Y5VaZjpmH6" class="uri">https://t.co/Y5VaZjpmH6</a>
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Next up, keynote by Rafa Irizarry <span class="citation">@rafalab</span> on probabilistic gene expression in single-cell RNAseq. He calls it an “ode to EDA” (exploratory days analysis) 👏#bioc2020
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Thanks to <span class="citation">@seandavis12</span> &amp;amp; workshop authors you can run the #BioC2020 workshop code on the cloud, your laptop or view 🖥️</p>
☁️ <a href="https://t.co/TyQWGtdDat" class="uri">https://t.co/TyQWGtdDat</a> (7GB RAM, 1.5 CPUs, max 3 hrs) 🐬<a href="https://t.co/j1vVMBcmAY" class="uri">https://t.co/j1vVMBcmAY</a> (Docker, same #rstats env) 🌍 or view online thx to #pkgdown <span class="citation">@github</span> actions <a href="https://t.co/hvgoueLSU3" class="uri">https://t.co/hvgoueLSU3</a>
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
seandavis12
</td>
<td style="text-align:left;">
With over 600 workshop instances served on a #kubernetes cluster on #GoogleCloud on just the first day of #bioc2020, the impact of <span class="citation">@isb_cgc</span> efforts and resources has been a multiplier for the fantastic work done by Bioconductor workshop contributors. <a href="https://t.co/VzVYcOi2Ru" class="uri">https://t.co/VzVYcOi2Ru</a>
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
mikelove
</td>
<td style="text-align:left;">
<p>For a #bioc2020 speaker it’s nice to see attendees adding a workshop to their schedules in advance 😁</p>
<p>Talks are recorded and workshop material is online. My and <span class="citation">@k3yavi</span>’s vignette on importing alevin scRNA-seq counts with tximeta (work w <span class="citation">@nomad421</span> et al)</p>
<a href="https://t.co/bVbJJnZcHH" class="uri">https://t.co/bVbJJnZcHH</a> <a href="https://t.co/htI3Lox2sr" class="uri">https://t.co/htI3Lox2sr</a>
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
#bioc2020 Dan Bunis, Atul Deskpande ask for colorblind friendly figures. Critical for single cell t-SNE/UMAP <span class="citation">@humancellatlas</span> Faceting figures, overlaid labels or even just letters (A,B,C) on clusters. Recommend less than 8 colors. dittoSeq sets a new standard for all #rstats <a href="https://t.co/xtAeY9kncD" class="uri">https://t.co/xtAeY9kncD</a>
</td>
<td style="text-align:right;">
9
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-retweeted" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most retweeted</h3>
<p><img src="figures/Bioc2020/most-retweeted-1.png" width="783" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="likes" class="section level2 tabset tabset-pills" number="6.2">
<h2 class="tabset tabset-pills" number="6.2"><span class="header-section-number">6.2</span> Likes</h2>
<div id="proportion-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/Bioc2020/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/Bioc2020/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Tomorrow I’ll be teaching 🎓 a workshop on #recount2 for #BioC2020</p>
<p>It’ll likely be my last #recount2 workshop 😢😇</p>
<p>🌐 <a href="https://t.co/navj34rcq4" class="uri">https://t.co/navj34rcq4</a> 💻 <a href="https://t.co/sDlZfkKMTo" class="uri">https://t.co/sDlZfkKMTo</a> 🎚️🌈 <a href="https://t.co/Rwy2pPrNJk" class="uri">https://t.co/Rwy2pPrNJk</a></p>
<p>🎅🏾🎁 is coming later in 2020 at a <span class="citation">@Bioconductor</span> 🪞 near you!☺️</p>
#rstats #RNAseq <a href="https://t.co/EhIIwXPvsq" class="uri">https://t.co/EhIIwXPvsq</a>
</td>
<td style="text-align:right;">
92
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
<p>#Bioc2020 are extremely grateful to phenomenal <span class="citation">@seandavis12</span> who is co-ordinating everything to run 22 live workshops</p>
<p><a href="https://t.co/RAv2Kmhgcu" class="uri">https://t.co/RAv2Kmhgcu</a></p>
He created a <span class="citation">@github</span> template for workshop presenters. It uses GitHub Actions to pkgdown websites and creates a docker image of each pkg <a href="https://t.co/wo9x7ccTxo" class="uri">https://t.co/wo9x7ccTxo</a>
</td>
<td style="text-align:right;">
64
</td>
</tr>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
Here is the full collection of posters for <span class="citation">@Bioconductor</span> #bioc2020. Slides for at least some talks will be posted there too, and a couple are up. <a href="https://t.co/QOOuTlvL3Z" class="uri">https://t.co/QOOuTlvL3Z</a>. This thread lists what’s there now - comments welcome.
</td>
<td style="text-align:right;">
62
</td>
</tr>
<tr>
<td style="text-align:left;">
seandavis12
</td>
<td style="text-align:left;">
With over 600 workshop instances served on a #kubernetes cluster on #GoogleCloud on just the first day of #bioc2020, the impact of <span class="citation">@isb_cgc</span> efforts and resources has been a multiplier for the fantastic work done by Bioconductor workshop contributors. <a href="https://t.co/VzVYcOi2Ru" class="uri">https://t.co/VzVYcOi2Ru</a>
</td>
<td style="text-align:right;">
49
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Next up, keynote by Rafa Irizarry <span class="citation">@rafalab</span> on probabilistic gene expression in single-cell RNAseq. He calls it an “ode to EDA” (exploratory days analysis) 👏#bioc2020
</td>
<td style="text-align:right;">
45
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
#bioc2020 is grateful to <span class="citation">@stickermule</span> for in-kind sponsorship so participants can get conference stickers. The <span class="citation">@Bioconductor</span> community’s expanding collection of package, event &amp;amp; diversity hex stickers are at <a href="https://t.co/gpV0EYNpKE" class="uri">https://t.co/gpV0EYNpKE</a> Add a sticker with pkg hexSticker::sticker <a href="https://t.co/OkQHAluEZV" class="uri">https://t.co/OkQHAluEZV</a>
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
mikelove
</td>
<td style="text-align:left;">
<p>For a #bioc2020 speaker it’s nice to see attendees adding a workshop to their schedules in advance 😁</p>
<p>Talks are recorded and workshop material is online. My and <span class="citation">@k3yavi</span>’s vignette on importing alevin scRNA-seq counts with tximeta (work w <span class="citation">@nomad421</span> et al)</p>
<a href="https://t.co/bVbJJnZcHH" class="uri">https://t.co/bVbJJnZcHH</a> <a href="https://t.co/htI3Lox2sr" class="uri">https://t.co/htI3Lox2sr</a>
</td>
<td style="text-align:right;">
36
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Chloe <span class="citation">@cmirzayi</span> shows how easily you can make causal diagrams using #daggity 📦</p>
#rstats #BioC2020 <a href="https://t.co/8cEElGwANs" class="uri">https://t.co/8cEElGwANs</a>
</td>
<td style="text-align:right;">
32
</td>
</tr>
<tr>
<td style="text-align:left;">
tangming2005
</td>
<td style="text-align:left;">
dplyr-based Access to Bioconductor Annotation Resources <a href="https://t.co/XoI1yOGNRT" class="uri">https://t.co/XoI1yOGNRT</a> #Bioc2020
</td>
<td style="text-align:right;">
31
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
#bioc2020 Dan Bunis, Atul Deskpande ask for colorblind friendly figures. Critical for single cell t-SNE/UMAP <span class="citation">@humancellatlas</span> Faceting figures, overlaid labels or even just letters (A,B,C) on clusters. Recommend less than 8 colors. dittoSeq sets a new standard for all #rstats <a href="https://t.co/xtAeY9kncD" class="uri">https://t.co/xtAeY9kncD</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-likes" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most likes</h3>
<p><img src="figures/Bioc2020/most-favorited-1.png" width="783" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="quotes" class="section level2 tabset tabset-pills" number="6.3">
<h2 class="tabset tabset-pills" number="6.3"><span class="header-section-number">6.3</span> Quotes</h2>
<div id="proportion-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/Bioc2020/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/Bioc2020/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
ivivek87
</td>
<td style="text-align:left;">
<p>Very insightful workshop <span class="citation">@cmirzayi</span> on causal diagrams.</p>
<p>#BioC2020</p>
<a href="https://t.co/Qu7SBjYbBJ" class="uri">https://t.co/Qu7SBjYbBJ</a> <a href="https://t.co/t5PVd9Ocoy" class="uri">https://t.co/t5PVd9Ocoy</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
TaniaGueA
</td>
<td style="text-align:left;">
I feel enlighted! Looking forward to try out #daggity on my own data and sit and cry once I confirm I did everything wrong the first time! lol🙈but at least now I know! Amazing talk and I’M LOVING THE WORKSHOPS PLATFORM! ❤️ #BioC2020 <a href="https://t.co/nlwAGlcHZd" class="uri">https://t.co/nlwAGlcHZd</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
Outstanding #bioc2020 workshop today by <span class="citation">@cmirzayi</span> of <span class="citation">@cunyisph</span> and <span class="citation">@CUNYSPH</span>, to a live audience of 140. If you missed it, <a href="https://t.co/lhyrHSz5ml" class="uri">https://t.co/lhyrHSz5ml</a> <a href="https://t.co/wVoNbspX4s" class="uri">https://t.co/wVoNbspX4s</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
ivivek87
</td>
<td style="text-align:left;">
<p><span class="citation">@siminaboca</span> <span class="citation">@OmicDataScience</span> <span class="citation">@fellgernon</span> Thoroughly enjoyed the last workshop session of Day 1 #BioC2020</p>
<a href="https://t.co/P2JIODWf6T" class="uri">https://t.co/P2JIODWf6T</a> <a href="https://t.co/J73N6DqMAB" class="uri">https://t.co/J73N6DqMAB</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
TaniaGueA
</td>
<td style="text-align:left;">
<p>on #tidybulk: Shut up and take my data 🤓 Thanks for the very nice explanation!</p>
#BioC2020 <a href="https://t.co/kCKTcTzojI" class="uri">https://t.co/kCKTcTzojI</a> <a href="https://t.co/MPDwUYlq4v" class="uri">https://t.co/MPDwUYlq4v</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
AggarwalAyush_
</td>
<td style="text-align:left;">
Really showing the power and easiness in using #tidyverse for #rnaseq analysis. Amazing workshop by <span class="citation">@steman_research</span> and maria doyle. My 1st bioc conference is turning out to be amazing. Lucky that it is virtual this time #BioC2020 #RStats <a href="https://t.co/Wgg85D1d2b" class="uri">https://t.co/Wgg85D1d2b</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Over 160, now 177 and counting participants are following live this first workshop!</p>
<p>We have switched to Zoom for the workshops</p>
<p>200 participants as I finished writing this tweet! 😍😊🤗</p>
<p>#bioc2020</p>
<a href="https://t.co/jMUIm7IRyC" class="uri">https://t.co/jMUIm7IRyC</a> <a href="https://t.co/li9L6llRGF" class="uri">https://t.co/li9L6llRGF</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
FelixMbuga
</td>
<td style="text-align:left;">
First #BioC2020 workshop on how to use BioConductor to access the oceans of publicly available data. <a href="https://t.co/jYhZ6237O6" class="uri">https://t.co/jYhZ6237O6</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
ivivek87
</td>
<td style="text-align:left;">
<p>#BioC2020</p>
<a href="https://t.co/O6xdMNAGYj" class="uri">https://t.co/O6xdMNAGYj</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
sisirakadambat
</td>
<td style="text-align:left;">
A very important aspect to be kept in mind in data visualization. #BioC2020 <a href="https://t.co/sqlzURJ2Px" class="uri">https://t.co/sqlzURJ2Px</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-quoted" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most quoted</h3>
<p><img src="figures/Bioc2020/most-quoted-1.png" width="783" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="media" class="section level1 tabset tabset-pills" number="7">
<h1 class="tabset tabset-pills" number="7"><span class="header-section-number">7</span> Media</h1>
<div id="proportion-3" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Proportion</h2>
<p><img src="figures/Bioc2020/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-3" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Top 10</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Tomorrow I’ll be teaching 🎓 a workshop on #recount2 for #BioC2020</p>
<p>It’ll likely be my last #recount2 workshop 😢😇</p>
<p>🌐 <a href="https://t.co/navj34rcq4" class="uri">https://t.co/navj34rcq4</a> 💻 <a href="https://t.co/sDlZfkKMTo" class="uri">https://t.co/sDlZfkKMTo</a> 🎚️🌈 <a href="https://t.co/Rwy2pPrNJk" class="uri">https://t.co/Rwy2pPrNJk</a></p>
<p>🎅🏾🎁 is coming later in 2020 at a <span class="citation">@Bioconductor</span> 🪞 near you!☺️</p>
#rstats #RNAseq <a href="https://t.co/EhIIwXPvsq" class="uri">https://t.co/EhIIwXPvsq</a>
</td>
<td style="text-align:right;">
92
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
<p>#Bioc2020 are extremely grateful to phenomenal <span class="citation">@seandavis12</span> who is co-ordinating everything to run 22 live workshops</p>
<p><a href="https://t.co/RAv2Kmhgcu" class="uri">https://t.co/RAv2Kmhgcu</a></p>
He created a <span class="citation">@github</span> template for workshop presenters. It uses GitHub Actions to pkgdown websites and creates a docker image of each pkg <a href="https://t.co/wo9x7ccTxo" class="uri">https://t.co/wo9x7ccTxo</a>
</td>
<td style="text-align:right;">
64
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
#bioc2020 is grateful to <span class="citation">@stickermule</span> for in-kind sponsorship so participants can get conference stickers. The <span class="citation">@Bioconductor</span> community’s expanding collection of package, event &amp;amp; diversity hex stickers are at <a href="https://t.co/gpV0EYNpKE" class="uri">https://t.co/gpV0EYNpKE</a> Add a sticker with pkg hexSticker::sticker <a href="https://t.co/OkQHAluEZV" class="uri">https://t.co/OkQHAluEZV</a>
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
mikelove
</td>
<td style="text-align:left;">
<p>For a #bioc2020 speaker it’s nice to see attendees adding a workshop to their schedules in advance 😁</p>
<p>Talks are recorded and workshop material is online. My and <span class="citation">@k3yavi</span>’s vignette on importing alevin scRNA-seq counts with tximeta (work w <span class="citation">@nomad421</span> et al)</p>
<a href="https://t.co/bVbJJnZcHH" class="uri">https://t.co/bVbJJnZcHH</a> <a href="https://t.co/htI3Lox2sr" class="uri">https://t.co/htI3Lox2sr</a>
</td>
<td style="text-align:right;">
36
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Chloe <span class="citation">@cmirzayi</span> shows how easily you can make causal diagrams using #daggity 📦</p>
#rstats #BioC2020 <a href="https://t.co/8cEElGwANs" class="uri">https://t.co/8cEElGwANs</a>
</td>
<td style="text-align:right;">
32
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
#bioc2020 Dan Bunis, Atul Deskpande ask for colorblind friendly figures. Critical for single cell t-SNE/UMAP <span class="citation">@humancellatlas</span> Faceting figures, overlaid labels or even just letters (A,B,C) on clusters. Recommend less than 8 colors. dittoSeq sets a new standard for all #rstats <a href="https://t.co/xtAeY9kncD" class="uri">https://t.co/xtAeY9kncD</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Thanks to <span class="citation">@seandavis12</span> &amp;amp; workshop authors you can run the #BioC2020 workshop code on the cloud, your laptop or view 🖥️</p>
☁️ <a href="https://t.co/TyQWGtdDat" class="uri">https://t.co/TyQWGtdDat</a> (7GB RAM, 1.5 CPUs, max 3 hrs) 🐬<a href="https://t.co/j1vVMBcmAY" class="uri">https://t.co/j1vVMBcmAY</a> (Docker, same #rstats env) 🌍 or view online thx to #pkgdown <span class="citation">@github</span> actions <a href="https://t.co/hvgoueLSU3" class="uri">https://t.co/hvgoueLSU3</a>
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
So so grateful to <span class="citation">@jo_rainer</span> for redesigning the #bioc2020 conference stickers to reflect its new virtual location. The virus lurks overhead, but there is light from the <span class="citation">@Bioconducto</span> logo on the laptop. The mug includes a nod to the intended physical location (Boston). <a href="https://t.co/h0iLDwRHvK" class="uri">https://t.co/h0iLDwRHvK</a>
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
So so grateful to Johannes Rainer for redesigning the #bioc2020 conference stickers to reflect its new virtual location. The virus lurks overhead, but there is light from the <span class="citation">@Bioconductor</span> logo on the laptop. The mug includes a nod to the intended physical location (Boston). <a href="https://t.co/rmi9o9LRq9" class="uri">https://t.co/rmi9o9LRq9</a>
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Martin <span class="citation">@mt_morgan</span> and Levi <span class="citation">@LeviWaldron1</span> started #BioC2020</p>
<p>🎚️ <a href="https://t.co/1Dj6CTkWWs" class="uri">https://t.co/1Dj6CTkWWs</a> 🖥️☁️ <a href="https://t.co/CgAANZklBP" class="uri">https://t.co/CgAANZklBP</a> by <span class="citation">@seandavis12</span></p>
It ended with a tribute to <span class="citation">@jxtx</span>. Thank you again James for everything you did! <a href="https://t.co/AxRCBXtHYo" class="uri">https://t.co/AxRCBXtHYo</a>
</td>
<td style="text-align:right;">
22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="section" class="section level1 unnumbered toc-ignore" number="">
<h1 class="unnumbered toc-ignore" number=""></h1>
<div id="most-liked-image" class="section level2" number="7.1">
<h2 number="7.1"><span class="header-section-number">7.1</span> Most liked image</h2>
<p><img src="http://pbs.twimg.com/media/Ed4x5ThWAAcjAck.jpg" /></p>
</div>
</div>
<div id="tweet-text" class="section level1" number="8">
<h1 number="8"><span class="header-section-number">8</span> Tweet text</h1>
<div id="word-cloud" class="section level2" number="8.1">
<h2 number="8.1"><span class="header-section-number">8.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/Bioc2020/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2" number="8.2">
<h2 number="8.2"><span class="header-section-number">8.2</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 3 times.</p>
<p><img src="figures/Bioc2020/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2" number="8.3">
<h2 number="8.3"><span class="header-section-number">8.3</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/Bioc2020/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset tabset-fade" number="8.3.1">
<h3 class="tabset tabset-fade" number="8.3.1"><span class="header-section-number">8.3.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 1</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
RiyueSunnyBao
</td>
<td style="text-align:left;">
<p>#BioC2020 day 1 kicking off!!!!!!!!</p>
<p>Lots of <em>super</em> exciting #talks &amp;amp; #workshops!</p>
<p>We are also actively #recruiting scientists to join our fight against #cancer <span class="citation">@UPMCHillmanCC</span>. Check job board!!</p>
You will find me on forum, channel, here and there … <em>open</em> to coffee chat! :) <a href="https://t.co/9dI4h9BRi4" class="uri">https://t.co/9dI4h9BRi4</a>
</td>
<td style="text-align:right;">
0.9955333
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
2nd workshop of the day! Chloe Mirzayi <span class="citation">@cmirzayi</span> introduces epidemiology for bioinformaticians! Detailed explanations on confounders, mediators, colliders. Selection bias (including loss to follow) can be a form of collider bias. #bioc2020
</td>
<td style="text-align:right;">
0.9953218
</td>
</tr>
<tr>
<td style="text-align:left;">
tangming2005
</td>
<td style="text-align:left;">
genes(txdb) can be a problem. e.g. a lncRNA can span the centromere. #Bioc2020 How to define a gene is a problem. I was checking where each gene is located in which cytoband and found some genes span multiple cytobands using genes(txdb) and I found that as well.
</td>
<td style="text-align:right;">
0.9950892
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Now we have Simina Boca <span class="citation">@siminaboca</span> on “Using the swfdr package to estimate false discovery rates conditional on covariates”</p>
<p>Simina writes excellent twitter summaries, so follow Simina on Twitter for more 📙🔖</p>
#BioC2020 #Rstats <a href="https://t.co/zjNBZ8WO76" class="uri">https://t.co/zjNBZ8WO76</a>
</td>
<td style="text-align:right;">
0.9950892
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>For our second 100 level workshop we have Chloe Mirzayi <span class="citation">@cmirzayi</span> who will present “A workshop for concepts of epidemiology” aka, Epidemiology for Bioinformaticians</p>
<p>Join and learn about causal inference (not casual!) Very relevant with #COVID19 nowadays!</p>
#BioC2020 #rstats <a href="https://t.co/hS74FLtVfF" class="uri">https://t.co/hS74FLtVfF</a>
</td>
<td style="text-align:right;">
0.9950892
</td>
</tr>
<tr>
<td style="text-align:left;">
ErickCuevasF
</td>
<td style="text-align:left;">
<span class="citation">@fellgernon</span> <span class="citation">@CDSBMexico</span> #Bioc2020 Orgullo! 👏👏🇲🇽 Package “recount2”: "Bioconductor package, and can quickly import gene, exon, exon-exon junction and base-pair coverage data for uniformly processed da <a href="https://t.co/Fn5HI0GsCP" class="uri">https://t.co/Fn5HI0GsCP</a>
</td>
<td style="text-align:right;">
0.9948324
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
I’m describing how to use the swfdr package <a href="https://t.co/D6jySIGvgy" class="uri">https://t.co/D6jySIGvgy</a> to estimate false discovery rates conditional on covariates. Package co-developed with <span class="citation">@jtleek</span>, <span class="citation">@JagerLeah</span>, and Tomasz Konopka. Slides at <a href="https://t.co/zMnM9SswbX" class="uri">https://t.co/zMnM9SswbX</a> #bioc2020
</td>
<td style="text-align:right;">
0.9942286
</td>
</tr>
<tr>
<td style="text-align:left;">
F1000Research
</td>
<td style="text-align:left;">
Who’s excited for #BioC2020 next week? 🎉To get into the swing of things, why not browse the <span class="citation">@Bioconductor</span> Gateway on <span class="citation">@F1000Research</span>? It’s packed with openly available, peer reviewed #bioinformatics research to get you ready for the conference: <a href="https://t.co/tmxhqw2MFm" class="uri">https://t.co/tmxhqw2MFm</a> <a href="https://t.co/eMWVoqMk40" class="uri">https://t.co/eMWVoqMk40</a>
</td>
<td style="text-align:right;">
0.9942286
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Thx to everyone who participated in the #recount2 workshop (recountWorkshop2020)!🤩</p>
<p>I have posted the questions and answers from the chat &amp;amp; poll at <a href="https://t.co/ZlXA7x3DVi" class="uri">https://t.co/ZlXA7x3DVi</a></p>
<p>Got more? Ask at:</p>
<p>📦 <a href="https://t.co/tVkeuKwZJT" class="uri">https://t.co/tVkeuKwZJT</a> feature requests <a href="https://t.co/BzWtAWouGw" class="uri">https://t.co/BzWtAWouGw</a></p>
#BioC2020 #rstats <a href="https://t.co/tp6cInYcVE" class="uri">https://t.co/tp6cInYcVE</a>
</td>
<td style="text-align:right;">
0.9942286
</td>
</tr>
<tr>
<td style="text-align:left;">
seandavis12
</td>
<td style="text-align:left;">
With over 600 workshop instances served on a #kubernetes cluster on #GoogleCloud on just the first day of #bioc2020, the impact of <span class="citation">@isb_cgc</span> efforts and resources has been a multiplier for the fantastic work done by Bioconductor workshop contributors. <a href="https://t.co/VzVYcOi2Ru" class="uri">https://t.co/VzVYcOi2Ru</a>
</td>
<td style="text-align:right;">
0.9938706
</td>
</tr>
<tr>
<td style="text-align:left;">
signormirko
</td>
<td style="text-align:left;">
At the moment, <span class="citation">@Bioconductor</span> &amp;amp; #RStats are also the reason why I’m in front of my laptop instead of in the go to bed phase (it’s 11 pm here)! One of the consequences of virtual conferences, where you get to attend from all parts of the worlds &amp;amp; all sorts of timezones 😀 #BioC2020 <a href="https://t.co/jX8odr1DvR" class="uri">https://t.co/jX8odr1DvR</a>
</td>
<td style="text-align:right;">
0.9938706
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 2</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
#bioc2020 Dan Bunis, Atul Deskpande ask for colorblind friendly figures. Critical for single cell t-SNE/UMAP <span class="citation">@humancellatlas</span> Faceting figures, overlaid labels or even just letters (A,B,C) on clusters. Recommend less than 8 colors. dittoSeq sets a new standard for all #rstats <a href="https://t.co/xtAeY9kncD" class="uri">https://t.co/xtAeY9kncD</a>
</td>
<td style="text-align:right;">
0.9962173
</td>
</tr>
<tr>
<td style="text-align:left;">
F1000Research
</td>
<td style="text-align:left;">
We’re thrilled to be supporting #BioC2020! To celebrate, we’re highlighting the awesome <span class="citation">@Bioconductor</span> Gateway on <span class="citation">@F1000Research</span>, so stay tuned this week for our top picks in #bioinformatics research 😊 And don’t forget to check out the conference posters: <a href="https://t.co/4WOQBJ4PIk" class="uri">https://t.co/4WOQBJ4PIk</a> <a href="https://t.co/7s9ZPxO0IB" class="uri">https://t.co/7s9ZPxO0IB</a>
</td>
<td style="text-align:right;">
0.9955333
</td>
</tr>
<tr>
<td style="text-align:left;">
CDSBMexico
</td>
<td style="text-align:left;">
<p>El próximo miércoles 7/29 a las 10 am EST 🗓️ <span class="citation">@josschavezf1</span> <span class="citation">@BarjonCar</span> <span class="citation">@EmilianoSotel10</span> <span class="citation">@areyesq</span> <span class="citation">@fellgernon</span> hablaran en #BioC2020 sobre nuestros esfuerzos incluyendo #regutools</p>
<p>🌎 <a href="https://t.co/KiWHZ394VG" class="uri">https://t.co/KiWHZ394VG</a></p>
<p>¡Te esperamos! 😊</p>
#rstatsES <span class="citation">@Bioconductor</span> <span class="citation">@RBioinformatica</span> <span class="citation">@nnb_unam</span> <a href="https://t.co/tExxEbQxLn" class="uri">https://t.co/tExxEbQxLn</a>
</td>
<td style="text-align:right;">
0.9953218
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Our 6th and last #BioC2020 workshop for today is by Stefano Mangiola <span class="citation">@steman_research</span> and Maria Doyle on “A tidy transcriptomics introduction to RNA-Seq analyses”</p>
<p>Join us tomorrow for more great content by our community =)</p>
<p><a href="https://t.co/lUB24kiDiE" class="uri">https://t.co/lUB24kiDiE</a></p>
#BioC2020 #rstats #RNAseq <a href="https://t.co/qVq7Zl9jaK" class="uri">https://t.co/qVq7Zl9jaK</a>
</td>
<td style="text-align:right;">
0.9953218
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p><span class="citation">@KevinRUE67</span> <span class="citation">@AedinCulhane</span> <span class="citation">@Bioconductor</span> I recommend diving this task in 2:</p>
<ul>
<li><p>The R code for creating the output files. The equivalent of rmarkdown::render()</p></li>
<li><p>Deploying. It’s easiest to use gh-pages. See the workshop example repo by <span class="citation">@seandavis12</span> for 1 case. Or dive into pkgdown::deploy_to_branch()</p></li>
</ul>
#BioC2020 #rstats <a href="https://t.co/39gEaaidww" class="uri">https://t.co/39gEaaidww</a>
</td>
<td style="text-align:right;">
0.9953218
</td>
</tr>
<tr>
<td style="text-align:left;">
CDSBMexico
</td>
<td style="text-align:left;">
<p><span class="citation">@josschavezf1</span> <span class="citation">@BarjonCar</span> <span class="citation">@EmilianoSotel10</span> <span class="citation">@areyesq</span> <span class="citation">@fellgernon</span> <span class="citation">@Bioconductor</span> <span class="citation">@RBioinformatica</span> <span class="citation">@nnb_unam</span> Next Wednesday 7/29 at 10 am EST 🗓️ <span class="citation">@josschavezf1</span> <span class="citation">@BarjonCar</span> <span class="citation">@EmilianoSotel10</span> <span class="citation">@areyesq</span> <span class="citation">@fellgernon</span> will speak at #BioC2020 about our efforts including #regutools</p>
<p>🌎 <a href="https://t.co/KiWHZ394VG" class="uri">https://t.co/KiWHZ394VG</a></p>
<p>Come join us! There’ll be a Q&amp;amp;A section</p>
#rstats <span class="citation">@Bioconductor</span> <span class="citation">@RBioinformatica</span> <span class="citation">@nnb_unam</span> <a href="https://t.co/ovO4cmaSS4" class="uri">https://t.co/ovO4cmaSS4</a>
</td>
<td style="text-align:right;">
0.9950892
</td>
</tr>
<tr>
<td style="text-align:left;">
ivivek87
</td>
<td style="text-align:left;">
<p><span class="citation">@siminaboca</span> <span class="citation">@OmicDataScience</span> <span class="citation">@fellgernon</span> Final workshop of Day 1 #BioC2020</p>
<p>“A tidy transcriptomics introduction to RNA-Seq analyses”</p>
<p>Maria Doyle and Stefano Mangiola.</p>
<p><a href="https://t.co/J3kSeLyudl" class="uri">https://t.co/J3kSeLyudl</a></p>
#tidyverse #RStats #transcription #RNA #bioinformatics
</td>
<td style="text-align:right;">
0.9950892
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Instrumental Variable (IV) Analysis/Mendelian Randomization (MR) try to mimic an RCT by including a variable that is a possible source of source of randomization (like the random assignment of treatments in an RCT) eg a genetic variant. #bioc2020
</td>
<td style="text-align:right;">
0.9948324
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>1st day of #BioC2020 ✅</p>
<p>Got a bit of info overload! 😵 but it went great :)</p>
<p>Sorry that I didn’t see live some ❓ during my #recount2 workshop but I’ve posted the answers below</p>
Virtual advantages: dressed up differently, took a shower 🚿 during a break <a href="https://t.co/MxIAqZ8MZE" class="uri">https://t.co/MxIAqZ8MZE</a> <a href="https://t.co/AWUiLUlMVR" class="uri">https://t.co/AWUiLUlMVR</a>
</td>
<td style="text-align:right;">
0.9948324
</td>
</tr>
<tr>
<td style="text-align:left;">
spaiglass
</td>
<td style="text-align:left;">
Looking forward to presenting our patient classifier #rstats package at virtual #bioc2020! Great teamwork with <span class="citation">@garybader1</span> <span class="citation">@ruth_isserlin</span> <span class="citation">@Shui33329350</span> <span class="citation">@janbaumbach</span>, P Weber, R Giugno, L Giudice, MA Shah, H Kaka, A Nohr. . Thank you organizers for your work! /1 <a href="https://t.co/wbE3Xok6i3" class="uri">https://t.co/wbE3Xok6i3</a>
</td>
<td style="text-align:right;">
0.9942286
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-3" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 3</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Yi Wang presented “Co-expression analysis is biased by a mean-correlation relationship”</p>
<p>Yi was the first presenter of the contributed talks session (sorry for the earlier mistake at <a href="https://t.co/gdfYgsSqey" class="uri">https://t.co/gdfYgsSqey</a> !)</p>
<p>Check <span class="citation">@siminaboca</span> summary: <a href="https://t.co/fk0yKQVovS" class="uri">https://t.co/fk0yKQVovS</a></p>
#BioC2020 #rstats <a href="https://t.co/MaMGz0Cuxw" class="uri">https://t.co/MaMGz0Cuxw</a>
</td>
<td style="text-align:right;">
0.9950892
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
RI’s group is working on collecting datasets across species and tissues to “combine information across thousands of genes and probabilistically assign cell-type identity.” Work w/ grad student Isabella Grabski. <a href="https://t.co/EMALCKcAsR" class="uri">https://t.co/EMALCKcAsR</a> #bioc2020
</td>
<td style="text-align:right;">
0.9948324
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Yi Wang is presenting work looking at correlation of pairs of genes and how their distribution depends on the mean expression level (“mean-correlation relationship”). She developed spatial quantile normalization (SpQN) to use b/f co-expression analysis. #bioc2020
</td>
<td style="text-align:right;">
0.9945472
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Many many methods available, depending on data types considered, type of hypothesis (are they all up- or down-regulated, or is there a change in either direction in a set?, two-group comparison vs. multi-group comparison), database of gene-sets considered etc. #bioc2020
</td>
<td style="text-align:right;">
0.9945472
</td>
</tr>
<tr>
<td style="text-align:left;">
SeqComplete
</td>
<td style="text-align:left;">
<p>#genomics #ngs <span class="citation">@Bioconductor</span>: ’Next up, Rafa <span class="citation">@rafalab</span> on work done by many collaborators: “their work as much as mine” 👏🏽</p>
<p>“Probabilistic Gene Expression Signatures for Single Cell RNA-seq Data”</p>
#BioC2020 #Genomics #… <a href="https://t.co/4YXFIPBN9T" class="uri">https://t.co/4YXFIPBN9T</a>, see more <a href="https://t.co/U7scMFApc9" class="uri">https://t.co/U7scMFApc9</a>
</td>
<td style="text-align:right;">
0.9945472
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
RI: Before reporting p-vals, look at data! Gives now-classical example of analysis that shows 1/4 of genes are differentially expressed between people of European and Asian origin… Seems very odd! Turns out the main driver is the year in which samples are processed #bioc2020
</td>
<td style="text-align:right;">
0.9945472
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Next up is Ting Huang <span class="citation">@huang704</span> on “Co-expression analysis is biased by a mean-correlation relationship”</p>
<p>Ting is kick starting the first contributed talks session</p>
<p>🏠 <a href="https://t.co/r2ChBU6T2J" class="uri">https://t.co/r2ChBU6T2J</a></p>
#BioC2020 #rstats <a href="https://t.co/8mifSTuCR1" class="uri">https://t.co/8mifSTuCR1</a>
</td>
<td style="text-align:right;">
0.9942286
</td>
</tr>
<tr>
<td style="text-align:left;">
matthewnmccall
</td>
<td style="text-align:left;">
#bioc2020 <span class="citation">@bioconductor</span> Day 5 (7/31) contributed talk: Bench pressing performant single cell RNA-seq preprocessing tools through pipeComp; a general framework for the evaluation of computational pipelines, presented by <span class="citation">@AnthonySonrel</span>
</td>
<td style="text-align:right;">
0.9942286
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Next up, Rafa <span class="citation">@rafalab</span> on work done by many collaborators: “their work as much as mine” 👏🏽</p>
<p>“Probabilistic Gene Expression Signatures for Single Cell RNA-seq Data”</p>
#BioC2020 #Genomics #rstats <a href="https://t.co/mLZXWySDlO" class="uri">https://t.co/mLZXWySDlO</a>
</td>
<td style="text-align:right;">
0.9938706
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Levi <span class="citation">@LeviWaldron1</span>:</p>
<p>Something that differentiates <span class="citation">@Bioconductor</span> from CRAN, #tidyverse is the use of common “core” data structures for sharing data: #SummarizedExperiment 📦 fam (RangedSummarizedExperiment, SingleCellExperiment, MultiAssayExperiment, RaggedExperiment)</p>
#BioC2020
</td>
<td style="text-align:right;">
0.9938706
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Next up, keynote by Rafa Irizarry <span class="citation">@rafalab</span> on probabilistic gene expression in single-cell RNAseq. He calls it an “ode to EDA” (exploratory days analysis) 👏#bioc2020
</td>
<td style="text-align:right;">
0.9938706
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 4</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Thanks to <span class="citation">@seandavis12</span> &amp;amp; workshop authors you can run the #BioC2020 workshop code on the cloud, your laptop or view 🖥️</p>
☁️ <a href="https://t.co/TyQWGtdDat" class="uri">https://t.co/TyQWGtdDat</a> (7GB RAM, 1.5 CPUs, max 3 hrs) 🐬<a href="https://t.co/j1vVMBcmAY" class="uri">https://t.co/j1vVMBcmAY</a> (Docker, same #rstats env) 🌍 or view online thx to #pkgdown <span class="citation">@github</span> actions <a href="https://t.co/hvgoueLSU3" class="uri">https://t.co/hvgoueLSU3</a>
</td>
<td style="text-align:right;">
0.9962173
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>To get started in day 2, we have Ludwig Geistlinger from Levi Waldron’s lab <span class="citation">@LeviWaldron1</span> with help from Marcel Ramos <span class="citation">@M2RUseR</span> on “Functional enrichment analysis of high-throughput omics data”</p>
<p>4th time teaching this workshop!</p>
#BioC2020 #rstats #workshops <a href="https://t.co/wsGgyGSBwQ" class="uri">https://t.co/wsGgyGSBwQ</a>
</td>
<td style="text-align:right;">
0.9957265
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
<p>#Bioc2020 are extremely grateful to phenomenal <span class="citation">@seandavis12</span> who is co-ordinating everything to run 22 live workshops</p>
<p><a href="https://t.co/RAv2Kmhgcu" class="uri">https://t.co/RAv2Kmhgcu</a></p>
He created a <span class="citation">@github</span> template for workshop presenters. It uses GitHub Actions to pkgdown websites and creates a docker image of each pkg <a href="https://t.co/wo9x7ccTxo" class="uri">https://t.co/wo9x7ccTxo</a>
</td>
<td style="text-align:right;">
0.9955333
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
#bioc2020 is grateful to <span class="citation">@stickermule</span> for in-kind sponsorship so participants can get conference stickers. The <span class="citation">@Bioconductor</span> community’s expanding collection of package, event &amp;amp; diversity hex stickers are at <a href="https://t.co/gpV0EYNpKE" class="uri">https://t.co/gpV0EYNpKE</a> Add a sticker with pkg hexSticker::sticker <a href="https://t.co/OkQHAluEZV" class="uri">https://t.co/OkQHAluEZV</a>
</td>
<td style="text-align:right;">
0.9953218
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
CM introduces the dagitty package for creating causal directed acyclic graphs (DAGs) <a href="https://t.co/XI7PXGGPcU" class="uri">https://t.co/XI7PXGGPcU</a> Can use it to find what you need to control for in a model to get the effect of a variable on another variable based on the knowledge encoded in the DAG! #bioc2020
</td>
<td style="text-align:right;">
0.9950892
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Ben Haibe-Kains <span class="citation">@bhaibeka</span> now introducing the PharmacoGx package <a href="https://t.co/2mtYcIotDg" class="uri">https://t.co/2mtYcIotDg</a> for drug sensitivity data (eg NCI60) and the ToxicoGx package <a href="https://t.co/RKEEsPWk9Z" class="uri">https://t.co/RKEEsPWk9Z</a> which has healthy cell data (don’t want drugs to be toxic to good cells.) #bioc2020
</td>
<td style="text-align:right;">
0.9950892
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
Ting Huang now presenting work on MSstatsTMT <span class="citation">@Bioconductor</span> package for performing protein significance analysis in shotgun MS-based proteomics with TMT labeling. Package at <a href="https://t.co/qMVRUcYyuc" class="uri">https://t.co/qMVRUcYyuc</a>, in collab w/ adviser <span class="citation">@olgavitek</span> and others. #bioc2020
</td>
<td style="text-align:right;">
0.9948324
</td>
</tr>
<tr>
<td style="text-align:left;">
tebiwankenebi
</td>
<td style="text-align:left;">
visualize your #multiplexed #imaging data with <span class="citation">@NilsEling</span> ’s cytomapper R package! cytomapper includes a shinyApp to generate training data for automated celltype annotation methods. check it out! #bioc2020 <a href="https://t.co/Z22fYhUkq1" class="uri">https://t.co/Z22fYhUkq1</a>
</td>
<td style="text-align:right;">
0.9945472
</td>
</tr>
<tr>
<td style="text-align:left;">
Addgene
</td>
<td style="text-align:left;">
With our growing amount of plasmid sequence data we’re becoming more and more interested in the intersection of software and biological data (like DYK we have a <span class="citation">@github</span> page for bioinformatics tools <a href="https://t.co/e4w1Jf9POL" class="uri">https://t.co/e4w1Jf9POL</a>). We’re excited to attend #bioc2020 to learn more!
</td>
<td style="text-align:right;">
0.9945472
</td>
</tr>
<tr>
<td style="text-align:left;">
LeviWaldron1
</td>
<td style="text-align:left;">
Live-coding by an audience running a different workshop environment every hour in the Cloud, with the number of instances scaling up and down automatically to the audience size is amazing. #BioC2020 <span class="citation">@isb_cgc</span> <a href="https://t.co/f1EmD2fNCW" class="uri">https://t.co/f1EmD2fNCW</a>
</td>
<td style="text-align:right;">
0.9938706
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 5</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p><span class="citation">@niteshturaga</span> <span class="citation">@mt_morgan</span> <span class="citation">@kaylainter1011</span> <span class="citation">@lshep712</span> <span class="citation">@M2RUseR</span> <span class="citation">@Docker</span> <span class="citation">@rstudio</span> Learn more details about the BioC <span class="citation">@Docker</span> images by <span class="citation">@niteshturaga</span> et al (based on #rocker by <span class="citation">@rOpenSci</span> members) at</p>
<p><a href="https://t.co/wz7lISNHH0" class="uri">https://t.co/wz7lISNHH0</a></p>
<p>Info on rocker: <a href="https://t.co/29Yp2JKhSV" class="uri">https://t.co/29Yp2JKhSV</a> <span class="citation">@noamross</span> <span class="citation">@cboettig</span> <span class="citation">@eddelbuettel</span></p>
#BioC2020 #rstats (#TakeTwo messed up my from acc the 1st time) <a href="https://t.co/cnufYROdx6" class="uri">https://t.co/cnufYROdx6</a>
</td>
<td style="text-align:right;">
0.9968251
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Nitesh <span class="citation">@niteshturaga</span> Martin <span class="citation">@mt_morgan</span> Kayla <span class="citation">@kaylainter1011</span> Lori <span class="citation">@lshep712</span> Marcel <span class="citation">@M2RUseR</span> &amp;amp; more at the BioC team are showing how you can use <span class="citation">@Docker</span> to simplify your BioC #rstats experience</p>
<p>Coming soon: binary installations like the RSPM <span class="citation">@rstudio</span> &amp;amp; newer ubuntu😍😎</p>
#BioC2020 <a href="https://t.co/DfjxmhGYR1" class="uri">https://t.co/DfjxmhGYR1</a>
</td>
<td style="text-align:right;">
0.9959037
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Next up we have Leonardo Collado Torres <span class="citation">@fellgernon</span> for our 4th workshop: “Human RNA-seq data from #recount2 and related packages”</p>
<p>It’s his 3rd BioC recount2 workshop (2017, 19, 20)</p>
<p>Leo will also present a poster &amp;amp; BoF at #BioC2020</p>
<p><a href="https://t.co/gMjxJOmqlE" class="uri">https://t.co/gMjxJOmqlE</a></p>
#rstats #RNAseq <a href="https://t.co/BsY8z1LfOB" class="uri">https://t.co/BsY8z1LfOB</a>
</td>
<td style="text-align:right;">
0.9957265
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>For our 5th workshop we have James W MacDonald &amp;amp; Lori Shepherd <span class="citation">@lshep712</span> on “Introduction to Bioconductor annotation resources”</p>
<p>James and Lori are super starts in the BioC community, particularly at our Support website <a href="https://t.co/a2Nad6ehm2" class="uri">https://t.co/a2Nad6ehm2</a> &amp;amp; ⭐ instructors</p>
#BioC2020 #rstats <a href="https://t.co/i7mu7t0cD6" class="uri">https://t.co/i7mu7t0cD6</a>
</td>
<td style="text-align:right;">
0.9955333
</td>
</tr>
<tr>
<td style="text-align:left;">
chaos35002422
</td>
<td style="text-align:left;">
<p>Essential Cell Biology (Day 1 p5) ・Cells are the fundamental units of life ・and diverse in their chemical requirements and activities ・ different from the outside but similar inside ・viruses are chemical zombies</p>
#100DaysOfBio #Bioinformatics #Biology #BioC2020
</td>
<td style="text-align:right;">
0.9953218
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Next up Levi Waldron <span class="citation">@LeviWaldron1</span> Sean Davis <span class="citation">@seandavis12</span> and Benjamin Haibe-Kains <span class="citation">@bhaibeka</span> will teach the first 100-level #BioC2020 workshop on “Public data resources in Bioconductor”</p>
From Slack (#bioc2020-workshops) Levi clarified the levels: 100 learn, 200 use, 500 develop <a href="https://t.co/wO6lvqyVBW" class="uri">https://t.co/wO6lvqyVBW</a>
</td>
<td style="text-align:right;">
0.9953218
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
Great tips for #bioc2020 talk. My 3 tips 1) Excite me, What motivated you? Why should I listen? 2) Tell a story, what you did &amp;amp; what it means 3) Easy-2-read slides, Labels esp x,y axis in big font. Folks will read or listen not both together. Your 3 tips? <a href="https://t.co/3IbpqHMSxs" class="uri">https://t.co/3IbpqHMSxs</a>
</td>
<td style="text-align:right;">
0.9950892
</td>
</tr>
<tr>
<td style="text-align:left;">
rnomics
</td>
<td style="text-align:left;">
<a href="https://t.co/gouXsZGbmz" class="uri">https://t.co/gouXsZGbmz</a> Bioconductor: ’Next up we have Leonardo Collado Torres fellgernon for our 4th workshop: “Human RNA-seq data from #recount2 and related packages” It’s his 3rd BioC recount2 workshop (2017, 19, 20) Leo will also present a poster &amp;amp; BoF at #BioC2020 …
</td>
<td style="text-align:right;">
0.9950892
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
LG: Many methods (some of them just Fisher’s exact test though!), hard to evaluate them -&amp;gt; Decided to develop an approach and package to evaluate them! -&amp;gt; GSEABenchmarkeR <a href="https://t.co/CgeO5d1FSC" class="uri">https://t.co/CgeO5d1FSC</a> Publication at <a href="https://t.co/iYWsUufhr2" class="uri">https://t.co/iYWsUufhr2</a> #bioc2020
</td>
<td style="text-align:right;">
0.9945472
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Are you using Windows &amp;amp; want to run the #BioC2020 workshops locally using <span class="citation">@docker</span>?</p>
<p>Are you getting an error ⚠️about Windows not being up to date despite Windows Update saying you are up to date when installing Docker Desktop? <a href="https://t.co/wapuKn2Mox" class="uri">https://t.co/wapuKn2Mox</a></p>
Use <a href="https://t.co/cmHDmLuYN6" class="uri">https://t.co/cmHDmLuYN6</a> <a href="https://t.co/NDYlGVqekv" class="uri">https://t.co/NDYlGVqekv</a>
</td>
<td style="text-align:right;">
0.9942286
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 6</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
mikelove
</td>
<td style="text-align:left;">
<p>For a #bioc2020 speaker it’s nice to see attendees adding a workshop to their schedules in advance 😁</p>
<p>Talks are recorded and workshop material is online. My and <span class="citation">@k3yavi</span>’s vignette on importing alevin scRNA-seq counts with tximeta (work w <span class="citation">@nomad421</span> et al)</p>
<a href="https://t.co/bVbJJnZcHH" class="uri">https://t.co/bVbJJnZcHH</a> <a href="https://t.co/htI3Lox2sr" class="uri">https://t.co/htI3Lox2sr</a>
</td>
<td style="text-align:right;">
0.9959037
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
So so grateful to Johannes Rainer for redesigning the #bioc2020 conference stickers to reflect its new virtual location. The virus lurks overhead, but there is light from the <span class="citation">@Bioconductor</span> logo on the laptop. The mug includes a nod to the intended physical location (Boston). <a href="https://t.co/rmi9o9LRq9" class="uri">https://t.co/rmi9o9LRq9</a>
</td>
<td style="text-align:right;">
0.9957265
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
#bioc2020 virtual conference is next week. Our virtual platform is at capacity &amp;amp; registration is closed. If you missed out, info on where to view posters and workshop materials are on <a href="https://t.co/MofhfF8m8k" class="uri">https://t.co/MofhfF8m8k</a>. Talks will be recorded &amp;amp; videos pushed to youtube after the conference <a href="https://t.co/ltlKBOta6z" class="uri">https://t.co/ltlKBOta6z</a>
</td>
<td style="text-align:right;">
0.9955333
</td>
</tr>
<tr>
<td style="text-align:left;">
AedinCulhane
</td>
<td style="text-align:left;">
So so grateful to <span class="citation">@jo_rainer</span> for redesigning the #bioc2020 conference stickers to reflect its new virtual location. The virus lurks overhead, but there is light from the <span class="citation">@Bioconducto</span> logo on the laptop. The mug includes a nod to the intended physical location (Boston). <a href="https://t.co/h0iLDwRHvK" class="uri">https://t.co/h0iLDwRHvK</a>
</td>
<td style="text-align:right;">
0.9955333
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>Next up is Ting Huang <span class="citation">@huang704</span> on “MSstatsTMT: Statistical detection of differentially abundant proteins in experiments with isobaric labeling and multiple mixtures”</p>
<p>Ting is the second speaker in the session (sorry about the mistake earlier)</p>
#BioC2020 #RStats <a href="https://t.co/TdRRIPxs2G" class="uri">https://t.co/TdRRIPxs2G</a>
</td>
<td style="text-align:right;">
0.9953218
</td>
</tr>
<tr>
<td style="text-align:left;">
Bioconductor
</td>
<td style="text-align:left;">
<p>The last speaker of this first contributed talks session is Philippe Boileau <span class="citation">@p_boileau</span> on “Exploring high-dimensional biological data with sparse contrastive principal component analysis”</p>
<p>Thank you <span class="citation">@matthewnmccal</span> &amp;amp; <span class="citation">@CSoneson</span> for moderating!</p>
#BioC2020 #rstats <a href="https://t.co/iqEH6Am2G6" class="uri">https://t.co/iqEH6Am2G6</a>
</td>
<td style="text-align:right;">
0.9950892
</td>
</tr>
<tr>
<td style="text-align:left;">
RiyueSunnyBao
</td>
<td style="text-align:left;">
The #BioC2020 #virtual #platform is amazing. You can view entire #schedule on front page, click on each one, reserve your spot, and there is a chat box right there on the same page where you can type in questions. Meanwhile you can see who else are there and enter polls! <a href="https://t.co/mKPdWc8HKJ" class="uri">https://t.co/mKPdWc8HKJ</a>
</td>
<td style="text-align:right;">
0.9948324
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
“scPCA combines the stability and interpretability of sparse PCA with contrastive PCA’s ability to disentangle biological signal from technical noise through the use of control data.” Method also described in paper here <a href="https://t.co/FUD7Y9WbUU" class="uri">https://t.co/FUD7Y9WbUU</a> #bioc2020
</td>
<td style="text-align:right;">
0.9948324
</td>
</tr>
<tr>
<td style="text-align:left;">
siminaboca
</td>
<td style="text-align:left;">
RI just described work by Will Townes <span class="citation">@sandakano</span> <a href="https://t.co/nesgql7qVC" class="uri">https://t.co/nesgql7qVC</a> that proposes multinomial models to deal with zero-inflation and describes “generalized principal component analysis (GLM-PCA) for non-normal distributions” #bioc2020
</td>
<td style="text-align:right;">
0.9945472
</td>
</tr>
<tr>
<td style="text-align:left;">
jma1991
</td>
<td style="text-align:left;">
The final talk in the session is from Philippe Boileau from the University of California, Berkeley talking about “Exploring high-dimensional biological data with sparse contrastive principal component analysis” #BioC2020
</td>
<td style="text-align:right;">
0.9945472
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="software" class="section level1" number="9">
<h1 number="9"><span class="header-section-number">9</span> Software</h1>
<p>Software mentioned in Tweets with links to GitHub, BitBucket, Bioconductor or CRAN.</p>
<table>
<colgroup>
<col width="19%" />
<col width="11%" />
<col width="68%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Name</th>
<th align="left">Type</th>
<th align="left">Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">BiocStickers</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/bioconductor/biocstickers/" class="uri">https://github.com/bioconductor/biocstickers/</a></td>
</tr>
<tr class="even">
<td align="left">curatedMetagenomicData</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/curatedMetagenomicData" class="uri">https://bioconductor.org/packages/curatedMetagenomicData</a></td>
</tr>
<tr class="odd">
<td align="left">curatedTCGAData</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/curatedTCGAData" class="uri">https://bioconductor.org/packages/curatedTCGAData</a></td>
</tr>
<tr class="even">
<td align="left">dagitty</td>
<td align="left">CRAN</td>
<td align="left"><a href="https://CRAN.R-project.org/package=dagitty" class="uri">https://CRAN.R-project.org/package=dagitty</a></td>
</tr>
<tr class="odd">
<td align="left">dittoSeq</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dtm2451/dittoseq" class="uri">https://github.com/dtm2451/dittoseq</a></td>
</tr>
<tr class="even">
<td align="left">EnrichmentBrowser</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/EnrichmentBrowser" class="uri">https://bioconductor.org/packages/EnrichmentBrowser</a></td>
</tr>
<tr class="odd">
<td align="left">GenomicDataCommons</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/GenomicDataCommons" class="uri">https://bioconductor.org/packages/GenomicDataCommons</a></td>
</tr>
<tr class="even">
<td align="left">ggdag</td>
<td align="left">CRAN</td>
<td align="left"><a href="https://CRAN.R-project.org/package=ggdag" class="uri">https://CRAN.R-project.org/package=ggdag</a></td>
</tr>
<tr class="odd">
<td align="left">glmpca</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/willtownes/glmpca" class="uri">https://github.com/willtownes/glmpca</a></td>
</tr>
<tr class="even">
<td align="left">GSEABenchmarkeR</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/GSEABenchmarkeR" class="uri">https://bioconductor.org/packages/GSEABenchmarkeR</a></td>
</tr>
<tr class="odd">
<td align="left">netDx</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/netDx" class="uri">https://bioconductor.org/packages/netDx</a></td>
</tr>
<tr class="even">
<td align="left">Presentations</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/siminab/presentations/blob/master/bioc%202020/boca_swfdr.pdf" class="uri">https://github.com/siminab/presentations/blob/master/bioc%202020/boca_swfdr.pdf</a></td>
</tr>
<tr class="odd">
<td align="left">Presentations</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/siminab/presentations/tree/master/bioc%202020" class="uri">https://github.com/siminab/presentations/tree/master/bioc%202020</a></td>
</tr>
<tr class="even">
<td align="left">recount</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/leekgroup/recount/issues" class="uri">https://github.com/leekgroup/recount/issues</a></td>
</tr>
<tr class="odd">
<td align="left">recountWorkshop2020</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lieberinstitute/recountworkshop2020/" class="uri">https://github.com/lieberinstitute/recountworkshop2020/</a></td>
</tr>
<tr class="even">
<td align="left">spqn</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/spqn" class="uri">https://bioconductor.org/packages/spqn</a></td>
</tr>
<tr class="odd">
<td align="left">swfdr</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/leekgroup/swfdr/tree/dev" class="uri">https://github.com/leekgroup/swfdr/tree/dev</a></td>
</tr>
<tr class="even">
<td align="left">TCGAbiolinks</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/TCGAbiolinks" class="uri">https://bioconductor.org/packages/TCGAbiolinks</a></td>
</tr>
</tbody>
</table>
</div>
<div id="session-info" class="section level1 unnumbered" number="">
<h1 class="unnumbered" number="">Session info</h1>
<pre><code>## R version 4.0.0 (2020-04-24)
## Platform: x86_64-apple-darwin17.0 (64-bit)
## Running under: macOS Catalina 10.15.6
## 
## Matrix products: default
## BLAS:   /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRblas.dylib
## LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] fs_1.4.2           here_0.1           knitr_1.29         magick_2.4.0      
##  [5] webshot_0.5.2      viridis_0.5.1      viridisLite_0.3.0  wordcloud_2.6     
##  [9] RColorBrewer_1.1-2 ggraph_2.0.3       ggrepel_0.8.2      ggplot2_3.3.1     
## [13] topicmodels_0.2-11 tidytext_0.2.4     igraph_1.2.5       stringr_1.4.0     
## [17] purrr_0.3.4        forcats_0.5.0      lubridate_1.7.9    tidyr_1.1.0       
## [21] dplyr_1.0.0        rtweet_0.7.0      
## 
## loaded via a namespace (and not attached):
##  [1] httr_1.4.1         tidygraph_1.2.0    jsonlite_1.7.0     assertthat_0.2.1  
##  [5] askpass_1.1        highr_0.8          stats4_4.0.0       yaml_2.2.1        
##  [9] progress_1.2.2     slam_0.1-47        pillar_1.4.4       backports_1.1.8   
## [13] lattice_0.20-41    glue_1.4.1         digest_0.6.25      polyclip_1.10-0   
## [17] colorspace_1.4-1   plyr_1.8.6         htmltools_0.5.0    Matrix_1.2-18     
## [21] tm_0.7-7           pkgconfig_2.0.3    scales_1.1.1       processx_3.4.2    
## [25] tweenr_1.0.1       ggforce_0.3.1      tibble_3.0.2       openssl_1.4.2     
## [29] generics_0.0.2     farver_2.0.3       ellipsis_0.3.1     withr_2.2.0       
## [33] cli_2.0.2          NLP_0.2-0          magrittr_1.5       crayon_1.3.4      
## [37] ps_1.3.3           evaluate_0.14      tokenizers_0.2.1   janeaustenr_0.1.5 
## [41] fansi_0.4.1        SnowballC_0.7.0    MASS_7.3-51.6      xml2_1.3.2        
## [45] tools_4.0.0        prettyunits_1.1.1  hms_0.5.3          lifecycle_0.2.0   
## [49] munsell_0.5.0      callr_3.4.3        compiler_4.0.0     rlang_0.4.6       
## [53] grid_4.0.0         labeling_0.3       rmarkdown_2.3      gtable_0.3.0      
## [57] curl_4.3           reshape2_1.4.4     graphlayouts_0.7.0 R6_2.4.1          
## [61] gridExtra_2.3      utf8_1.1.4         rprojroot_1.3-2    modeltools_0.2-23 
## [65] stringi_1.4.6      parallel_4.0.0     Rcpp_1.0.4.6       png_0.1-7         
## [69] vctrs_0.3.1        tidyselect_1.1.0   xfun_0.15</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
