<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>biocasia2019</title>

<script src="site_libs/header-attrs-2.3/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "Óâô";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "Óâô";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lazappi's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/lazappi/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">biocasia2019</h1>
<h3 class="subtitle">BioC Asia 2019</h3>
<h4 class="date">Last built: 2020-09-01 18:04:36</h4>

</div>


<p>
<button class="btn btn-primary" data-toggle="collapse" data-target="#Parameters">
Parameters
</button>
</p>
<div id="Parameters" class="collapse">
<table>
<thead>
<tr class="header">
<th align="left">Parameter</th>
<th align="left">Value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">hashtag</td>
<td align="left">#biocasia</td>
</tr>
<tr class="even">
<td align="left">start_day</td>
<td align="left">2019-12-05</td>
</tr>
<tr class="odd">
<td align="left">end_day</td>
<td align="left">2019-12-06</td>
</tr>
<tr class="even">
<td align="left">timezone</td>
<td align="left">Australia/Sydney</td>
</tr>
<tr class="odd">
<td align="left">theme</td>
<td align="left">theme_light</td>
</tr>
<tr class="even">
<td align="left">accent</td>
<td align="left">#3f92ac</td>
</tr>
<tr class="odd">
<td align="left">accent2</td>
<td align="left">#9FC8D5</td>
</tr>
<tr class="even">
<td align="left">kcore</td>
<td align="left">2</td>
</tr>
<tr class="odd">
<td align="left">topics_k</td>
<td align="left">6</td>
</tr>
<tr class="even">
<td align="left">bigram_filter</td>
<td align="left">3</td>
</tr>
<tr class="odd">
<td align="left">fixed</td>
<td align="left">TRUE</td>
</tr>
<tr class="even">
<td align="left">seed</td>
<td align="left">1</td>
</tr>
</tbody>
</table>
</div>
<div id="introduction" class="section level1 unnumbered" number="">
<h1 class="unnumbered" number="">Introduction</h1>
<p>An analysis of tweets from the #biocasia hashtag. A total of 498 tweets from 113 users were collected using the <code>rtweet</code> R package.</p>
<p>An analysis of tweets from the #biocasia hashtag for the <a href="https://bioconductor.github.io/BiocAsia/" title="BioC Asia">BioC Asia</a> conference 2019.</p>
<p>A total of 498 tweets from 113 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1" number="1">
<h1 number="1"><span class="header-section-number">1</span> Timeline</h1>
<div id="tweets-by-day" class="section level2" number="1.1">
<h2 number="1.1"><span class="header-section-number">1.1</span> Tweets by day</h2>
<p><img src="figures/biocasia2019/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2" number="1.2">
<h2 number="1.2"><span class="header-section-number">1.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2019-12-05 - 2019-12-06 in the Australia/Sydney timezone.</p>
<p><img src="figures/biocasia2019/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1" number="2">
<h1 number="2"><span class="header-section-number">2</span> Users</h1>
<div id="top-tweeters" class="section level2 tabset tabset-pills" number="2.1">
<h2 class="tabset tabset-pills" number="2.1"><span class="header-section-number">2.1</span> Top tweeters</h2>
<div id="overall" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Overall</h3>
<p><img src="figures/biocasia2019/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="original" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Original</h3>
<p><img src="figures/biocasia2019/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweets" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Retweets</h3>
<p><img src="figures/biocasia2019/tweets-top-users-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweet-proportion" class="section level2" number="2.2">
<h2 number="2.2"><span class="header-section-number">2.2</span> Retweet proportion</h2>
<p><img src="figures/biocasia2019/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-timeline" class="section level2" number="2.3">
<h2 number="2.3"><span class="header-section-number">2.3</span> Top tweeters timeline</h2>
<p><img src="figures/biocasia2019/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2 tabset tabset-pills" number="2.4">
<h2 class="tabset tabset-pills" number="2.4"><span class="header-section-number">2.4</span> Top tweeters by day</h2>
<div id="overall-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Overall</h3>
<div id="day-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/biocasia2019/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/biocasia2019/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Original</h3>
<div id="day-1-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/biocasia2019/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/biocasia2019/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Retweets</h3>
<div id="day-1-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/biocasia2019/tweets-top-users-retweets-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/biocasia2019/tweets-top-users-retweets-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
</div>
<div id="sources" class="section level1 tabset tabset-pills" number="3">
<h1 class="tabset tabset-pills" number="3"><span class="header-section-number">3</span> Sources</h1>
<div id="users-1" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Users</h2>
<p><img src="figures/biocasia2019/tweets-top-sources-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Tweets</h2>
<p><img src="figures/biocasia2019/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="networks" class="section level1" number="4">
<h1 number="4"><span class="header-section-number">4</span> Networks</h1>
<div id="replies" class="section level2" number="4.1">
<h2 number="4.1"><span class="header-section-number">4.1</span> Replies</h2>
<p>The ‚Äúreplies network‚Äù, composed from users who reply directly to one another, coloured by PageRank.</p>
<p><img src="figures/biocasia2019/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2" number="4.2">
<h2 number="4.2"><span class="header-section-number">4.2</span> Mentions</h2>
<p>The ‚Äúmentions network‚Äù, where users mention other users in their tweets. Filtered for a k-core of 2. Node colour and size adjusted according to PageRank score.</p>
<p><img src="figures/biocasia2019/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="tweet-types" class="section level1" number="5">
<h1 number="5"><span class="header-section-number">5</span> Tweet types</h1>
<div id="retweets-2" class="section level2 tabset tabset-pills" number="5.1">
<h2 class="tabset tabset-pills" number="5.1"><span class="header-section-number">5.1</span> Retweets</h2>
<div id="proportion" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/biocasia2019/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/biocasia2019/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
_StuartLee
</td>
<td style="text-align:left;">
For #BiocAsia I‚Äôve written a workflow package for exploring and combining results from RNA-seq and ATAC-seq data. First iteration is up here: <a href="https://t.co/03oRjSfThQ" class="uri">https://t.co/03oRjSfThQ</a>
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
By constructing ‚Äòpsuedobulk‚Äô samples from scRNA-seq data, we can use tried and tested methods for differential expression analysis (limma, edgeR, DESeq2, etc) <span class="citation">@CrowellHL</span> benchmarked this approach along with other methods in <a href="https://t.co/SMNPfC7iyO" class="uri">https://t.co/SMNPfC7iyO</a> #biocasia <a href="https://t.co/jZwcPFN13i" class="uri">https://t.co/jZwcPFN13i</a>
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
trashystats
</td>
<td style="text-align:left;">
<p>For #BiocAsia <span class="citation">@PeteHaitch</span> and me are giving a tutorial on how to make a Bioconductor package üì¶ with <span class="citation">@rstudio</span>. You can find the tutorial here (feedback very welcome):</p>
<a href="https://t.co/y9vkKBgYSA" class="uri">https://t.co/y9vkKBgYSA</a>
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
G‚Äôday from #biocasia <a href="https://t.co/f2e2XNNkmG" class="uri">https://t.co/f2e2XNNkmG</a>
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Martin Morgan making the pitch at #BioInfoSummer / #biocasia to learn <span class="citation">@Bioconductor</span> There‚Äôs lots of learning material available: <a href="https://t.co/xjRvliJxmb" class="uri">https://t.co/xjRvliJxmb</a> And the project is essential to lots of current research: <a href="https://t.co/qppX0pRQTO" class="uri">https://t.co/qppX0pRQTO</a>
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
There are 27 <span class="citation">@Bioconductor</span> workflows for common genomic analyses (<a href="https://t.co/7qUnzfWepD" class="uri">https://t.co/7qUnzfWepD</a>) Since we‚Äôre at #biocasia/#BioInfoSummer, <span class="citation">@mritchieau</span> is highlighting the true blue, dinky di, Aussie-developed RNAseq123 workflow (<a href="https://t.co/z5niD5vbBx" class="uri">https://t.co/z5niD5vbBx</a>) available in Chinese and English!
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Delighted to have <span class="citation">@CrowellHL</span> as a keynote speaker at #biocasia! Helena and her colleagues in <span class="citation">@markrobinsonca</span>‚Äôs lab develop great <span class="citation">@Bioconductor</span> packages for CyTOF (<a href="https://t.co/ifzdnjDbLa" class="uri">https://t.co/ifzdnjDbLa</a>) and scRNA-seq (<a href="https://t.co/YAYEhwEEus" class="uri">https://t.co/YAYEhwEEus</a>) data. <a href="https://t.co/dGh9rwdla7" class="uri">https://t.co/dGh9rwdla7</a>
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
<span class="citation">@shazanfar</span> from the <span class="citation">@MarioniLab</span> introduces scHOT for flexible analysis of higher-order interactions in #singlecell data: shows us examples of variability, correlation and spatial analysis #biocasia. Code available at <a href="https://t.co/p8l2m21jqt" class="uri">https://t.co/p8l2m21jqt</a> <a href="https://t.co/41Nbf73QtU" class="uri">https://t.co/41Nbf73QtU</a>
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
<span class="citation">@JovMaksimovic</span> develops the missMethyl package which has the best logo in the business #biocasia <a href="https://t.co/JFMQdiJUoa" class="uri">https://t.co/JFMQdiJUoa</a> <a href="https://t.co/D5uXSQNtJB" class="uri">https://t.co/D5uXSQNtJB</a>
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
PuXue Qiao sharing the resources she, along with Ruqian Lyu and <span class="citation">@davisjmcc</span>, developed for a 2-day scRNA-seq course <a href="https://t.co/sq2q2HgEA6" class="uri">https://t.co/sq2q2HgEA6</a> #biocasia
</td>
<td style="text-align:right;">
8
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-retweeted" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most retweeted</h3>
<p><img src="figures/biocasia2019/most-retweeted-1.png" width="258" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="likes" class="section level2 tabset tabset-pills" number="5.2">
<h2 class="tabset tabset-pills" number="5.2"><span class="header-section-number">5.2</span> Likes</h2>
<div id="proportion-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/biocasia2019/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/biocasia2019/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
_StuartLee
</td>
<td style="text-align:left;">
For #BiocAsia I‚Äôve written a workflow package for exploring and combining results from RNA-seq and ATAC-seq data. First iteration is up here: <a href="https://t.co/03oRjSfThQ" class="uri">https://t.co/03oRjSfThQ</a>
</td>
<td style="text-align:right;">
56
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
G‚Äôday from #biocasia <a href="https://t.co/f2e2XNNkmG" class="uri">https://t.co/f2e2XNNkmG</a>
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
By constructing ‚Äòpsuedobulk‚Äô samples from scRNA-seq data, we can use tried and tested methods for differential expression analysis (limma, edgeR, DESeq2, etc) <span class="citation">@CrowellHL</span> benchmarked this approach along with other methods in <a href="https://t.co/SMNPfC7iyO" class="uri">https://t.co/SMNPfC7iyO</a> #biocasia <a href="https://t.co/jZwcPFN13i" class="uri">https://t.co/jZwcPFN13i</a>
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
trashystats
</td>
<td style="text-align:left;">
What a great keynote!!! It is quite unbelievable that <span class="citation">@CrowellHL</span> is only in her first year of her PhD. #biocasia Outstanding. <a href="https://t.co/DyR64Qx5ou" class="uri">https://t.co/DyR64Qx5ou</a>
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Charity Law, developer of voom, introducing an extension to voom to handle group-specific variance estimation and how she‚Äôs thinking about adapting it to scRNA-seq differential expression analysis #biocasia <a href="https://t.co/HOguf3wWpl" class="uri">https://t.co/HOguf3wWpl</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
trashystats
</td>
<td style="text-align:left;">
<p>For #BiocAsia <span class="citation">@PeteHaitch</span> and me are giving a tutorial on how to make a Bioconductor package üì¶ with <span class="citation">@rstudio</span>. You can find the tutorial here (feedback very welcome):</p>
<a href="https://t.co/y9vkKBgYSA" class="uri">https://t.co/y9vkKBgYSA</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
CrowellHL
</td>
<td style="text-align:left;">
Off to üá¶üá∫ for some Bioinformatics fun üë©üèª‚Äçüíª, to talk &amp;amp; teach about CyTOF at #BiocAsia / BioInfoSummer in Sydney, and explore other parts of AUS. Stressed, grateful and excited! - Now looking forward to 20h of reading &amp;amp; movies ‚úàÔ∏èüò¥ That completes my #BiocBingo ü•≥
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
It takes 5+ bioinformaticians to get the bloody projector working, thanks to <span class="citation">@JovMaksimovic</span> for being willing to improvise! Excited to hear her work on gene set testing for differentially methylated regions #biocasia <a href="https://t.co/R8OUlu4Hy6" class="uri">https://t.co/R8OUlu4Hy6</a>
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
You know it‚Äôs a <span class="citation">@DongXueyi</span> talk when Pokemon make an appearance. Xueyi is presenting her work on RNA-seq analysis of <span class="citation">@nanopore</span> data #biocasia <a href="https://t.co/PblubdtTSH" class="uri">https://t.co/PblubdtTSH</a>
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Lots of people in the house to learn about <span class="citation">@Bioconductor</span> from the leader of the project, Martin Morgan at #biocasia #BioInfoSummer <a href="https://t.co/lwPLmrBJJM" class="uri">https://t.co/lwPLmrBJJM</a>
</td>
<td style="text-align:right;">
20
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-likes" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most likes</h3>
<p><img src="figures/biocasia2019/most-favorited-1.png" width="258" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="quotes" class="section level2 tabset tabset-pills" number="5.3">
<h2 class="tabset tabset-pills" number="5.3"><span class="header-section-number">5.3</span> Quotes</h2>
<div id="proportion-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/biocasia2019/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/biocasia2019/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
JovMaksimovic
</td>
<td style="text-align:left;">
Hear, hear! <em>group hug</em> #biocasia <a href="https://t.co/OCvPxeCRj8" class="uri">https://t.co/OCvPxeCRj8</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
MTrussart
</td>
<td style="text-align:left;">
Wonderful talk from <span class="citation">@CrowellHL</span> at #BiocAsia showcasing #Catalyst and #diffcyt #R packages, i loved her enthusiasm and emotion towards the #Bioconductor community! <a href="https://t.co/DmF10cf1gC" class="uri">https://t.co/DmF10cf1gC</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
So many cool papers <span class="citation">@FertigLab</span> <span class="citation">@GenesOfEve</span> and colleagues to learn more about these methods e.g.¬†<a href="https://t.co/OsJH622gAo" class="uri">https://t.co/OsJH622gAo</a> and <a href="https://t.co/woblN8CxhC" class="uri">https://t.co/woblN8CxhC</a> #biocasia <a href="https://t.co/7NhkR0zyuG" class="uri">https://t.co/7NhkR0zyuG</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
JovMaksimovic
</td>
<td style="text-align:left;">
Final talk for #BiocAsia #BioInfoSummer <a href="https://t.co/YGwlhBuLlk" class="uri">https://t.co/YGwlhBuLlk</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
joshuawkho
</td>
<td style="text-align:left;">
Please come join me at the #BioCAsia conference! <a href="https://t.co/RVzv1jvCzz" class="uri">https://t.co/RVzv1jvCzz</a>
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
#biocasia <a href="https://t.co/q4IjB0zb27" class="uri">https://t.co/q4IjB0zb27</a>
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
JovMaksimovic
</td>
<td style="text-align:left;">
I really need to get onto the <span class="citation">@Bioconductor</span> slack #biocasia <a href="https://t.co/GffPVaX5F8" class="uri">https://t.co/GffPVaX5F8</a>
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
HannahCoughlan4
</td>
<td style="text-align:left;">
Thanks for the opportunity to present my work! #biocasia <a href="https://t.co/g9i3Abkm3E" class="uri">https://t.co/g9i3Abkm3E</a>
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
trashystats
</td>
<td style="text-align:left;">
This was a most impressive undergrad talk. Beautiful visualizations! #biocasia <a href="https://t.co/DZeUiqexlu" class="uri">https://t.co/DZeUiqexlu</a>
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
#biocasia <a href="https://t.co/DhQnm7H9md" class="uri">https://t.co/DhQnm7H9md</a>
</td>
<td style="text-align:right;">
1
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-quoted" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most quoted</h3>
<p><img src="figures/biocasia2019/most-quoted-1.png" width="258" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="media" class="section level1 tabset tabset-pills" number="6">
<h1 class="tabset tabset-pills" number="6"><span class="header-section-number">6</span> Media</h1>
<div id="proportion-3" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Proportion</h2>
<p><img src="figures/biocasia2019/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-3" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Top 10</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
G‚Äôday from #biocasia <a href="https://t.co/f2e2XNNkmG" class="uri">https://t.co/f2e2XNNkmG</a>
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
By constructing ‚Äòpsuedobulk‚Äô samples from scRNA-seq data, we can use tried and tested methods for differential expression analysis (limma, edgeR, DESeq2, etc) <span class="citation">@CrowellHL</span> benchmarked this approach along with other methods in <a href="https://t.co/SMNPfC7iyO" class="uri">https://t.co/SMNPfC7iyO</a> #biocasia <a href="https://t.co/jZwcPFN13i" class="uri">https://t.co/jZwcPFN13i</a>
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Charity Law, developer of voom, introducing an extension to voom to handle group-specific variance estimation and how she‚Äôs thinking about adapting it to scRNA-seq differential expression analysis #biocasia <a href="https://t.co/HOguf3wWpl" class="uri">https://t.co/HOguf3wWpl</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
It takes 5+ bioinformaticians to get the bloody projector working, thanks to <span class="citation">@JovMaksimovic</span> for being willing to improvise! Excited to hear her work on gene set testing for differentially methylated regions #biocasia <a href="https://t.co/R8OUlu4Hy6" class="uri">https://t.co/R8OUlu4Hy6</a>
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
You know it‚Äôs a <span class="citation">@DongXueyi</span> talk when Pokemon make an appearance. Xueyi is presenting her work on RNA-seq analysis of <span class="citation">@nanopore</span> data #biocasia <a href="https://t.co/PblubdtTSH" class="uri">https://t.co/PblubdtTSH</a>
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Lots of people in the house to learn about <span class="citation">@Bioconductor</span> from the leader of the project, Martin Morgan at #biocasia #BioInfoSummer <a href="https://t.co/lwPLmrBJJM" class="uri">https://t.co/lwPLmrBJJM</a>
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Exciting talk by <span class="citation">@shazanfar</span> bringing scHOT (<a href="https://t.co/1NSzfqWwLe" class="uri">https://t.co/1NSzfqWwLe</a>) to spatial transcriptomics, the cool new kid on the single cell block #biocasia <a href="https://t.co/4SsGLInsmk" class="uri">https://t.co/4SsGLInsmk</a>
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
annaquagli
</td>
<td style="text-align:left;">
Thanks to <span class="citation">@trashystats</span> for the detailed documentation and workshop on making <span class="citation">@Bioconductor</span> üì¶ . Did you know that you can go inside the R/myfun.R -&amp;gt; click on ‚ÄòCode‚Äô -&amp;gt; ‚ÄòInsert Roxygen Skeleton‚Äô ?ü§Ø #BiocAsia <a href="https://t.co/GDDjQW9Gny" class="uri">https://t.co/GDDjQW9Gny</a> <a href="https://t.co/urkJfLMP7v" class="uri">https://t.co/urkJfLMP7v</a>
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
The ‚Äúboring to some people but I think is really cool result‚Äù (<span class="citation">@CrowellHL</span>‚Äôs words, but I concur) is that running edgeR or limma on the summed counts works best and is many, many times faster and simpler than running a full-blown mixed model in their experience #biocasia <a href="https://t.co/YdYzI7UnpJ" class="uri">https://t.co/YdYzI7UnpJ</a>
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#biocasia #bioinfosummer <span class="citation">@trashystats</span> addressing the issue of overplotting of many many coloured points in a scatterplot, with Bioconductor package schex (pron ‚Äúess-see-hex‚Äù ü§£). Saskia is super cool under the live demo pressure, well done!! <a href="https://t.co/mywX9y9MMN" class="uri">https://t.co/mywX9y9MMN</a>
</td>
<td style="text-align:right;">
17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="section" class="section level1 unnumbered toc-ignore" number="">
<h1 class="unnumbered toc-ignore" number=""></h1>
<div id="most-liked-image" class="section level2" number="6.1">
<h2 number="6.1"><span class="header-section-number">6.1</span> Most liked image</h2>
<p><img src="http://pbs.twimg.com/media/EK_EFCQUEAAkHZ2.jpg" /></p>
</div>
</div>
<div id="tweet-text" class="section level1" number="7">
<h1 number="7"><span class="header-section-number">7</span> Tweet text</h1>
<div id="word-cloud" class="section level2" number="7.1">
<h2 number="7.1"><span class="header-section-number">7.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/biocasia2019/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="hashtags" class="section level2" number="7.2">
<h2 number="7.2"><span class="header-section-number">7.2</span> Hashtags</h2>
<p>Other hashtags used 5 or more times.</p>
<p><img src="figures/biocasia2019/hashtags-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="emojis" class="section level2" number="7.3">
<h2 number="7.3"><span class="header-section-number">7.3</span> Emojis</h2>
<p><img src="figures/biocasia2019/emojis-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2" number="7.4">
<h2 number="7.4"><span class="header-section-number">7.4</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 3 times.</p>
<p><img src="figures/biocasia2019/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2" number="7.5">
<h2 number="7.5"><span class="header-section-number">7.5</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/biocasia2019/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset tabset-fade" number="7.5.1">
<h3 class="tabset tabset-fade" number="7.5.1"><span class="header-section-number">7.5.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 1</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
paulfharrison
</td>
<td style="text-align:left;">
<p>Workshop by <span class="citation">@_StuartLee</span>: plyranges for fluent genomic ranges manipulation, loading Salmon data with tximeta, joining RNA-Seq and ATAC-Seq by range, and a whole lot of useful tricks along the way.</p>
<p><a href="https://t.co/7W0yEqWQ7d" class="uri">https://t.co/7W0yEqWQ7d</a></p>
#biocasia <a href="https://t.co/fnI4sRhwPY" class="uri">https://t.co/fnI4sRhwPY</a>
</td>
<td style="text-align:right;">
0.9963609
</td>
</tr>
<tr>
<td style="text-align:left;">
MnM_HealthCare
</td>
<td style="text-align:left;">
<p>Pharmacy Automation Market: Global Key Players, Trends, Industry Size</p>
<p>#Pharmacy #Automation #Market #robot #healthcare #medical #devices #health #healthy #BiocAsia</p>
<a href="https://t.co/fcBZ9eptTf" class="uri">https://t.co/fcBZ9eptTf</a>
</td>
<td style="text-align:right;">
0.9961702
</td>
</tr>
<tr>
<td style="text-align:left;">
malhamdoosh
</td>
<td style="text-align:left;">
Professor Pei Wang from <span class="citation">@IcahnMountSinai</span> at #biocasia 2019 explaining her DreamAI method at her workshop on methods and tools for preprocessing and imputation of mass spectrometry-based proteomics data <a href="https://t.co/2jZbPIIwaj" class="uri">https://t.co/2jZbPIIwaj</a>
</td>
<td style="text-align:right;">
0.9959585
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
There are 27 <span class="citation">@Bioconductor</span> workflows for common genomic analyses (<a href="https://t.co/7qUnzfWepD" class="uri">https://t.co/7qUnzfWepD</a>) Since we‚Äôre at #biocasia/#BioInfoSummer, <span class="citation">@mritchieau</span> is highlighting the true blue, dinky di, Aussie-developed RNAseq123 workflow (<a href="https://t.co/z5niD5vbBx" class="uri">https://t.co/z5niD5vbBx</a>) available in Chinese and English!
</td>
<td style="text-align:right;">
0.9957220
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>For those interested in keeping up to date with the festival of bioinformatics this &amp;amp; next week in Sydney, here‚Äôs my Tweetdeck search üòÇ</p>
<p>#BioInfoSummer OR <span class="citation">@bioinfosummer</span> OR #biocasia OR #abacbs OR <span class="citation">@abacbs</span> OR #GIWABACBS2019 OR #abacbs2019 OR #abacbs19</p>
did I cover all bases? üòÇ
</td>
<td style="text-align:right;">
0.9951549
</td>
</tr>
<tr>
<td style="text-align:left;">
_StuartLee
</td>
<td style="text-align:left;">
For #BiocAsia I‚Äôve written a workflow package for exploring and combining results from RNA-seq and ATAC-seq data. First iteration is up here: <a href="https://t.co/03oRjSfThQ" class="uri">https://t.co/03oRjSfThQ</a>
</td>
<td style="text-align:right;">
0.9948110
</td>
</tr>
<tr>
<td style="text-align:left;">
JovMaksimovic
</td>
<td style="text-align:left;">
Using formal data structures and data import functions in <span class="citation">@Bioconductor</span> can help you avoid common pitfalls such as dealing with 0 vs.¬†1 based coordinates #biocasia
</td>
<td style="text-align:right;">
0.9939524
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
<span class="citation">@joshuakho</span>‚Äôs paper on this work is at <a href="https://t.co/aBQvNFUCcc" class="uri">https://t.co/aBQvNFUCcc</a> Great collaboration with David Humphreys from <span class="citation">@VictorChangInst</span> driving methods and software development #biocasia <a href="https://t.co/OiRtwPzVtg" class="uri">https://t.co/OiRtwPzVtg</a>
</td>
<td style="text-align:right;">
0.9939524
</td>
</tr>
<tr>
<td style="text-align:left;">
trashystats
</td>
<td style="text-align:left;">
Last round of workshops. <span class="citation">@_StuartLee</span> is giving his very interesting workshop of using plyranges to highlight the genomic context of your DA or DE results. #biocasia <a href="https://t.co/fikQxgovdR" class="uri">https://t.co/fikQxgovdR</a>
</td>
<td style="text-align:right;">
0.9939524
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>Added, thanks Andrew!!</p>
#BioInfoSummer OR <span class="citation">@bioinfosummer</span> OR #biocasia OR #abacbs OR <span class="citation">@abacbs</span> OR #GIWABACBS2019 OR #abacbs2019 OR #abacbs19 OR #COMBINE19 OR <span class="citation">@combine_au</span> OR #COMBINE2019 <a href="https://t.co/GcPXYQY7ML" class="uri">https://t.co/GcPXYQY7ML</a>
</td>
<td style="text-align:right;">
0.9934070
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 2</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>What is ethics? Inner morals &amp;amp; culture play a role, but are not the same as ethical theories.</p>
<p>E.g.s: consequentialism (the goodness of the outcome is what matters), deontology (what is moral is good), &amp;amp; virtue ethics (is the person‚Äôs motivation good?)</p>
#biocasia #bioinfosummer
</td>
<td style="text-align:right;">
0.9959585
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
<span class="citation">@Bioconductor</span> invests time and effort into developing reusable data structures that are specifically designed for bioinformatics datasets, like SummarizedExperiment. Using these saves you developing your own and improves interoperability between packages #biocasia <a href="https://t.co/WCnMuTI0RK" class="uri">https://t.co/WCnMuTI0RK</a>
</td>
<td style="text-align:right;">
0.9959585
</td>
</tr>
<tr>
<td style="text-align:left;">
annaquagli
</td>
<td style="text-align:left;">
<span class="citation">@MTrussart</span> discussing all of #Cytof at #biocasia. Chat to her to know more about her debarcoding method using Gaussian mixtures, #reproducibility and removing batch effect with the #ruv #rstats üì¶, #normalization and cell #clustering! <a href="https://t.co/Ysr9InoDhs" class="uri">https://t.co/Ysr9InoDhs</a> <a href="https://t.co/2Beaa8jXqU" class="uri">https://t.co/2Beaa8jXqU</a>
</td>
<td style="text-align:right;">
0.9959585
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
<span class="citation">@shazanfar</span> Yay to #biocasia for having this! üéâ I think all conferences should have one invited ‚Äúcommunity talk‚Äù that is relevant to the audience but not a research project. Ethics is one example but also patient stories, mental health, workplace practices, career prospects etc.
</td>
<td style="text-align:right;">
0.9959585
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Delighted to have <span class="citation">@CrowellHL</span> as a keynote speaker at #biocasia! Helena and her colleagues in <span class="citation">@markrobinsonca</span>‚Äôs lab develop great <span class="citation">@Bioconductor</span> packages for CyTOF (<a href="https://t.co/ifzdnjDbLa" class="uri">https://t.co/ifzdnjDbLa</a>) and scRNA-seq (<a href="https://t.co/YAYEhwEEus" class="uri">https://t.co/YAYEhwEEus</a>) data. <a href="https://t.co/dGh9rwdla7" class="uri">https://t.co/dGh9rwdla7</a>
</td>
<td style="text-align:right;">
0.9954561
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Martin Morgan making the pitch at #BioInfoSummer / #biocasia to learn <span class="citation">@Bioconductor</span> There‚Äôs lots of learning material available: <a href="https://t.co/xjRvliJxmb" class="uri">https://t.co/xjRvliJxmb</a> And the project is essential to lots of current research: <a href="https://t.co/qppX0pRQTO" class="uri">https://t.co/qppX0pRQTO</a>
</td>
<td style="text-align:right;">
0.9948110
</td>
</tr>
<tr>
<td style="text-align:left;">
trashystats
</td>
<td style="text-align:left;">
<p><span class="citation">@shazanfar</span> is absolutely filling John‚Äôs massive shoes (literally and figuratively) by presenting scHOT which allows investigation of higher order statistics in single cell data #biocasia.</p>
<a href="https://t.co/FNSAlOjfup" class="uri">https://t.co/FNSAlOjfup</a>
</td>
<td style="text-align:right;">
0.9948110
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
It takes 5+ bioinformaticians to get the bloody projector working, thanks to <span class="citation">@JovMaksimovic</span> for being willing to improvise! Excited to hear her work on gene set testing for differentially methylated regions #biocasia <a href="https://t.co/R8OUlu4Hy6" class="uri">https://t.co/R8OUlu4Hy6</a>
</td>
<td style="text-align:right;">
0.9948110
</td>
</tr>
<tr>
<td style="text-align:left;">
JovMaksimovic
</td>
<td style="text-align:left;">
.@mritchieau telling everyone at #biocasia / #BioInfoSummer about his lab‚Äôs RNAseq 123 <span class="citation">@Bioconductor</span> workflow and giving a shout out to Melbourne‚Äôs local #bioconductor and #rstats Jedi Master - Gordon Smyth. <a href="https://t.co/V817fxFEeZ" class="uri">https://t.co/V817fxFEeZ</a>
</td>
<td style="text-align:right;">
0.9948110
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
<span class="citation">@MTrussart</span> introduces us to CyTOF analysis, how to preprocess these data, a novel barcoding strategy to increase cell number and the use of RUV for removing batch effects #biocasia <a href="https://t.co/v1tiB4bSlB" class="uri">https://t.co/v1tiB4bSlB</a>
</td>
<td style="text-align:right;">
0.9948110
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-3" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 3</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Dr Lisa Drive, bioethicist from the USyd Philosophy dept, discussing concepts of ethics in the context of precision medicine. It‚Äôs impossible to deny the wider impact that medical/clinical bioinformatics can have upon members of society #biocasia #bioinfosummer
</td>
<td style="text-align:right;">
0.9965334
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
<span class="citation">@MTrussart</span> telling us the best way to deal with batch effects in CyTOF (or indeed any experiment) is to design the experiment properly! If you plan by expecting batch effects, e.g.¬†including control samples in each batch, then you can adjust for them #biocasia <a href="https://t.co/DX40j4O1B8" class="uri">https://t.co/DX40j4O1B8</a>
</td>
<td style="text-align:right;">
0.9963609
</td>
</tr>
<tr>
<td style="text-align:left;">
carroll_jono
</td>
<td style="text-align:left;">
I‚Äôll be in Sydney for #BioCAsia as of tomorrow afternoon - any #rstats peeps willing to brave the smoke for dinner and/or drinks around Circular Quay? I‚Äôve never been to Sydney so I‚Äôll be checking out the gardens, Opera House, and the bridge like a proper tourist. <a href="https://t.co/QR8VD62gg4" class="uri">https://t.co/QR8VD62gg4</a>
</td>
<td style="text-align:right;">
0.9963609
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#biocasia <span class="citation">@paulfharrison</span> presenting Topconfects (pub <a href="https://t.co/5mYDprcT27" class="uri">https://t.co/5mYDprcT27</a> ) that ranks genes based on confidence bounds on the log-fold changes. Furthering <span class="citation">@davisjmcc</span> &amp;amp; Smyth‚Äôs TREAT (pub <a href="https://t.co/iZeszEIkDU" class="uri">https://t.co/iZeszEIkDU</a> ) Software: <a href="https://t.co/5H3KA4lFZs" class="uri">https://t.co/5H3KA4lFZs</a> works nicely with limma/edgeR
</td>
<td style="text-align:right;">
0.9961702
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>#BioInfoSummer #biocasia <span class="citation">@FertigLab</span> describes software for performing non-negative matrix factorisation (CoGAPS) and projection transfer learning approaches (projectR) for high throughput genomics data</p>
<p><a href="https://t.co/41ldgzcdxK" class="uri">https://t.co/41ldgzcdxK</a></p>
<a href="https://t.co/5FojtTftWe" class="uri">https://t.co/5FojtTftWe</a>
</td>
<td style="text-align:right;">
0.9959585
</td>
</tr>
<tr>
<td style="text-align:left;">
Al__Forrest
</td>
<td style="text-align:left;">
Enjoying #biocasia We have a great lineup of speakers in human genomics at #HGM2020 in Perth April 5-8 <a href="https://t.co/S4dGO1LRpe" class="uri">https://t.co/S4dGO1LRpe</a> Earlybird closes Dec 16th. Over 40 speakers to be selected from the submitted abstracts!!
</td>
<td style="text-align:right;">
0.9957220
</td>
</tr>
<tr>
<td style="text-align:left;">
annaquagli
</td>
<td style="text-align:left;">
Thanks to <span class="citation">@trashystats</span> for the detailed documentation and workshop on making <span class="citation">@Bioconductor</span> üì¶ . Did you know that you can go inside the R/myfun.R -&amp;gt; click on ‚ÄòCode‚Äô -&amp;gt; ‚ÄòInsert Roxygen Skeleton‚Äô ?ü§Ø #BiocAsia <a href="https://t.co/GDDjQW9Gny" class="uri">https://t.co/GDDjQW9Gny</a> <a href="https://t.co/urkJfLMP7v" class="uri">https://t.co/urkJfLMP7v</a>
</td>
<td style="text-align:right;">
0.9948110
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
And that‚Äôs a wrap for #BioInfoSummer #biocasia 2019. Congrats to the poster prize winners and thanks again to all the speakers, participants, organisers and generous sponsors especially <span class="citation">@DiscoverAMSI</span> for an awesome event! #staycurious <a href="https://t.co/e4JvGiYzKm" class="uri">https://t.co/e4JvGiYzKm</a>
</td>
<td style="text-align:right;">
0.9948110
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
<span class="citation">@rafalab</span> presenting his work estimating death rates following Hurricane Maria in Puerto Rico. Here‚Äôs the blog post showing how much of the work was in cleaning the data <a href="https://t.co/gi8ixEkSZQ" class="uri">https://t.co/gi8ixEkSZQ</a> #biocasia #BioInfoSummer
</td>
<td style="text-align:right;">
0.9944145
</td>
</tr>
<tr>
<td style="text-align:left;">
PNgsabrina
</td>
<td style="text-align:left;">
Beautiful work and personal journey by <span class="citation">@CrowellHL</span> from <span class="citation">@UZH_en</span> on the masterminds behind CyTOF, Catalyst, then muscat. #bioconductor #BiocAsia <span class="citation">@Bioinfosummer</span> üòä <span class="citation">@DiscoverAMSI</span> It‚Äôs inspiring to see one the cherishs her work with great passion. <a href="https://t.co/tI9wHl4jbK" class="uri">https://t.co/tI9wHl4jbK</a>
</td>
<td style="text-align:right;">
0.9939524
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 4</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
annaquagli
</td>
<td style="text-align:left;">
Super interesting discussion by Charity Law from <span class="citation">@mritchieau</span> lab about how different levels of variation (BCV in the #edgeR world) infleluence the significance of differential expression analysis‚Ä¶ PS. I have rarely seen BCV &amp;lt; 0.4/0.5 in my samples üò≠ #biocasia <a href="https://t.co/NlXzghbNb7" class="uri">https://t.co/NlXzghbNb7</a>
</td>
<td style="text-align:right;">
0.9968337
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>PuXue Xiao with her newly minted PhD (congrats!!), postdoc with <span class="citation">@davisjmcc</span> presenting a great talk on the important problem of nearest-neighbour graph construction prior to single cell clustering #biocasia</p>
PuXue will be presenting a single cell workshop later today! <a href="https://t.co/7YOssXZpGN" class="uri">https://t.co/7YOssXZpGN</a>
</td>
<td style="text-align:right;">
0.9965334
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
The ‚Äúboring to some people but I think is really cool result‚Äù (<span class="citation">@CrowellHL</span>‚Äôs words, but I concur) is that running edgeR or limma on the summed counts works best and is many, many times faster and simpler than running a full-blown mixed model in their experience #biocasia <a href="https://t.co/YdYzI7UnpJ" class="uri">https://t.co/YdYzI7UnpJ</a>
</td>
<td style="text-align:right;">
0.9965334
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
<span class="citation">@mt_morgan</span> encourages us to participate in #bioconductor #slack. Has been a great vehicle for community participation &amp;amp; collaboration. Highlights include development of iSEE &amp;amp; the #singlecell analysis cookbook <a href="https://t.co/CLrXYW2kmB" class="uri">https://t.co/CLrXYW2kmB</a>. Sign up at <a href="https://t.co/Qa7OwOPQdG" class="uri">https://t.co/Qa7OwOPQdG</a> #biocasia <a href="https://t.co/tRJlxAPWYY" class="uri">https://t.co/tRJlxAPWYY</a>
</td>
<td style="text-align:right;">
0.9961702
</td>
</tr>
<tr>
<td style="text-align:left;">
DrHelenMcGuire
</td>
<td style="text-align:left;">
#IAmAnImmunologist is INVITED SPEAKING! Super pleased to be invited to share my passion for #immunology and #highdimensional #masscytometry at #Bioinfosummer #biocasia <span class="citation">@Bioinfosummer</span> hosted at <span class="citation">@Sydney_Uni</span> <span class="citation">@CPC_EMCR</span> <span class="citation">@SydneyCytometry</span> <span class="citation">@ASImmunology</span> <span class="citation">@ACScytometry</span> <a href="https://t.co/Ln14hNjSyM" class="uri">https://t.co/Ln14hNjSyM</a>
</td>
<td style="text-align:right;">
0.9959585
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#biocasia 2019 is now coming to a close, organiser <span class="citation">@PeteHaitch</span> closing the meeting with news that BiocAsia will be coming to China in 2020! Thank you to the organising committee Pete, Steve Pederson, <span class="citation">@trashystats</span> and Dario Strbenac for an excellent meeting! <a href="https://t.co/z26y8LFOnI" class="uri">https://t.co/z26y8LFOnI</a>
</td>
<td style="text-align:right;">
0.9959585
</td>
</tr>
<tr>
<td style="text-align:left;">
JovMaksimovic
</td>
<td style="text-align:left;">
Cab driver got me to Sydney airport in plenty of time for my flight, despite peak hour traffic. It was the jerkiest ride though - my stomach is still recovering from the g-forces ü§¢ü§Æ #biocasia
</td>
<td style="text-align:right;">
0.9951549
</td>
</tr>
<tr>
<td style="text-align:left;">
JovMaksimovic
</td>
<td style="text-align:left;">
I‚Äôve a great time at #biocasia but it‚Äôs nearly time to jet back to Melbourne. Thanks to the organising committee for a fab meeting: <span class="citation">@PeteHaitch</span> <span class="citation">@trashystats</span> Dario Strbenac and Stephen Pederson üèÜ
</td>
<td style="text-align:right;">
0.9939524
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
So many cool papers <span class="citation">@FertigLab</span> <span class="citation">@GenesOfEve</span> and colleagues to learn more about these methods e.g.¬†<a href="https://t.co/OsJH622gAo" class="uri">https://t.co/OsJH622gAo</a> and <a href="https://t.co/woblN8CxhC" class="uri">https://t.co/woblN8CxhC</a> #biocasia <a href="https://t.co/7NhkR0zyuG" class="uri">https://t.co/7NhkR0zyuG</a>
</td>
<td style="text-align:right;">
0.9934070
</td>
</tr>
<tr>
<td style="text-align:left;">
brian_gloss
</td>
<td style="text-align:left;">
hey #Bioinfosummer #biocasia people (talking about spillover), is it possible to just consider each fluorophore from its distribution across n-dimensions (depending on detectors) and do away with correcting?
</td>
<td style="text-align:right;">
0.9927534
</td>
</tr>
<tr>
<td style="text-align:left;">
brian_gloss
</td>
<td style="text-align:left;">
<span class="citation">@CrowellH</span> highlighting the vital importance of a ground truth in evaluating methods especially in emerging technologies #Bioinfosummer #biocasia
</td>
<td style="text-align:right;">
0.9927534
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 5</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
<span class="citation">@paulfharrison</span> introduces us to the ‚ÄòFalse Coverage-statement Rate‚Äô concept which seems like a neat idea. Works with limma, edgeR, DESeq2 differential expression analysis pipelines and can be used in #geneset testing. Slides on topconfects at <a href="https://t.co/wbAOqjtCw2" class="uri">https://t.co/wbAOqjtCw2</a> #biocasia <a href="https://t.co/tHtpyYPWg9" class="uri">https://t.co/tHtpyYPWg9</a>
</td>
<td style="text-align:right;">
0.9965334
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#bioinfosummer #biocasia Elana Fertig <span class="citation">@FertigLab</span> presenting work on non-negative matrix factorisation techniques for genomics data. Using the gamma distribution (equiv to sum of exponentials) allows for generalisability as well as reflecting sparsity in the underlying system <a href="https://t.co/ZCKtjN8743" class="uri">https://t.co/ZCKtjN8743</a>
</td>
<td style="text-align:right;">
0.9963609
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Charity Law, developer of voom, introducing an extension to voom to handle group-specific variance estimation and how she‚Äôs thinking about adapting it to scRNA-seq differential expression analysis #biocasia <a href="https://t.co/HOguf3wWpl" class="uri">https://t.co/HOguf3wWpl</a>
</td>
<td style="text-align:right;">
0.9961702
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
By constructing ‚Äòpsuedobulk‚Äô samples from scRNA-seq data, we can use tried and tested methods for differential expression analysis (limma, edgeR, DESeq2, etc) <span class="citation">@CrowellHL</span> benchmarked this approach along with other methods in <a href="https://t.co/SMNPfC7iyO" class="uri">https://t.co/SMNPfC7iyO</a> #biocasia <a href="https://t.co/jZwcPFN13i" class="uri">https://t.co/jZwcPFN13i</a>
</td>
<td style="text-align:right;">
0.9961702
</td>
</tr>
<tr>
<td style="text-align:left;">
CFlensburg
</td>
<td style="text-align:left;">
<p>FOMO alert! Everyone in bioinformatics and #rstats should have a scroll through the bioconductor #biocasia hashtag from the last couple of days!</p>
And Monday next week I‚Äôm joining Sydney for the bioinformatics conference #abacbs2019, I might be live-tweeting a bit. :) <a href="https://t.co/scPHdyoNcu" class="uri">https://t.co/scPHdyoNcu</a>
</td>
<td style="text-align:right;">
0.9961702
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
We‚Äôve had scRNA, CyTOF, and now Hi-C with <span class="citation">@HannahCoughlan4</span> telling us about a gene-oriented analysis approach using limma and edgeR, tools originally developed for differential expression analysis. Absolute workhorses of <span class="citation">@Bioconductor</span> #biocasia <a href="https://t.co/4ZI4qf6rkD" class="uri">https://t.co/4ZI4qf6rkD</a>
</td>
<td style="text-align:right;">
0.9959585
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
<span class="citation">@DongXueyi</span> tells us how to analyse #longread <span class="citation">@nanopore</span> #RNAseq data using #bioconductor. Schmd1 data (both long and short reads) from <span class="citation">@BlewittMarnie</span> lab. New tools still needed for accurate isoform analysis #biocasia <a href="https://t.co/rjrExr6jv6" class="uri">https://t.co/rjrExr6jv6</a>
</td>
<td style="text-align:right;">
0.9957220
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
<span class="citation">@paulfharrison</span> advocating for the use of confidence intervals over p-values when reporting differential expression analysis results #biocasia This idea implemented in topconfects <span class="citation">@Bioconductor</span> package: <a href="https://t.co/8bKQQk0Bg2" class="uri">https://t.co/8bKQQk0Bg2</a> Slides: <a href="https://t.co/qJqFLwWzbB" class="uri">https://t.co/qJqFLwWzbB</a> <a href="https://t.co/O3qL6A7Ov7" class="uri">https://t.co/O3qL6A7Ov7</a>
</td>
<td style="text-align:right;">
0.9957220
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#biocasia #bioinfosummer <span class="citation">@trashystats</span> addressing the issue of overplotting of many many coloured points in a scatterplot, with Bioconductor package schex (pron ‚Äúess-see-hex‚Äù ü§£). Saskia is super cool under the live demo pressure, well done!! <a href="https://t.co/mywX9y9MMN" class="uri">https://t.co/mywX9y9MMN</a>
</td>
<td style="text-align:right;">
0.9957220
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Closing out the first session of #biocasia, we‚Äôve got our first presentation from an undergrad! Kathleen Zeglinski telling us about gmoviz, a package she is developing at <span class="citation">@CSL</span> to visualise genome-editing data <a href="https://t.co/GsixCMALXD" class="uri">https://t.co/GsixCMALXD</a>
</td>
<td style="text-align:right;">
0.9954561
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 6</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Steve Pederson breaking the space-time continuum in 5min talk at #biocasia, reading out a letter to his future self from his past self about the experience of submitting his first <span class="citation">@Bioconductor</span> package. Love someone who gets creative for a presentation <a href="https://t.co/XfcLxmbZYW" class="uri">https://t.co/XfcLxmbZYW</a>
</td>
<td style="text-align:right;">
0.9961702
</td>
</tr>
<tr>
<td style="text-align:left;">
CrowellHL
</td>
<td style="text-align:left;">
Off to üá¶üá∫ for some Bioinformatics fun üë©üèª‚Äçüíª, to talk &amp;amp; teach about CyTOF at #BiocAsia / BioInfoSummer in Sydney, and explore other parts of AUS. Stressed, grateful and excited! - Now looking forward to 20h of reading &amp;amp; movies ‚úàÔ∏èüò¥ That completes my #BiocBingo ü•≥
</td>
<td style="text-align:right;">
0.9957220
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Phenomenal start to Helena Crowell‚Äôs <span class="citation">@CrowellHL</span> ( <span class="citation">@markrobinsonca</span> lab ) #biocasia #bioinfosummer keynote on Differential discovery in high-dimensional cytometry data. Beginning with a timeline of important work in this space by a number of key contributors <a href="https://t.co/VStuFvtEMA" class="uri">https://t.co/VStuFvtEMA</a>
</td>
<td style="text-align:right;">
0.9957220
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
<span class="citation">@shazanfar</span> from the <span class="citation">@MarioniLab</span> introduces scHOT for flexible analysis of higher-order interactions in #singlecell data: shows us examples of variability, correlation and spatial analysis #biocasia. Code available at <a href="https://t.co/p8l2m21jqt" class="uri">https://t.co/p8l2m21jqt</a> <a href="https://t.co/41Nbf73QtU" class="uri">https://t.co/41Nbf73QtU</a>
</td>
<td style="text-align:right;">
0.9954561
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#biocasia Joshua Ho <span class="citation">@joshuawkho</span> presenting Ularcirc (a back-splice event on Circular :D ), circular RNA analysis package available on <span class="citation">@Bioconductor</span> <a href="https://t.co/jFxiHNhnOg" class="uri">https://t.co/jFxiHNhnOg</a> Tutorial video <a href="https://t.co/bnZhk3tNCu" class="uri">https://t.co/bnZhk3tNCu</a>
</td>
<td style="text-align:right;">
0.9948110
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Exciting talk by <span class="citation">@shazanfar</span> bringing scHOT (<a href="https://t.co/1NSzfqWwLe" class="uri">https://t.co/1NSzfqWwLe</a>) to spatial transcriptomics, the cool new kid on the single cell block #biocasia <a href="https://t.co/4SsGLInsmk" class="uri">https://t.co/4SsGLInsmk</a>
</td>
<td style="text-align:right;">
0.9944145
</td>
</tr>
<tr>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
The first of the AMSI #BioInfoSummer 2019 joint sessions with #biocasia will start at 1.30pm with 3 workshops to choose from sponsored by <span class="citation">@CMRI_AUS</span> on our mass spec analytics day
</td>
<td style="text-align:right;">
0.9944145
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
PuXue Qiao sharing the resources she, along with Ruqian Lyu and <span class="citation">@davisjmcc</span>, developed for a 2-day scRNA-seq course <a href="https://t.co/sq2q2HgEA6" class="uri">https://t.co/sq2q2HgEA6</a> #biocasia
</td>
<td style="text-align:right;">
0.9934070
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>Great talk by Ruqian Lyu <span class="citation">@Ruqian_lyu</span> on calculating genetic length from crossover events using COmapR, on Bioconductor! <a href="https://t.co/VNTMQBJAj1" class="uri">https://t.co/VNTMQBJAj1</a></p>
#BioInfoSummer #biocasia
</td>
<td style="text-align:right;">
0.9934070
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Watch the demo of using the Ularcirc #bioconductor package for visualising and analysing circular RNAs at <a href="https://t.co/P0ViJ7pi43" class="uri">https://t.co/P0ViJ7pi43</a> #biocasia <a href="https://t.co/3olI9ltMST" class="uri">https://t.co/3olI9ltMST</a>
</td>
<td style="text-align:right;">
0.9927534
</td>
</tr>
<tr>
<td style="text-align:left;">
One_News_Page
</td>
<td style="text-align:left;">
Afghans demand justice amid attempt to probe war crimes at ICC: <a href="https://t.co/atyUQAyww8" class="uri">https://t.co/atyUQAyww8</a> #biocasia
</td>
<td style="text-align:right;">
0.9927534
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
<span class="citation">@haoyyang</span> bringing us home on Day 1 with an overview of software for single-cell ATAC-seq #biocasia <a href="https://t.co/RyAq7Zhyhz" class="uri">https://t.co/RyAq7Zhyhz</a>
</td>
<td style="text-align:right;">
0.9927534
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="links" class="section level1" number="8">
<h1 number="8"><span class="header-section-number">8</span> Links</h1>
<p>Links to GitHub, GitLab, BitBucket, Bioconductor or CRAN mentioned in Tweets.</p>
<table>
<colgroup>
<col width="9%" />
<col width="3%" />
<col width="4%" />
<col width="3%" />
<col width="77%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Name</th>
<th align="right">Tweets</th>
<th align="right">Retweets</th>
<th align="left">Type</th>
<th align="left">Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">CATALYST</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">Bioc</td>
<td align="left"><a href="https://bioconductor.org/packages/CATALYST">bioconductor.org/packages/CATALYST</a></td>
</tr>
<tr class="even">
<td align="left">CoGAPS.html</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">Bioc</td>
<td align="left"><a href="https://bioconductor.org/packages/CoGAPS.html">bioconductor.org/packages/CoGAPS.html</a></td>
</tr>
<tr class="odd">
<td align="left">muscat</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">Bioc</td>
<td align="left"><a href="https://bioconductor.org/packages/muscat">bioconductor.org/packages/muscat</a></td>
</tr>
<tr class="even">
<td align="left">projectR.html</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">Bioc</td>
<td align="left"><a href="https://bioconductor.org/packages/projectR.html">bioconductor.org/packages/projectR.html</a></td>
</tr>
<tr class="odd">
<td align="left">scHOT2019</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/marionilab/schot2019">marionilab/schot2019</a></td>
</tr>
<tr class="even">
<td align="left">topconfects</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">Bioc</td>
<td align="left"><a href="https://bioconductor.org/packages/topconfects">bioconductor.org/packages/topconfects</a></td>
</tr>
<tr class="odd">
<td align="left">topconfects.html</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">Bioc</td>
<td align="left"><a href="https://bioconductor.org/packages/topconfects.html">bioconductor.org/packages/topconfects.html</a></td>
</tr>
<tr class="even">
<td align="left">ttBulk</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/stemangiola/ttbulk">stemangiola/ttbulk</a></td>
</tr>
<tr class="odd">
<td align="left">Ularcirc.html</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">Bioc</td>
<td align="left"><a href="https://bioconductor.org/packages/Ularcirc.html">bioconductor.org/packages/Ularcirc.html</a></td>
</tr>
<tr class="even">
<td align="left">muscat-comparison</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/helenalc/muscat-comparison/blob/master/magl/docs/index.html">helenalc/muscat-comparison/blob/master/magl/docs/index.html</a></td>
</tr>
</tbody>
</table>
</div>
<div id="session-info" class="section level1 unnumbered" number="">
<h1 class="unnumbered" number="">Session info</h1>
<p>
<button class="btn btn-primary" data-toggle="collapse" data-target="#SessionInfo">
Session info
</button>
</p>
<div id="SessionInfo" class="collapse">
<pre><code>## ‚îÄ Session info ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ
##  setting  value                       
##  version  R version 4.0.0 (2020-04-24)
##  os       macOS Catalina 10.15.6      
##  system   x86_64, darwin17.0          
##  ui       X11                         
##  language (EN)                        
##  collate  en_US.UTF-8                 
##  ctype    en_US.UTF-8                 
##  tz       Europe/Berlin               
##  date     2020-09-01                  
## 
## ‚îÄ Packages ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ
##  package      * version    date       lib source                          
##  askpass        1.1        2019-01-13 [1] CRAN (R 4.0.0)                  
##  assertthat     0.2.1      2019-03-21 [1] CRAN (R 4.0.0)                  
##  backports      1.1.8      2020-06-17 [1] CRAN (R 4.0.0)                  
##  bitops         1.0-6      2013-08-17 [1] CRAN (R 4.0.0)                  
##  callr          3.4.3      2020-03-28 [1] CRAN (R 4.0.0)                  
##  clamour      * 0.1.0      2020-09-01 [1] Github (lazappi/clamour@c8ea1c7)
##  cli            2.0.2      2020-02-28 [1] CRAN (R 4.0.0)                  
##  colorspace     1.4-1      2019-03-18 [1] CRAN (R 4.0.0)                  
##  crayon         1.3.4      2017-09-16 [1] CRAN (R 4.0.0)                  
##  curl           4.3        2019-12-02 [1] CRAN (R 4.0.0)                  
##  digest         0.6.25     2020-02-23 [1] CRAN (R 4.0.0)                  
##  dplyr        * 1.0.1      2020-07-31 [1] CRAN (R 4.0.2)                  
##  ellipsis       0.3.1      2020-05-15 [1] CRAN (R 4.0.0)                  
##  emo          * 0.0.0.9000 2020-08-17 [1] Github (hadley/emo@3f03b11)     
##  evaluate       0.14       2019-05-28 [1] CRAN (R 4.0.0)                  
##  fansi          0.4.1      2020-01-08 [1] CRAN (R 4.0.0)                  
##  farver         2.0.3      2020-01-16 [1] CRAN (R 4.0.0)                  
##  forcats      * 0.5.0      2020-03-01 [1] CRAN (R 4.0.0)                  
##  fs           * 1.5.0      2020-07-31 [1] CRAN (R 4.0.2)                  
##  generics       0.0.2      2018-11-29 [1] CRAN (R 4.0.0)                  
##  ggforce        0.3.2      2020-06-23 [1] CRAN (R 4.0.2)                  
##  ggplot2      * 3.3.2      2020-06-19 [1] CRAN (R 4.0.2)                  
##  ggraph       * 2.0.3      2020-05-20 [1] CRAN (R 4.0.0)                  
##  ggrepel      * 0.8.2      2020-03-08 [1] CRAN (R 4.0.0)                  
##  ggtext       * 0.1.0      2020-06-04 [1] CRAN (R 4.0.2)                  
##  glue           1.4.1      2020-05-13 [1] CRAN (R 4.0.0)                  
##  graphlayouts   0.7.0      2020-04-25 [1] CRAN (R 4.0.0)                  
##  gridExtra      2.3        2017-09-09 [1] CRAN (R 4.0.0)                  
##  gridtext       0.1.1      2020-02-24 [1] CRAN (R 4.0.2)                  
##  gtable         0.3.0      2019-03-25 [1] CRAN (R 4.0.0)                  
##  here         * 0.1        2017-05-28 [1] CRAN (R 4.0.0)                  
##  highr          0.8        2019-03-20 [1] CRAN (R 4.0.0)                  
##  htmltools      0.5.0      2020-06-16 [1] CRAN (R 4.0.0)                  
##  httr           1.4.2      2020-07-20 [1] CRAN (R 4.0.2)                  
##  igraph       * 1.2.5      2020-03-19 [1] CRAN (R 4.0.0)                  
##  janeaustenr    0.1.5      2017-06-10 [1] CRAN (R 4.0.0)                  
##  jsonlite       1.7.0      2020-06-25 [1] CRAN (R 4.0.0)                  
##  kableExtra   * 1.2.1      2020-08-27 [1] CRAN (R 4.0.2)                  
##  knitr        * 1.29       2020-06-23 [1] CRAN (R 4.0.0)                  
##  labeling       0.3        2014-08-23 [1] CRAN (R 4.0.0)                  
##  lattice        0.20-41    2020-04-02 [1] CRAN (R 4.0.0)                  
##  lifecycle      0.2.0      2020-03-06 [1] CRAN (R 4.0.0)                  
##  lubridate    * 1.7.9      2020-06-08 [1] CRAN (R 4.0.0)                  
##  magick       * 2.4.0      2020-06-23 [1] CRAN (R 4.0.0)                  
##  magrittr       1.5        2014-11-22 [1] CRAN (R 4.0.0)                  
##  markdown       1.1        2019-08-07 [1] CRAN (R 4.0.0)                  
##  MASS           7.3-51.6   2020-04-26 [1] CRAN (R 4.0.0)                  
##  Matrix         1.2-18     2019-11-27 [1] CRAN (R 4.0.0)                  
##  modeltools     0.2-23     2020-03-05 [1] CRAN (R 4.0.0)                  
##  munsell        0.5.0      2018-06-12 [1] CRAN (R 4.0.0)                  
##  NLP            0.2-0      2018-10-18 [1] CRAN (R 4.0.0)                  
##  openssl        1.4.2      2020-06-27 [1] CRAN (R 4.0.0)                  
##  pillar         1.4.6      2020-07-10 [1] CRAN (R 4.0.2)                  
##  pkgconfig      2.0.3      2019-09-22 [1] CRAN (R 4.0.0)                  
##  plyr           1.8.6      2020-03-03 [1] CRAN (R 4.0.0)                  
##  png            0.1-7      2013-12-03 [1] CRAN (R 4.0.0)                  
##  polyclip       1.10-0     2019-03-14 [1] CRAN (R 4.0.0)                  
##  processx       3.4.3      2020-07-05 [1] CRAN (R 4.0.2)                  
##  ps             1.3.3      2020-05-08 [1] CRAN (R 4.0.0)                  
##  purrr        * 0.3.4      2020-04-17 [1] CRAN (R 4.0.0)                  
##  R6             2.4.1      2019-11-12 [1] CRAN (R 4.0.0)                  
##  RColorBrewer * 1.1-2      2014-12-07 [1] CRAN (R 4.0.0)                  
##  Rcpp           1.0.5      2020-07-06 [1] CRAN (R 4.0.0)                  
##  RCurl          1.98-1.2   2020-04-18 [1] CRAN (R 4.0.0)                  
##  reshape2       1.4.4      2020-04-09 [1] CRAN (R 4.0.0)                  
##  rlang          0.4.7      2020-07-09 [1] CRAN (R 4.0.2)                  
##  rmarkdown      2.3        2020-06-18 [1] CRAN (R 4.0.0)                  
##  rprojroot      1.3-2      2018-01-03 [1] CRAN (R 4.0.0)                  
##  rstudioapi     0.11       2020-02-07 [1] CRAN (R 4.0.0)                  
##  rtweet       * 0.7.0      2020-01-08 [1] CRAN (R 4.0.0)                  
##  rvest        * 0.3.6      2020-07-25 [1] CRAN (R 4.0.2)                  
##  scales         1.1.1      2020-05-11 [1] CRAN (R 4.0.0)                  
##  selectr        0.4-2      2019-11-20 [1] CRAN (R 4.0.0)                  
##  sessioninfo    1.1.1      2018-11-05 [1] CRAN (R 4.0.0)                  
##  slam           0.1-47     2019-12-21 [1] CRAN (R 4.0.0)                  
##  SnowballC      0.7.0      2020-04-01 [1] CRAN (R 4.0.0)                  
##  stringi        1.4.6      2020-02-17 [1] CRAN (R 4.0.0)                  
##  stringr      * 1.4.0      2019-02-10 [1] CRAN (R 4.0.0)                  
##  tibble         3.0.3      2020-07-10 [1] CRAN (R 4.0.2)                  
##  tidygraph      1.2.0      2020-05-12 [1] CRAN (R 4.0.0)                  
##  tidyr        * 1.1.1      2020-07-31 [1] CRAN (R 4.0.2)                  
##  tidyselect     1.1.0      2020-05-11 [1] CRAN (R 4.0.0)                  
##  tidytext     * 0.2.5      2020-07-11 [1] CRAN (R 4.0.2)                  
##  tm             0.7-7      2019-12-12 [1] CRAN (R 4.0.0)                  
##  tokenizers     0.2.1      2018-03-29 [1] CRAN (R 4.0.0)                  
##  topicmodels  * 0.2-11     2020-04-19 [1] CRAN (R 4.0.0)                  
##  tweenr         1.0.1      2018-12-14 [1] CRAN (R 4.0.0)                  
##  usethis        1.6.1      2020-04-29 [1] CRAN (R 4.0.0)                  
##  utf8           1.1.4      2018-05-24 [1] CRAN (R 4.0.0)                  
##  vctrs          0.3.2      2020-07-15 [1] CRAN (R 4.0.2)                  
##  viridis      * 0.5.1      2018-03-29 [1] CRAN (R 4.0.0)                  
##  viridisLite  * 0.3.0      2018-02-01 [1] CRAN (R 4.0.0)                  
##  webshot      * 0.5.2      2019-11-22 [1] CRAN (R 4.0.0)                  
##  withr          2.2.0      2020-04-20 [1] CRAN (R 4.0.0)                  
##  wordcloud    * 2.6        2018-08-24 [1] CRAN (R 4.0.0)                  
##  xfun           0.16       2020-07-24 [1] CRAN (R 4.0.2)                  
##  xml2         * 1.3.2      2020-04-23 [1] CRAN (R 4.0.0)                  
##  yaml           2.2.1      2020-02-01 [1] CRAN (R 4.0.0)                  
## 
## [1] /Library/Frameworks/R.framework/Versions/4.0/Resources/library</code></pre>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
