<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>GI2019</title>

<script src="site_libs/header-attrs-2.3/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lazappi's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/lazappi/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">GI2019</h1>
<h3 class="subtitle">Genome Informatics conference 2019</h3>
<h4 class="date">Last built: 2020-09-01 18:16:49</h4>

</div>


<p>
<button class="btn btn-primary" data-toggle="collapse" data-target="#Parameters">
Parameters
</button>
</p>
<div id="Parameters" class="collapse">
<table>
<thead>
<tr class="header">
<th align="left">Parameter</th>
<th align="left">Value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">hashtag</td>
<td align="left">#gi2019</td>
</tr>
<tr class="even">
<td align="left">start_day</td>
<td align="left">2019-11-06</td>
</tr>
<tr class="odd">
<td align="left">end_day</td>
<td align="left">2019-11-09</td>
</tr>
<tr class="even">
<td align="left">timezone</td>
<td align="left">America/New_York</td>
</tr>
<tr class="odd">
<td align="left">theme</td>
<td align="left">theme_light</td>
</tr>
<tr class="even">
<td align="left">accent</td>
<td align="left">#005190</td>
</tr>
<tr class="odd">
<td align="left">accent2</td>
<td align="left">#7FA8C7</td>
</tr>
<tr class="even">
<td align="left">kcore</td>
<td align="left">5</td>
</tr>
<tr class="odd">
<td align="left">topics_k</td>
<td align="left">9</td>
</tr>
<tr class="even">
<td align="left">bigram_filter</td>
<td align="left">5</td>
</tr>
<tr class="odd">
<td align="left">fixed</td>
<td align="left">TRUE</td>
</tr>
<tr class="even">
<td align="left">seed</td>
<td align="left">1</td>
</tr>
</tbody>
</table>
</div>
<div id="introduction" class="section level1 unnumbered" number="">
<h1 class="unnumbered" number="">Introduction</h1>
<p>An analysis of tweets from the #gi2019 hashtag for the [Genome Informatics conference][GI2019], 6-9 November 2019, at Cold Spring Harbor Laboratories, New York, USA.</p>
<p>A total of 3820 tweets from 1053 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1" number="1">
<h1 number="1"><span class="header-section-number">1</span> Timeline</h1>
<div id="tweets-by-day" class="section level2" number="1.1">
<h2 number="1.1"><span class="header-section-number">1.1</span> Tweets by day</h2>
<p><img src="figures/gi2019/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2" number="1.2">
<h2 number="1.2"><span class="header-section-number">1.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2019-11-06 - 2019-11-09 in the America/New_York timezone.</p>
<p><img src="figures/gi2019/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1" number="2">
<h1 number="2"><span class="header-section-number">2</span> Users</h1>
<div id="top-tweeters" class="section level2 tabset tabset-pills" number="2.1">
<h2 class="tabset tabset-pills" number="2.1"><span class="header-section-number">2.1</span> Top tweeters</h2>
<div id="overall" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Overall</h3>
<p><img src="figures/gi2019/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="original" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Original</h3>
<p><img src="figures/gi2019/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweets" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Retweets</h3>
<p><img src="figures/gi2019/tweets-top-users-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweet-proportion" class="section level2" number="2.2">
<h2 number="2.2"><span class="header-section-number">2.2</span> Retweet proportion</h2>
<p><img src="figures/gi2019/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-timeline" class="section level2" number="2.3">
<h2 number="2.3"><span class="header-section-number">2.3</span> Top tweeters timeline</h2>
<p><img src="figures/gi2019/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2 tabset tabset-pills" number="2.4">
<h2 class="tabset tabset-pills" number="2.4"><span class="header-section-number">2.4</span> Top tweeters by day</h2>
<div id="overall-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Overall</h3>
<div id="day-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/gi2019/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/gi2019/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 3</h4>
<p><img src="figures/gi2019/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 4</h4>
<p><img src="figures/gi2019/tweets-top-users-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Original</h3>
<div id="day-1-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/gi2019/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/gi2019/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 3</h4>
<p><img src="figures/gi2019/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 4</h4>
<p><img src="figures/gi2019/tweets-top-users-orig-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Retweets</h3>
<div id="day-1-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/gi2019/tweets-top-users-retweets-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/gi2019/tweets-top-users-retweets-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 3</h4>
<p><img src="figures/gi2019/tweets-top-users-retweets-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 4</h4>
<p><img src="figures/gi2019/tweets-top-users-retweets-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
</div>
<div id="sources" class="section level1 tabset tabset-pills" number="3">
<h1 class="tabset tabset-pills" number="3"><span class="header-section-number">3</span> Sources</h1>
<div id="users-1" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Users</h2>
<p><img src="figures/gi2019/tweets-top-sources-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Tweets</h2>
<p><img src="figures/gi2019/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="networks" class="section level1" number="4">
<h1 number="4"><span class="header-section-number">4</span> Networks</h1>
<div id="replies" class="section level2" number="4.1">
<h2 number="4.1"><span class="header-section-number">4.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by PageRank.</p>
<p><img src="figures/gi2019/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2" number="4.2">
<h2 number="4.2"><span class="header-section-number">4.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Filtered for a k-core of 5. Node colour and size adjusted according to PageRank score.</p>
<p><img src="figures/gi2019/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="tweet-types" class="section level1" number="5">
<h1 number="5"><span class="header-section-number">5</span> Tweet types</h1>
<div id="retweets-2" class="section level2 tabset tabset-pills" number="5.1">
<h2 class="tabset tabset-pills" number="5.1"><span class="header-section-number">5.1</span> Retweets</h2>
<div id="proportion" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/gi2019/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/gi2019/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
I highly recommend “How to read (single-cell RNA-seq) PCA plots” (by <span class="citation">@vallens</span>): <a href="https://t.co/Co3A47SZTR" class="uri">https://t.co/Co3A47SZTR</a> #gi2019 He notes that “A particular danger [in interpreting T-shapes] is that it is tempting to interpret this as a bifurcation in the data.” (it almost never is). #gi2019
</td>
<td style="text-align:right;">
256
</td>
</tr>
<tr>
<td style="text-align:left;">
lh3lh3
</td>
<td style="text-align:left;">
I am looking for one or two postdocs working on bioinformatics and computational biology in general. Find me at #gi2019 if you are around. <a href="https://t.co/cZpnhzMAtr" class="uri">https://t.co/cZpnhzMAtr</a>
</td>
<td style="text-align:right;">
204
</td>
</tr>
<tr>
<td style="text-align:left;">
aphillippy
</td>
<td style="text-align:left;">
Heng is amazing. Go work for him. If he turns you down, come talk to me. I am also looking for students and postdocs at #gi2019 😆 <a href="https://t.co/uuuMq6gXLJ" class="uri">https://t.co/uuuMq6gXLJ</a>
</td>
<td style="text-align:right;">
74
</td>
</tr>
<tr>
<td style="text-align:left;">
ACSCevents
</td>
<td style="text-align:left;">
Save the date - Genome Informatics 2020 - 14-17 Sept at the genome campus #GI2020 #gi2019 <a href="https://t.co/EfCc74qrCR" class="uri">https://t.co/EfCc74qrCR</a>
</td>
<td style="text-align:right;">
62
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
A reminder: “cluster sizes in a t-SNE plot mean nothing” <a href="https://t.co/GUe2BTFLEO" class="uri">https://t.co/GUe2BTFLEO</a> #gi2019
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
I’m presenting STRling, our new method to detect novel (and reference) STR expansions from short-read data. <a href="https://t.co/hPyL7WBhkI" class="uri">https://t.co/hPyL7WBhkI</a> #gi2019 Big thanks to <span class="citation">@brent_p</span> who has done a huge amount of work on this algorithm and is an all around fantastic guy to work with.
</td>
<td style="text-align:right;">
42
</td>
</tr>
<tr>
<td style="text-align:left;">
BenLangmead
</td>
<td style="text-align:left;">
<p>Slides from my #gi2019 talk this morning, about work primarily by <span class="citation">@dnb_hopkins</span>: <a href="https://t.co/lZYnBK8dhm" class="uri">https://t.co/lZYnBK8dhm</a></p>
<p>Software: <a href="https://t.co/JALMMmfSca" class="uri">https://t.co/JALMMmfSca</a></p>
Library: <a href="https://t.co/8Yifw0NUsz" class="uri">https://t.co/8Yifw0NUsz</a>
</td>
<td style="text-align:right;">
28
</td>
</tr>
<tr>
<td style="text-align:left;">
deannachurch
</td>
<td style="text-align:left;">
Also- I’m looking for someone to lead my Informatics/analysis group- and single cell experience is a must! Help build the next generation of genomics tools using genome engineering! <a href="https://t.co/v8vzvssQUM" class="uri">https://t.co/v8vzvssQUM</a> #gi2019
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
infoecho
</td>
<td style="text-align:left;">
Real bioinformatics “file parsing”!!#gi2019 <a href="https://t.co/4vll3qI61u" class="uri">https://t.co/4vll3qI61u</a>
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
We’ve posted the kallisto | bustools poster presented by <span class="citation">@sinabooeshaghi</span> at #gi2019 on <span class="citation">@F1000Research</span> <a href="https://t.co/EVxAXZvGAC" class="uri">https://t.co/EVxAXZvGAC</a> <a href="https://t.co/inzS4JAeUZ" class="uri">https://t.co/inzS4JAeUZ</a>
</td>
<td style="text-align:right;">
22
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-retweeted" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most retweeted</h3>
<p><img src="figures/gi2019/most-retweeted-1.png" width="258" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="likes" class="section level2 tabset tabset-pills" number="5.2">
<h2 class="tabset tabset-pills" number="5.2"><span class="header-section-number">5.2</span> Likes</h2>
<div id="proportion-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/gi2019/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/gi2019/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
I highly recommend “How to read (single-cell RNA-seq) PCA plots” (by <span class="citation">@vallens</span>): <a href="https://t.co/Co3A47SZTR" class="uri">https://t.co/Co3A47SZTR</a> #gi2019 He notes that “A particular danger [in interpreting T-shapes] is that it is tempting to interpret this as a bifurcation in the data.” (it almost never is). #gi2019
</td>
<td style="text-align:right;">
757
</td>
</tr>
<tr>
<td style="text-align:left;">
lh3lh3
</td>
<td style="text-align:left;">
I am looking for one or two postdocs working on bioinformatics and computational biology in general. Find me at #gi2019 if you are around. <a href="https://t.co/cZpnhzMAtr" class="uri">https://t.co/cZpnhzMAtr</a>
</td>
<td style="text-align:right;">
230
</td>
</tr>
<tr>
<td style="text-align:left;">
samstudio8
</td>
<td style="text-align:left;">
Made it to #gi2019 at <span class="citation">@CSHL</span>. My metagenomic assembly graph artwork is the book cover! <a href="https://t.co/T2X6EPqweC" class="uri">https://t.co/T2X6EPqweC</a>
</td>
<td style="text-align:right;">
187
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
A reminder: “cluster sizes in a t-SNE plot mean nothing” <a href="https://t.co/GUe2BTFLEO" class="uri">https://t.co/GUe2BTFLEO</a> #gi2019
</td>
<td style="text-align:right;">
166
</td>
</tr>
<tr>
<td style="text-align:left;">
aphillippy
</td>
<td style="text-align:left;">
Heng is amazing. Go work for him. If he turns you down, come talk to me. I am also looking for students and postdocs at #gi2019 😆 <a href="https://t.co/uuuMq6gXLJ" class="uri">https://t.co/uuuMq6gXLJ</a>
</td>
<td style="text-align:right;">
165
</td>
</tr>
<tr>
<td style="text-align:left;">
ACSCevents
</td>
<td style="text-align:left;">
Save the date - Genome Informatics 2020 - 14-17 Sept at the genome campus #GI2020 #gi2019 <a href="https://t.co/EfCc74qrCR" class="uri">https://t.co/EfCc74qrCR</a>
</td>
<td style="text-align:right;">
136
</td>
</tr>
<tr>
<td style="text-align:left;">
sexchrlab
</td>
<td style="text-align:left;">
<p>I can’t begin to tell you how much I love the openness of Genome Informatics. I know science has a long way to go, but the culture of sharing data and code, and emphasizing validation; it warms my heart.</p>
#GI2019
</td>
<td style="text-align:right;">
131
</td>
</tr>
<tr>
<td style="text-align:left;">
infoecho
</td>
<td style="text-align:left;">
A Manhattan plot for #GI2019 <a href="https://t.co/Jpamfgex8e" class="uri">https://t.co/Jpamfgex8e</a>
</td>
<td style="text-align:right;">
108
</td>
</tr>
<tr>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
I’m presenting STRling, our new method to detect novel (and reference) STR expansions from short-read data. <a href="https://t.co/hPyL7WBhkI" class="uri">https://t.co/hPyL7WBhkI</a> #gi2019 Big thanks to <span class="citation">@brent_p</span> who has done a huge amount of work on this algorithm and is an all around fantastic guy to work with.
</td>
<td style="text-align:right;">
103
</td>
</tr>
<tr>
<td style="text-align:left;">
chrisamiller
</td>
<td style="text-align:left;">
<p>SN: Not every bioinformatician gets to go to the algorithm ball. Some of them have to stay home and write pipelines</p>
Preach, brother 😆 #gi2019
</td>
<td style="text-align:right;">
98
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-likes" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most likes</h3>
<p><img src="figures/gi2019/most-favorited-1.png" width="258" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="quotes" class="section level2 tabset tabset-pills" number="5.3">
<h2 class="tabset tabset-pills" number="5.3"><span class="header-section-number">5.3</span> Quotes</h2>
<div id="proportion-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/gi2019/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/gi2019/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
We already have two new posters on our <span class="citation">@F1000Research</span> Channel for this year’s conference #Gi2019. Check them out at your leisure <a href="https://t.co/Qi3BJYQCs5" class="uri">https://t.co/Qi3BJYQCs5</a>
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
We already have 5 posters up on the website for you to look at before the poster session #GI2019 <a href="https://t.co/xrktvzLo6G" class="uri">https://t.co/xrktvzLo6G</a> <a href="https://t.co/qSNoUoImSe" class="uri">https://t.co/qSNoUoImSe</a>
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
dsurujon
</td>
<td style="text-align:left;">
What a great idea! Here are my slides for my #gi2019 talk on Friday on how transcriptomic chaos can predict bacterial fitness <a href="https://t.co/zXVnqfAcfx" class="uri">https://t.co/zXVnqfAcfx</a> <a href="https://t.co/4DBC2pID7I" class="uri">https://t.co/4DBC2pID7I</a>
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
ascendox
</td>
<td style="text-align:left;">
RT mike_schatz "RT AliciaOshlack: We already have two new posters on our F1000Research Channel for this year’s conference #Gi2019. Check them out at your leisure <a href="https://t.co/pz1UEPRaQ3%22" class="uri">https://t.co/pz1UEPRaQ3"</a>;
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
AdamJOrr
</td>
<td style="text-align:left;">
Guest tweeter in <span class="citation">@lh3lh3</span> ’s #gi2019 talk <a href="https://t.co/oLWLyS6ucx" class="uri">https://t.co/oLWLyS6ucx</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
nephantes
</td>
<td style="text-align:left;">
Heng Li <span class="citation">@lh3lh3</span>: nextflow pipeline used for liftover hg38 -&amp;gt; hg19 ☑️ sucessfully. :) <span class="citation">@nextflowio</span> #gi2019 <a href="https://t.co/zdDOrcO8zo" class="uri">https://t.co/zdDOrcO8zo</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Sad that <span class="citation">@sexchrlab</span> wasn’t able to be here to do the opening of #gi2019 with us because of stupid flight cancellations <a href="https://t.co/DbTTm0vPsr" class="uri">https://t.co/DbTTm0vPsr</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
I actually really like this photo. Who would have thought that <span class="citation">@pathogenomenick</span> and I only met for the first time 10 mintues before that #gi2019 <a href="https://t.co/DbTTm0vPsr" class="uri">https://t.co/DbTTm0vPsr</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
aphillippy
</td>
<td style="text-align:left;">
Heng is amazing. Go work for him. If he turns you down, come talk to me. I am also looking for students and postdocs at #gi2019 😆 <a href="https://t.co/uuuMq6gXLJ" class="uri">https://t.co/uuuMq6gXLJ</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
michelebusby
</td>
<td style="text-align:left;">
#gi2019 PhDs: Please don’t take yourself out the running for this because you’ve been using Heng’s tools for the last 5 years and are intimidated! I always learn a lot when I talk to Heng (as does the whole community) and this is a great opportunity. Take your shot. <a href="https://t.co/5grj3gBP1j" class="uri">https://t.co/5grj3gBP1j</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-quoted" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most quoted</h3>
<p><img src="figures/gi2019/most-quoted-1.png" width="258" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="media" class="section level1 tabset tabset-pills" number="6">
<h1 class="tabset tabset-pills" number="6"><span class="header-section-number">6</span> Media</h1>
<div id="proportion-3" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Proportion</h2>
<p><img src="figures/gi2019/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-3" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Top 10</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
samstudio8
</td>
<td style="text-align:left;">
Made it to #gi2019 at <span class="citation">@CSHL</span>. My metagenomic assembly graph artwork is the book cover! <a href="https://t.co/T2X6EPqweC" class="uri">https://t.co/T2X6EPqweC</a>
</td>
<td style="text-align:right;">
187
</td>
</tr>
<tr>
<td style="text-align:left;">
ACSCevents
</td>
<td style="text-align:left;">
Save the date - Genome Informatics 2020 - 14-17 Sept at the genome campus #GI2020 #gi2019 <a href="https://t.co/EfCc74qrCR" class="uri">https://t.co/EfCc74qrCR</a>
</td>
<td style="text-align:right;">
136
</td>
</tr>
<tr>
<td style="text-align:left;">
infoecho
</td>
<td style="text-align:left;">
A Manhattan plot for #GI2019 <a href="https://t.co/Jpamfgex8e" class="uri">https://t.co/Jpamfgex8e</a>
</td>
<td style="text-align:right;">
108
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
We’ve posted the kallisto | bustools poster presented by <span class="citation">@sinabooeshaghi</span> at #gi2019 on <span class="citation">@F1000Research</span> <a href="https://t.co/EVxAXZvGAC" class="uri">https://t.co/EVxAXZvGAC</a> <a href="https://t.co/inzS4JAeUZ" class="uri">https://t.co/inzS4JAeUZ</a>
</td>
<td style="text-align:right;">
89
</td>
</tr>
<tr>
<td style="text-align:left;">
infoecho
</td>
<td style="text-align:left;">
Real bioinformatics “file parsing”!!#gi2019 <a href="https://t.co/4vll3qI61u" class="uri">https://t.co/4vll3qI61u</a>
</td>
<td style="text-align:right;">
74
</td>
</tr>
<tr>
<td style="text-align:left;">
cshlmeetings
</td>
<td style="text-align:left;">
As #gi2019 wraps up, we’d like to thank meeting sponsor <span class="citation">@genome_gov</span>; co-orgs <span class="citation">@pathogenomenick</span>, <span class="citation">@AliciaOshlack</span> and <span class="citation">@sexchrlab</span>; and those who live-tweeted during the entire meeting (especially during the lightning talk session)! See everyone at #gi2021! <a href="https://t.co/awkC8GZHTW" class="uri">https://t.co/awkC8GZHTW</a> <a href="https://t.co/RAEBm3zce3" class="uri">https://t.co/RAEBm3zce3</a>
</td>
<td style="text-align:right;">
70
</td>
</tr>
<tr>
<td style="text-align:left;">
aphillippy
</td>
<td style="text-align:left;">
<span class="citation">@AliciaOshlack</span> and <span class="citation">@pathogenomenick</span> kicking off #gi2019. So many legendary bioinformagicians in attendance. Love this meeting! <a href="https://t.co/BIMjUNC1E2" class="uri">https://t.co/BIMjUNC1E2</a>
</td>
<td style="text-align:right;">
65
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>yoson</strong>
</td>
<td style="text-align:left;">
Guess which session this was (poll below) #gi2019 <a href="https://t.co/UBQJZdFLyn" class="uri">https://t.co/UBQJZdFLyn</a>
</td>
<td style="text-align:right;">
57
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Chalkboard update for Genome Informatics meeting 2020 and 2021. Put it in your diaries. #gi2019 <a href="https://t.co/3fwbfGaVsx" class="uri">https://t.co/3fwbfGaVsx</a>
</td>
<td style="text-align:right;">
47
</td>
</tr>
<tr>
<td style="text-align:left;">
iddux
</td>
<td style="text-align:left;">
<span class="citation">@samstudio8</span> at #gi2019 on the difference between CPU and GPU tasks. “CPU is like two postdocs, they can do complex work but are easily distracted and need to do other things too. GPU is like an army of babies that can do just one simple thing, but can do it quickly and forever” <a href="https://t.co/rjQMzm7Tv3" class="uri">https://t.co/rjQMzm7Tv3</a>
</td>
<td style="text-align:right;">
45
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="section" class="section level1 unnumbered toc-ignore" number="">
<h1 class="unnumbered toc-ignore" number=""></h1>
<div id="most-liked-image" class="section level2" number="6.1">
<h2 number="6.1"><span class="header-section-number">6.1</span> Most liked image</h2>
<p><img src="http://pbs.twimg.com/media/EIuW-SJW4AExEUJ.jpg" /></p>
</div>
</div>
<div id="tweet-text" class="section level1" number="7">
<h1 number="7"><span class="header-section-number">7</span> Tweet text</h1>
<div id="word-cloud" class="section level2" number="7.1">
<h2 number="7.1"><span class="header-section-number">7.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/gi2019/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="hashtags" class="section level2" number="7.2">
<h2 number="7.2"><span class="header-section-number">7.2</span> Hashtags</h2>
<p>Other hashtags used 5 or more times.</p>
<p><img src="figures/gi2019/hashtags-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="emojis" class="section level2" number="7.3">
<h2 number="7.3"><span class="header-section-number">7.3</span> Emojis</h2>
<p><img src="figures/gi2019/emojis-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2" number="7.4">
<h2 number="7.4"><span class="header-section-number">7.4</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 5 times.</p>
<p><img src="figures/gi2019/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2" number="7.5">
<h2 number="7.5"><span class="header-section-number">7.5</span> Topic modelling</h2>
<p>Top 10 words associated with 9 topics identified by LDA.</p>
<p><img src="figures/gi2019/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset tabset-fade" number="7.5.1">
<h3 class="tabset tabset-fade" number="7.5.1"><span class="header-section-number">7.5.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 1</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Refasho
</td>
<td style="text-align:left;">
The fitting shoes #PixarSoul #gi2019 #WIZONELOVEIZONE #ThankfulThursday #BWFWarren #Obsession #ODMUnleashesGoons #JusticeForAsim #1026大切なユタの日 #10yearsofdanandphil #10월에_태어난_병아리_윈윈 #nyc #ny #344 #bloomfieldav #uppermontclair #montclair #newark #kearny #edison #byc <a href="https://t.co/efM3QsnsQH" class="uri">https://t.co/efM3QsnsQH</a>
</td>
<td style="text-align:right;">
0.9905997
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Bzikadze: centroFlye for centromere assembly - identify centromeric reads, classify as prefix/internal/suffix, find rare/unique k-mers –&amp;gt; represent reads as k-mer clouds –&amp;gt; build contigs –&amp;gt; polish. Rub: need to deal w errors #gi2019
</td>
<td style="text-align:right;">
0.9899113
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
STR: several methods for reference STR (all using puns including the letters STR). Canvas (ataxia) disease, reference genome has alu element and AAAAG_n motif, disease elements have AAGGG_n motif, reference based methods fail to detect these #gi2019
</td>
<td style="text-align:right;">
0.9895278
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
SK: can get 4x enrichment of a fungal genome within a bacterial community. Human gene enrichment, 28 genes from breast cancer panel. Mapping to a 3.5M reference, eject reads after 3sec. 2.8x enrichment. Showing fresh results obtained on Monday #gi2019
</td>
<td style="text-align:right;">
0.9895278
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
SK: ONT real time, read-until can monitor the sequence, up to 512 channels at 450bp/sec. UNCALLED maps 10kbp/sec per thread, 75% of reads classified within 1 sec, [aside: ONT can eject reads from pores, saving on sequencing]. #gi2019
</td>
<td style="text-align:right;">
0.9886662
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
KR: multiple global genomic measures being considered; telomere length, genome complexity, coding mutational burden, mutational signatures describing factors that have impacted genome e.g. smoking/age #gi2019
</td>
<td style="text-align:right;">
0.9876502
</td>
</tr>
<tr>
<td style="text-align:left;">
pathogenomenick
</td>
<td style="text-align:left;">
Andrey Bzikadze talking about centromere assembly with centroFlye: bigging up <span class="citation">@khmiga</span> and <span class="citation">@aphillippy</span>’s burgeoning T2T Consortium data which has huge haul of ultra-long reads and complementary technologies for human haploid genome: <a href="https://t.co/bNU6QpvEfI" class="uri">https://t.co/bNU6QpvEfI</a> #gi2019
</td>
<td style="text-align:right;">
0.9876502
</td>
</tr>
<tr>
<td style="text-align:left;">
ascendox
</td>
<td style="text-align:left;">
RT mike_schatz “RT Magdoll: Aganezov: how to retrace cancer rearrangement history? formulate as minimal eulerian decomposition problem. to deal with non-unique solutions, form as consistent contig covering problem (&amp;lt;–this is new to me). #gi2019 prep… <a href="https://t.co/tKWOjW7ITd" class="uri">https://t.co/tKWOjW7ITd</a>”
</td>
<td style="text-align:right;">
0.9870707
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
AB: length of centromere of X is ~2.8Mbp, consists of repeats of 171bp repeats, in 12 units that are DXZ1. Centromeres are 3% of reference, play a critical role in chrom segregation, variations are linked to cancer and infertility #gi2019
</td>
<td style="text-align:right;">
0.9870707
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 <span class="citation">@AndreyBzikadze</span>: centroFlye-Assembling centromeres w/ long error-prone reads</p>
<p>Telomere-to-tel consortium: <a href="https://t.co/HdmPfXmTTk" class="uri">https://t.co/HdmPfXmTTk</a> <a href="https://t.co/Sg82pHMR3q" class="uri">https://t.co/Sg82pHMR3q</a></p>
centroFlye: ID unique k-mers in centromere, use for centro. assembly <a href="https://t.co/X6KKGbvJmv" class="uri">https://t.co/X6KKGbvJmv</a> <a href="https://t.co/0luMvnOzYm" class="uri">https://t.co/0luMvnOzYm</a>
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 2</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
HL: Incremental graph construction. Invariant property: linear path (original genome) and acyclic. Seq to graph alignment: Find seeds in segments, use linear chaining ignoring topology. Graph chaining, use graph topology to line up chains. Search best path with heuristics #gi2019
</td>
<td style="text-align:right;">
0.9905997
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Heng Li <span class="citation">@lh3lh3</span> The construct &amp;amp; utility of reference pan-genome graphs</p>
<p>Plan: start w/ GRCh38, incrementally add other genomes to construct graph, blacklist &amp;amp; decoy seqs for linear tools, updates preserve coordinate sys</p>
<a href="https://t.co/B6XVbWQX50" class="uri">https://t.co/B6XVbWQX50</a> <a href="https://t.co/NcTsAbyPyg" class="uri">https://t.co/NcTsAbyPyg</a>
</td>
<td style="text-align:right;">
0.9899113
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
HL: VCF, hard to represent small variants on long insertions wrt reference. GFA format, two components. S-line segments that are sequences, L-line: links between segments. Most popular assembly format. GFA coordinates are unstable, can split segment and change coordintes #gi2019
</td>
<td style="text-align:right;">
0.9899113
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Shilpa Garg: Efficient chr-scale haplotype-resolved assembly of human individuals</p>
<p>method leverages long accurate reads+long-range conformation data for individuals to generate chr-scale phased assembly w/in a day</p>
👀👇 <a href="https://t.co/4M17NhWId8" class="uri">https://t.co/4M17NhWId8</a> <a href="https://t.co/XhXlNj9TBl" class="uri">https://t.co/XhXlNj9TBl</a>
</td>
<td style="text-align:right;">
0.9895278
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 M Pertea, Efficient &amp;amp; robust transcriptome reconstruction from long-read RNA-seq alignments</p>
<p>Based on StringTie, short read transcriptome assembler: <a href="https://t.co/Kjg3ng0GPt" class="uri">https://t.co/Kjg3ng0GPt</a></p>
StringTie2, long read sln, overcome error rate and lower throughput 👀👇 <a href="https://t.co/00KzRLeRnH" class="uri">https://t.co/00KzRLeRnH</a>
</td>
<td style="text-align:right;">
0.9891140
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
Heng Li, <span class="citation">@lh3lh3</span>: Pangenomes, construction and utility. Styles of pangenome graphs: assembly style graph (large messy), linear graph (string of bubbles). Linearity no 2 bases on a haplotype can be mapped to the same graph position. VCF is linear #gi2019
</td>
<td style="text-align:right;">
0.9886662
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
HL: sad stories about Gnomad taking 6 years to move to Grch38 and someone running liftover to hg19 from Gnomad v3. New restricted format rGFA. Can incrementally add genomes to construct a graph. Updates preserve coordinate system #gi2019
</td>
<td style="text-align:right;">
0.9886662
</td>
</tr>
<tr>
<td style="text-align:left;">
kasukawa
</td>
<td style="text-align:left;">
<p>Utilization of consensus genome, in which minor alleles are replaced to major ones.</p>
<p>Interestingly, pan-human consensus genome is adequate to reduce error rates. Even if using consensus genomes specific to subpopulations, the error rates were not reduced.</p>
#gi2019
</td>
<td style="text-align:right;">
0.9881800
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
EE: Phased assembly, uses linked reads to phase, hifi reads to construct contigs. 81% of hifi reads can be assigned to a haplotype. Regions that were problematic for canu and peregrine assemblers, about 250, colocalized with segmental duplications. #gi2019
</td>
<td style="text-align:right;">
0.9876502
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
MK: discussing cow rumen metagenome 11.4 Gb, N50=11Kb. assembly 268 Mr, N50=230Kb. 44 contigs&amp;gt; 1Mb After assemble using tools found 649 bubbles, 96 superbubbles, 210 shared bubbles #gi2019
</td>
<td style="text-align:right;">
0.9876502
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-3" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 3</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
TG: Filtering FPs using dual strand filter, nanopolish has highest sensitivity at high coverage. Phased variants in TP53 region using WhatsHap. Phasing of normal vs tumor tissue revealed cancer specific snps and a copy number difference between alleles in tumor #gi2019
</td>
<td style="text-align:right;">
0.9895278
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
TG: High coverage of targeted regions means you can call SNPs better (except for homopolymer sites). Comp of tools, samtools, clair, medaka and nanopolish. 0.98 sensitivity for nanopolish and samtools at 200x cov (still a number of FPs), FPs often from a single strand #gi2019
</td>
<td style="text-align:right;">
0.9891140
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
Daniel Cameron on structural variation and CNV from cancer samples. Fully automated oncology reports, no possibility of manual intervention. Wrote 3 software for SV calling (GRIDSS), CNV calling (PURPLE) and interpretation (LINX) #gi2019
</td>
<td style="text-align:right;">
0.9881800
</td>
</tr>
<tr>
<td style="text-align:left;">
chrisamiller
</td>
<td style="text-align:left;">
<p>DC: How can we make good, clinic-ready structural variant calls from tumor patients?</p>
<ul>
<li><p>110x WGS tumor/normal.</p></li>
<li><p>Pipeline uses GRIDSS (breakend assembly) -&amp;gt; PURPLE (CN calling) -&amp;gt; LINX (annotation) #gi2019</p>
</td>
<td style="text-align:right;">
<p>0.9881800</p>
</td>
</tr>
<tr>
<td style="text-align:left;">
<p>YiXing77</p>
</td>
<td style="text-align:left;">
<p>My student Yang Pan <span class="citation">@ypnngaa</span> will present his work on PEGASAS, pathway guided analysis of alternative splicing, at the upcoming Penn #RNA club next Tuesday! He also has a poster about PEGASAS at CSHL #gi2019 this week. #IOTN #Moonshot <a href="https://t.co/jzIJSD6wWU" class="uri">https://t.co/jzIJSD6wWU</a></p>
</td>
<td style="text-align:right;">
<p>0.9876502</p>
</td>
</tr>
<tr>
<td style="text-align:left;">
<p>Repealist_</p>
</td>
<td style="text-align:left;">
<p>Commuting to <span class="citation">@CSHL</span> #gi2019 meet and stopped off at Sutphin Blvd waiting for a train</p></li>
</ul>
<p>Have already talked to three Bangla families and one Korean gal</p>
<p>And the morning fishmarkets are truly the view of dreams</p>
One for you <span class="citation">@MichaelsCoDub</span> <a href="https://t.co/glRsqxmruN" class="uri">https://t.co/glRsqxmruN</a>
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
DC: developed a set of tools to better call SVs. Wrote a new GRIDSS based tool for SVs, Purple specifically for CNV, and a tool called LINX for variant interpretation #gi2019
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
<tr>
<td style="text-align:left;">
byuhobbes
</td>
<td style="text-align:left;">
<span class="citation">@BenLangmead</span> alternative to the bottom k approach (aka min hashing) is k-partition approach; break up k-mer space into k partitions, take the bottom element of each partition; similar characteristics to min hashing #gi2019
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Gangavarapu: West Nile virus shown to persist through the seasons and circulate across all regions in California. Unable to link it to surviving in birds (or mosquitos? someone correct me) in winter (no seasonality). #gi2019
</td>
<td style="text-align:right;">
0.9849523
</td>
</tr>
<tr>
<td style="text-align:left;">
byuhobbes
</td>
<td style="text-align:left;">
<span class="citation">@MedhatHelmy7</span> <span class="citation">@MedhatHelmy7</span> Benchmarked variant calling and phasing tools with GIAB data catalog; better SNV calling performance with 10x coverage from PacBio CCs vs 25x from PacBio CLR or 25x from ONT #gi2019
</td>
<td style="text-align:right;">
0.9849523
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 4</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 A Balsubramani; Multi-resolution, interactive, atlas-scale integration of single-cell assays &amp;amp; exper.</p>
<p>Unify datasets from diff experiments? Connect w/ anchors, impute/co-embedding, refine on subpops and repeat.</p>
Their approach=Musica [I don’t think released quite yet]
</td>
<td style="text-align:right;">
0.9891140
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
Now Akshay Balsubramani talking about integrating data from single-cell RNA-seq experiments (w/ <span class="citation">@anshulkundaje</span>). This is an important topic. Full disclosure: we’ve been partial to <span class="citation">@_romain_lopez_</span>’ et al.’s scVI in my lab but constant exciting new ideas in this area. #gi2019
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Balsubramani: unifying single cell RNA-seq experiments. steps: anchor (find cells of similar cell types) –&amp;gt; impute features –&amp;gt; refine co-embedding. repeat. #gi2019
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
byuhobbes
</td>
<td style="text-align:left;">
<span class="citation">@Magdoll</span> <span class="citation">@Magdoll</span> Suggests that single-cell platforms produce shorter transcripts than bulk full-length cDNA; using proximity to CAGE peaks as orthogonal evidence of 5’ completeness; polyadenylation signals for 3’ completeness #gi2019
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
I highly recommend “How to read (single-cell RNA-seq) PCA plots” (by <span class="citation">@vallens</span>): <a href="https://t.co/Co3A47SZTR" class="uri">https://t.co/Co3A47SZTR</a> #gi2019 He notes that “A particular danger [in interpreting T-shapes] is that it is tempting to interpret this as a bifurcation in the data.” (it almost never is). #gi2019
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
A special privilege of attending a <span class="citation">@CSHL</span> meeting is that one can learn about recent research via glimpses at unpublished data. But when it comes to methods talks at meetings such as #gi2019 I think there should be a public Github repo or equivalent. <a href="https://t.co/kYT0vLduvQ" class="uri">https://t.co/kYT0vLduvQ</a>
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
<tr>
<td style="text-align:left;">
sexchrlab
</td>
<td style="text-align:left;">
<p>In past years, we tried to add a place for abstract submitters to add a link to software repos/web interfaces; I think this would be great to bring back.</p>
<p>Genome Informatics slides and posters have and can be posted here (please share!): <a href="https://t.co/aWCJRpKmnD" class="uri">https://t.co/aWCJRpKmnD</a></p>
#gi2019
</td>
<td style="text-align:right;">
0.9849523
</td>
</tr>
<tr>
<td style="text-align:left;">
YiXing77
</td>
<td style="text-align:left;">
<span class="citation">@DNAwyman</span> of UCI presenting the ENCODE TALON software for long read RNA-seq data analysis <a href="https://t.co/7QDZvNwxcn" class="uri">https://t.co/7QDZvNwxcn</a>. Also nice to see a figure from our AJHG review <a href="https://t.co/QS52kMJaiB" class="uri">https://t.co/QS52kMJaiB</a> by <span class="citation">@RNAEddie</span> used in the Intro! #gi2019 <a href="https://t.co/MhKOMKNUxO" class="uri">https://t.co/MhKOMKNUxO</a>
</td>
<td style="text-align:right;">
0.9849523
</td>
</tr>
<tr>
<td style="text-align:left;">
cshlmeetings
</td>
<td style="text-align:left;">
Meet <span class="citation">@samstudio8</span> of <span class="citation">@unibirmingham</span>! He’s busy at #gi2019. He gave a talk, his work is featured on the cover of the meeting’s abstract book, and is meeting in person the Twitter avatars with whom he’s been messaging the past year or so! #cshlvisitor <a href="https://t.co/6HmQ9eKFCb" class="uri">https://t.co/6HmQ9eKFCb</a>
</td>
<td style="text-align:right;">
0.9840830
</td>
</tr>
<tr>
<td style="text-align:left;">
winhide
</td>
<td style="text-align:left;">
#gi2019 we are looking for a post doc who wants to work with an amazing team (I mean it!) Want to find the best drug that drives resilience against Alzheimer’s? Join us! - the NIH funded this work. Chat to Sarah Morgan at the meeting. <a href="https://t.co/0b4pyljOJl" class="uri">https://t.co/0b4pyljOJl</a>
</td>
<td style="text-align:right;">
0.9831071
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 5</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
SLH: Process is RNA-seq, run data (on Dragen on AWS) through FusionMap, JAFFA, STARfusion, FusionCatcher, MapSplice, and TopHat fusion with overlap analysis, hierarchical prioritization, knowledge based filtering, and then CAP-CLIA validation of clinically relevant vars #gi2019
</td>
<td style="text-align:right;">
0.9902676
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 S LaHaye: Utilization of ensemble approach for ID of driver fusions in peds cancer</p>
Ensemble approach reduces false +s 6 tools (FusionMap, JAFFA, STARfusion, FusionCatcher, MapSplice, TopHat Fusion) DRAGEN to speedup [seems to ID 98% in test] Seraseq as truth
</td>
<td style="text-align:right;">
0.9899113
</td>
</tr>
<tr>
<td style="text-align:left;">
chrisamiller
</td>
<td style="text-align:left;">
SL: Using a ensemble approach of 6 gene-fusion callers to identify fusions in ped cancer. Overlap (by at least 2 callers), prioritize based on evidence, and compare to a knowledge base. Final step is CAP/CLIA RT-PCR or sanger validation #gi2019
</td>
<td style="text-align:right;">
0.9895278
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Xi Chen, ‘Tissue-specific enhancer functional networks for associating distal regulatory regions to disease’</p>
<p>How interpret/predict fxn of enhancers/disease-associated enhancers? Bayesian inference: <a href="https://t.co/JjuT1Ar8pq" class="uri">https://t.co/JjuT1Ar8pq</a> (try it!)</p>
autism e.g.: <a href="https://t.co/TlWmLl7CXY" class="uri">https://t.co/TlWmLl7CXY</a>
</td>
<td style="text-align:right;">
0.9886662
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
<span class="citation">@sinabooeshaghi</span> <span class="citation">@GoogleColab</span> <span class="citation">@jspacker</span> <span class="citation">@jisaacmurray</span> <span class="citation">@JunhyongKim</span> <span class="citation">@sinabooeshaghi</span>: kallisto | bustools runs up to 100 times faster than Cell Ranger, 25 times faster than Alevin and 4 times faster than STARsolo <a href="https://t.co/GGveUbyaUu" class="uri">https://t.co/GGveUbyaUu</a> Many features and capabilities not available with other tools. See poster this afternoon at #gi2019.
</td>
<td style="text-align:right;">
0.9876502
</td>
</tr>
<tr>
<td style="text-align:left;">
chrisamiller
</td>
<td style="text-align:left;">
<p>LINE1 activity correlates with poor immunotherapy response.</p>
Suggests that LINE1 high tumors are cloaked in a way that’s sidesteps activation with std imm. therapy (if they weren’t cloaked, they’d already be gone, since high LINE1 triggers response in normal cells!) #gi2019
</td>
<td style="text-align:right;">
0.9870707
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
gloria integration of ALzheimers disease genetics and myeloid cell genomics to revel novel risk mechanisms HiC, epigenomics, GWAS hits to identify causal genes and loci and validated in microglia #gi2019
</td>
<td style="text-align:right;">
0.9870707
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
anoushka joglekar on enhanced interpretation of single-cell isoform RNA sequencing to reveal distinct programs of alternative splicing in the developing mouse brain correlated to RNA splicing machinery #gi2019
</td>
<td style="text-align:right;">
0.9870707
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
.@dsurujon : gene expression is more chaotic in low fitness bacteria, this means using entropy (measuring chaos in gene expression) can predict fitness. but need to model for co-expressed genes that break the independence assumption. #gi2019
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
Katherin Woolley-Allen asking what is the weakest level off transcriptional factor binding that can exert an effect; evolution has gone further towards lower and lower affinity binding sites acting in combination #gi2019
</td>
<td style="text-align:right;">
0.9849523
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 6</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
byuhobbes
</td>
<td style="text-align:left;">
<span class="citation">@s_ramach</span> <span class="citation">@s_ramach</span> Question: how are ROHs distributed in the genome? Parent-child data in 23andMe has revealed extremely long IBD segments in putatively normal individuals; reflection of uniparental disomy, where both haplotypes inherited from a single parent #gi2019
</td>
<td style="text-align:right;">
0.9876502
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Karine Le Roch, ‘Comparative 3D genome organization in Apicomplexan parasites’</p>
<p>Overall, work supports link between genome organization and gene expression in more virulent pathogens</p>
learn more from their recent pubs: <a href="https://t.co/0bG5uj8tvD" class="uri">https://t.co/0bG5uj8tvD</a> <a href="https://t.co/Fc3Ar3U8re" class="uri">https://t.co/Fc3Ar3U8re</a>
</td>
<td style="text-align:right;">
0.9876502
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
SA: using a combination of 30x 10X, 40x ONT, and 40x PacBio and aligning with Long ranger and NGMLR. calling with lumpy, manta, SvABA, GROCSV, NAIBR, LongRanger, sniffles, PBSV and combined using survivor methods #gi2019
</td>
<td style="text-align:right;">
0.9876502
</td>
</tr>
<tr>
<td style="text-align:left;">
GenomeBiology
</td>
<td style="text-align:left;">
SR: has been looking at runs of homozygosity (ROH) in 23andMe data. Parent-child data reveals extremely long shared segments eg child has whole chromosome with no sequence shared with father - called uniparental disomy #gi2019
</td>
<td style="text-align:right;">
0.9870707
</td>
</tr>
<tr>
<td style="text-align:left;">
CalvinH43357785
</td>
<td style="text-align:left;">
<p>Uncanny Attraction #astroworldfestival #AUSvPAK #Bigil #BREAKING #crush #ChangeForWonho #ChaseYoung #DeathStrading #DoctorSleepMovie #EricCiaramella #EpsteinSuicideCoverUp #EXOonearewe #Ethiopia #FlashbackFriday #FursuitFriday #gi2019 #GymJordan #G_I_DLE</p>
<a href="https://t.co/FZVvn45dCp" class="uri">https://t.co/FZVvn45dCp</a>
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
TwilightForce1
</td>
<td style="text-align:left;">
<p>sPaCe oVeRtAkEn</p>
<p>#astroworldfestival #AUSvPAK #Bigil #BREAKING #crush #ChangeForWonho #ChaseYoung #DeathStrading #DoctorSleepMovie #EricCiaramella #EpsteinSuicideCoverUp #EXOonearewe #Ethiopia #FlashbackFriday #FursuitFriday #gi2019 #GymJordan #G_I_DLE</p>
<a href="https://t.co/HAxVeeEsFk" class="uri">https://t.co/HAxVeeEsFk</a>
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Nakka: runs of homozygosity (ROH) are associated w divergence from Africa. Diff classes of ROH have diff mean/variance in population. How to study ROH distribution within a genome? <span class="citation">@23andMe</span> parent-child data! #gi2019
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
kasukawa
</td>
<td style="text-align:left;">
<p>quantifying isoform expression in scRNA-seq data with STARsolo-Quant #gi2019</p>
Quantifying each isoform based on the distribution function (by Maximum likelihood estimation) for distance to TTS and UMI barcoding.
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
Interesting mention of kernel Canonical Correlation Analysis (KCCA) at #gi2019 by <span class="citation">@KarineLeRoch1</span> for coupling genome organization data with gene expression data. Nonlinear CCA is complex and still not widely used in genomics. Related: <a href="https://t.co/ux3wgq7XpQ" class="uri">https://t.co/ux3wgq7XpQ</a>
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Nakka: mining 23&amp;amp;me parent-child data revealed cases of uniparental disomy (UPD) where both chromosomes of a parent is preserved with no copy from the other parent. This occurs due to non-disjunction during meiosis O__O #gi2019
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-7" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 7</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
KG: main goal of consortia is to provide real-time high definition reconstructions of the micro- (e.g., city/county) and macro-level (e.g., state/country) spread and evolution of WNV by sequencing from infected birds, mosquitoes, and patients #gi2019
</td>
<td style="text-align:right;">
0.9891140
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Dilthey: Applies MetMaps to PacBio HMP &amp;amp; ONT Zymo mock community achieves high read assignment on both. Caveat: reduced recall due to min RL 1kb; HMW DNA required. #gi2019 Paper here: <a href="https://t.co/KxePwF8jwM" class="uri">https://t.co/KxePwF8jwM</a>
</td>
<td style="text-align:right;">
0.9881800
</td>
</tr>
<tr>
<td style="text-align:left;">
byuhobbes
</td>
<td style="text-align:left;">
<span class="citation">@jkpritch</span> <span class="citation">@jkpritch</span> Recent metaanalysis of schizophrenia GWAS studies found 108 genome-wide significant loci; most hits are difficult to interpret, and only explain 10% of the observed heritability (“missing heritability” problem); enormous number of small effects #gi2019
</td>
<td style="text-align:right;">
0.9876502
</td>
</tr>
<tr>
<td style="text-align:left;">
FadelBerkadar
</td>
<td style="text-align:left;">
<p>A few words about novoSplice poster for the Genome Informatics meeting at CSHL #gi2019</p>
<ul>
<li>The poster explains the general workflow of novoSplice besides preliminary benchmarking results using both simulated and real reads. Algorithmic details of the splic…<a href="https://t.co/fxeHNScOal" class="uri">https://t.co/fxeHNScOal</a>
</td>
<td style="text-align:right;">
0.9870707
</td>
</tr>
<tr>
<td style="text-align:left;">
BBFCreativeEnt
</td>
<td style="text-align:left;">
It must suck to support a racist president. This quote will bite you in the ass when you are up for reelection….It must suck to loose BIG to a democrat…and have to work in the private sector!#ThursdayMotivation #redcup #gi2019 #WashingtonPost #NYTimes #art #washingtinTimes <a href="https://t.co/aK9vZlnawo" class="uri">https://t.co/aK9vZlnawo</a>
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
Mahmoud: PRICNESS: mapping (NGMLR/minimap2), SNV (Clair), SV (Sniffles), Phasing (WhatsHap, Princess-subtools), optionally can use parental SNPs and methylation (nano polish) . Also stats. #gi2019
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
ascendox
</td>
<td style="text-align:left;">
RT mike_schatz “RT StevenSalzberg1: SamKovaka talking about”read until" sequencing on nanopore sequencers. Still experimental, but very cool that the sequencer can recognize DNA species in real time, and stop sequencing a read by reversing the current #gi2019"
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
at #gi2019 RP is suggesting that transcriptome alignment (substrate for RSEM, eXpress, kallisto..) is not as good as genome alignment. A comparison was performed by <span class="citation">@alexlachmann</span> et al. in <a href="https://t.co/Bg5QhHA4EY" class="uri">https://t.co/Bg5QhHA4EY</a>. They conclude “the quality of the predictions is almost identical” <a href="https://t.co/M8XM0eKAer" class="uri">https://t.co/M8XM0eKAer</a>
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
David_McGaughey
</td>
<td style="text-align:left;">
Made it to #gi2019. First time staying at cshl Banbury campus. If you do scRNA stuffs make time to bother me about me 750k+, 14 study ocular integration project while you wait to talk to <span class="citation">@pmelsted</span> at poster session 1 (105). <a href="https://t.co/D87VAc39T9" class="uri">https://t.co/D87VAc39T9</a>
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
philip kleinert; CADD-SV tool to score the effect of SVs - whether they have strong effect on health by training to differentiate between benign and pathogenic. Use evolutionary motivated approach #gi2019
</td>
<td style="text-align:right;">
0.9849523
</td>
</tr>
</tbody>
</table></li>
</ul>
</div>
<div id="topic-8" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 8</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Naihui Zhou: Exploring 3D spatial dependency of gene expr using Markov random fields</p>
<p>hypothesis: expr is spatially dependent</p>
developed PHiMRF <a href="https://t.co/YQjn6CRXGO" class="uri">https://t.co/YQjn6CRXGO</a> prob model for RNA-seq data accounting for gene locals in 3D genome (spatial dependency for count data)
</td>
<td style="text-align:right;">
0.9914725
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Sam Kovaka: Rapidly mapping raw <span class="citation">@nanopore</span> signal w/ UNCALLED to enable real-time targeted seq</p>
<p>ReadUntil seq bg: <a href="https://t.co/L5obZVjusq" class="uri">https://t.co/L5obZVjusq</a></p>
UNCALLED: Maps raw nanopore signals from fast5 files to large DNA refs (preprint soon!) <a href="https://t.co/gyahq5GDBd" class="uri">https://t.co/gyahq5GDBd</a>
</td>
<td style="text-align:right;">
0.9895278
</td>
</tr>
<tr>
<td style="text-align:left;">
Repealist_
</td>
<td style="text-align:left;">
<p>Incidental window seat right near the front, space to get work done,</p>
<p>and it starts raining just as I’m leaving Dublin?</p>
<p>All the promising makings of a great journey to #gi2019 and #SUNY.</p>
Slán a chairde 🇮🇪 tá sí ag dul go Nua-Eabhrac 🇺🇸! <a href="https://t.co/GTcyz95iNn" class="uri">https://t.co/GTcyz95iNn</a>
</td>
<td style="text-align:right;">
0.9881800
</td>
</tr>
<tr>
<td style="text-align:left;">
pmelsted
</td>
<td style="text-align:left;">
BL: Hyperlog (HLL) can process array of bits using SIMD values. Uses a bit more space by using a different base 1.19, fills up 8-bits rather than 6-bits with base 2. HLL performs better with lopsided data (large ratio of sizes between two sets) #gi2019
</td>
<td style="text-align:right;">
0.9881800
</td>
</tr>
<tr>
<td style="text-align:left;">
lpachter
</td>
<td style="text-align:left;">
I agree we should not hold scientific meetings on weekends. I also think we should not hold scientific meetings on weekdays. Scientists with kids face enormous logistical challenges on weekdays (kid pickup/dropoff, school events, activities, etc.) #gi2019 <a href="https://t.co/ifuJZuV3ci" class="uri">https://t.co/ifuJZuV3ci</a>
</td>
<td style="text-align:right;">
0.9876502
</td>
</tr>
<tr>
<td style="text-align:left;">
chrisamiller
</td>
<td style="text-align:left;">
FN: answering why ancient LINEs so frequently show up in RNAseq. To make a long story short, seq similarity causes mismapping, and their tool TeXP can correct for this. Shows that basically all autonomous transcription is from recent families, as expected #gi2019
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<p>#gi2019 Ben Langmead: Genomic sketching w/ HyperLogLog</p>
<p>Sketching=how to sift&amp;amp;summarize huge datasets so can answer similarity ?s later</p>
<p>Dashing w/ HyperLogLog <a href="https://t.co/5MWtP6xfPD" class="uri">https://t.co/5MWtP6xfPD</a> <a href="https://t.co/byoPzN12XA" class="uri">https://t.co/byoPzN12XA</a> <a href="https://t.co/GSykouIW4c" class="uri">https://t.co/GSykouIW4c</a></p>
[review of interest: <a href="https://t.co/OO5AklzeAz" class="uri">https://t.co/OO5AklzeAz</a>]
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
<tr>
<td style="text-align:left;">
GrameneDatabase
</td>
<td style="text-align:left;">
If you missed <span class="citation">@ajo2995</span> Andrew Olson’s delightful talk on Crop PanGenome Sites at #gi2019, look for us at the <span class="citation">@cshlmeetings</span> Wine and Cheese to savor a TE Cabernet and avoid being haunted by the ghost of Barbara McClintock <a href="https://t.co/eM88kVKzXv" class="uri">https://t.co/eM88kVKzXv</a>
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
plot twist: Narechania, just like everyone else, is ditching base-alignments and going for k-mers. Alright, k-mer based ortholog matrix to infer HGT. But too many k-mers, so gotta compress them without info loss. #gi2019
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
<tr>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
Yesterday I got my official PhD passed email while I was on stage at #gi2019 talking about work I’ve done since submitting my PhD in June. That is considered a relatively fast turn-around for an Australian PhD assessment! Thanks for being an awesome PhD supervisor <span class="citation">@AliciaOshlack</span>
</td>
<td style="text-align:right;">
0.9849523
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-9" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 9</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Magdoll
</td>
<td style="text-align:left;">
(not related to #gi2019 but another TV idea - patient is admitted to Nationwide Childrens Hospital w rare disease and - panning to dramatic gradient boosted decision tree - reveals the patient has some ancestry from exotic land coinciding w mother’s younger traveling years)
</td>
<td style="text-align:right;">
0.9886662
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<span class="citation">@lizworthey</span> <span class="citation">@UABSOM</span> <span class="citation">@UABPathology</span> <span class="citation">@uabpeds</span> #gi2019 <span class="citation">@lizworthey</span> showing examples of software developed by her group for use by non-informatics experts and cases evaluated by her team’s secondary analysis pipeline including a ‘B-allele’ tool to look for chimerism/mosaicism missed by traditional pipelines
</td>
<td style="text-align:right;">
0.9870707
</td>
</tr>
<tr>
<td style="text-align:left;">
kasukawa
</td>
<td style="text-align:left;">
<p>Plant genomes. Highly variable in the maize genome.</p>
<p>Gramene is a curated, open-source, integrated data resource for comparative functional genomics in crops and model plant species. <a href="https://t.co/8TFAvyA56a" class="uri">https://t.co/8TFAvyA56a</a></p>
#gi2019
</td>
<td style="text-align:right;">
0.9864341
</td>
</tr>
<tr>
<td style="text-align:left;">
GenomeBiology
</td>
<td style="text-align:left;">
KR: looking at chronic lymphocytic leukemia. More prevalent in men than women. Patients can survive for years without treatment. 8-12% of patients have TP53 mutation, which corresponds with lower survival. Over 50% of patients with refractory CLL have no known marker #gi2019
</td>
<td style="text-align:right;">
0.9857315
</td>
</tr>
<tr>
<td style="text-align:left;">
Ailith_Ewing
</td>
<td style="text-align:left;">
Anyone interested in chromothripsis in high grade serous ovarian cancer? Check out Stuart Brown’s poster this afternoon! And chat to Stuart, a PhD student with <span class="citation">@DrColinSemple</span>, Charlie Gourley and myself, to hear all about it! #gi2019
</td>
<td style="text-align:right;">
0.9849523
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<span class="citation">@lizworthey</span> <span class="citation">@UABSOM</span> <span class="citation">@UABPathology</span> <span class="citation">@uabpeds</span> #gi2019 sequencing approaches have been great, also seeing advances in software engineering/genome informatics/comp bio allowing us to identify other important variation (e.g., mobile insertions, etc.) <span class="citation">@lizworthey</span>
</td>
<td style="text-align:right;">
0.9849523
</td>
</tr>
<tr>
<td style="text-align:left;">
byuhobbes
</td>
<td style="text-align:left;">
MM: SURFDAWave also trained a model to discriminate between neutral, sweep, and adaptive introgression; classification performance increased by supplementing 1D stats with 2D stats #gi2019
</td>
<td style="text-align:right;">
0.9840830
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
AR: In house dataset contained pedigree data allowing access to confirmed parental ancestries so were able to determine if methods accurately predicted the composition in such individuals #gi2019
</td>
<td style="text-align:right;">
0.9840830
</td>
</tr>
<tr>
<td style="text-align:left;">
lizworthey
</td>
<td style="text-align:left;">
Jiao Sun discussing computational methods to explore the role of post-transcriptional regulation in cancer; including integrative model for alternative polyadenylation #gi2019
</td>
<td style="text-align:right;">
0.9831071
</td>
</tr>
<tr>
<td style="text-align:left;">
bnlasse
</td>
<td style="text-align:left;">
<span class="citation">@lizworthey</span> <span class="citation">@UABSOM</span> <span class="citation">@UABPathology</span> <span class="citation">@uabpeds</span> #gi2019 <span class="citation">@lizworthey</span> see another tool here: <a href="https://t.co/T7tCfipNOu" class="uri">https://t.co/T7tCfipNOu</a>; also discussing strategies they are developing for examining modifier variants that might explain differences in disease presentation and outcomes
</td>
<td style="text-align:right;">
0.9831071
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="links" class="section level1" number="8">
<h1 number="8"><span class="header-section-number">8</span> Links</h1>
<p>Links to GitHub, GitLab, BitBucket, Bioconductor or CRAN mentioned in Tweets.</p>
<table>
<colgroup>
<col width="20%" />
<col width="4%" />
<col width="5%" />
<col width="4%" />
<col width="64%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Name</th>
<th align="right">Tweets</th>
<th align="right">Retweets</th>
<th align="left">Type</th>
<th align="left">Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">lancet</td>
<td align="right">2</td>
<td align="right">13</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/nygenome/lancet">nygenome/lancet</a></td>
</tr>
<tr class="even">
<td align="left">gridss-purple-linx</td>
<td align="right">3</td>
<td align="right">9</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/hartwigmedical/gridss-purple-linx">hartwigmedical/gridss-purple-linx</a></td>
</tr>
<tr class="odd">
<td align="left">gridss</td>
<td align="right">2</td>
<td align="right">7</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/papenfusslab/gridss">papenfusslab/gridss</a></td>
</tr>
<tr class="even">
<td align="left">STRling</td>
<td align="right">7</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/quinlan-lab/strling">quinlan-lab/strling</a></td>
</tr>
<tr class="odd">
<td align="left">dashing</td>
<td align="right">6</td>
<td align="right">2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dnbaker/dashing">dnbaker/dashing</a></td>
</tr>
<tr class="even">
<td align="left">UNCALLED</td>
<td align="right">6</td>
<td align="right">2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/skovaka/uncalled">skovaka/uncalled</a></td>
</tr>
<tr class="odd">
<td align="left">princess</td>
<td align="right">5</td>
<td align="right">2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/mehelmy/princess">mehelmy/princess</a></td>
</tr>
<tr class="even">
<td align="left">stringtie</td>
<td align="right">1</td>
<td align="right">6</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/gpertea/stringtie">gpertea/stringtie</a></td>
</tr>
<tr class="odd">
<td align="left">Palantir</td>
<td align="right">2</td>
<td align="right">4</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dpeerlab/palantir">dpeerlab/palantir</a></td>
</tr>
<tr class="even">
<td align="left">RCK</td>
<td align="right">2</td>
<td align="right">4</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/aganezov/rck">aganezov/rck</a></td>
</tr>
<tr class="odd">
<td align="left">mmseqs2</td>
<td align="right">1</td>
<td align="right">5</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/soedinglab/mmseqs2">soedinglab/mmseqs2</a></td>
</tr>
<tr class="even">
<td align="left">centroFlye</td>
<td align="right">5</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/seryrzu/centroflye">seryrzu/centroflye</a></td>
</tr>
<tr class="odd">
<td align="left">reticulatus</td>
<td align="right">2</td>
<td align="right">3</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/samstudio8/reticulatus">samstudio8/reticulatus</a></td>
</tr>
<tr class="even">
<td align="left">MetaMaps</td>
<td align="right">3</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/diltheylab/metamaps">diltheylab/metamaps</a></td>
</tr>
<tr class="odd">
<td align="left">sketch</td>
<td align="right">3</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dnbaker/sketch">dnbaker/sketch</a></td>
</tr>
<tr class="even">
<td align="left">sveval</td>
<td align="right">3</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/jmonlong/sveval">jmonlong/sveval</a></td>
</tr>
<tr class="odd">
<td align="left">WHdenovo</td>
<td align="right">3</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/shilpagarg/whdenovo">shilpagarg/whdenovo</a></td>
</tr>
<tr class="even">
<td align="left">assembly</td>
<td align="right">2</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/hsnguyen/assembly">hsnguyen/assembly</a></td>
</tr>
<tr class="odd">
<td align="left">minigraph</td>
<td align="right">2</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lh3/minigraph">lh3/minigraph</a></td>
</tr>
<tr class="even">
<td align="left">PhiMRF</td>
<td align="right">2</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/ashleyzhou972/phimrf">ashleyzhou972/phimrf</a></td>
</tr>
<tr class="odd">
<td align="left">pyranges</td>
<td align="right">1</td>
<td align="right">2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/biocore-ntnu/pyranges">biocore-ntnu/pyranges</a></td>
</tr>
<tr class="even">
<td align="left">STRling</td>
<td align="right">1</td>
<td align="right">2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/quinlan-lab/strling">quinlan-lab/strling</a></td>
</tr>
<tr class="odd">
<td align="left">contig-covering</td>
<td align="right">2</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/izban/contig-covering">izban/contig-covering</a></td>
</tr>
<tr class="even">
<td align="left">vg</td>
<td align="right">2</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/vgteam/vg">vgteam/vg</a></td>
</tr>
<tr class="odd">
<td align="left">All-FIT</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/khiabanianlab/all-fit">khiabanianlab/all-fit</a></td>
</tr>
<tr class="even">
<td align="left">bustools</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/bustools/bustools">bustools/bustools</a></td>
</tr>
<tr class="odd">
<td align="left">Flye</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/fenderglass/flye">fenderglass/flye</a></td>
</tr>
<tr class="even">
<td align="left">kallisto</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/pachterlab/kallisto">pachterlab/kallisto</a></td>
</tr>
<tr class="odd">
<td align="left">kb_python</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/pachterlab/kb_python">pachterlab/kb_python</a></td>
</tr>
<tr class="even">
<td align="left">RCK</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/aganezov/rck">aganezov/rck</a></td>
</tr>
<tr class="odd">
<td align="left">readfq</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lh3/readfq">lh3/readfq</a></td>
</tr>
<tr class="even">
<td align="left">scVI</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/yoseflab/scvi">yoseflab/scvi</a></td>
</tr>
<tr class="odd">
<td align="left">conterminator</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/martin-steinegger/conterminator">martin-steinegger/conterminator</a></td>
</tr>
<tr class="even">
<td align="left">dashing</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dnbaker/dashing">dnbaker/dashing</a></td>
</tr>
<tr class="odd">
<td align="left">Fl</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/fenderglass/fl">fenderglass/fl</a></td>
</tr>
<tr class="even">
<td align="left">Flye</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/fenderglass/flye">fenderglass/flye</a></td>
</tr>
<tr class="odd">
<td align="left">gfatools</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lh3/gfatools/blob/master/doc/rgfa.md">lh3/gfatools/blob/master/doc/rgfa.md</a></td>
</tr>
<tr class="even">
<td align="left">GTEx_Brains</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sexchmlab/gtex_brains">sexchmlab/gtex_brains</a></td>
</tr>
<tr class="odd">
<td align="left">LayeredGraph</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/hudsonalpha/layeredgraph">hudsonalpha/layeredgraph</a></td>
</tr>
<tr class="even">
<td align="left">Met</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/diltheylab/met">diltheylab/met</a></td>
</tr>
<tr class="odd">
<td align="left">minigraph</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lh3/minigraph">lh3/minigraph</a></td>
</tr>
<tr class="even">
<td align="left">nanopore-methylation-utilities</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/isaclee/nanopore-methylation-utilities">isaclee/nanopore-methylation-utilities</a></td>
</tr>
<tr class="odd">
<td align="left">npBarcode</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/hsnguyen/npbarcode">hsnguyen/npbarcode</a></td>
</tr>
<tr class="even">
<td align="left">npScarf</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/mdcao/npscarf">mdcao/npscarf</a></td>
</tr>
<tr class="odd">
<td align="left">parag</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/parag">illumina/parag</a></td>
</tr>
<tr class="even">
<td align="left">phimrf</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/ashleyzhou972/phimrf">ashleyzhou972/phimrf</a></td>
</tr>
<tr class="odd">
<td align="left">plass</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/soedinglab/plass">soedinglab/plass</a></td>
</tr>
<tr class="even">
<td align="left">prince</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/mehelmy/prince">mehelmy/prince</a></td>
</tr>
<tr class="odd">
<td align="left">RefineInsertions</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/mkirsche/refineinsertions">mkirsche/refineinsertions</a></td>
</tr>
<tr class="even">
<td align="left">ret</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/samstudio8/ret">samstudio8/ret</a></td>
</tr>
<tr class="odd">
<td align="left">SDA</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/mvollger/sda">mvollger/sda</a></td>
</tr>
<tr class="even">
<td align="left">ShinyOmics</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dsurujon/shinyomics">dsurujon/shinyomics</a></td>
</tr>
<tr class="odd">
<td align="left">sketch</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dnbaker/sketch">dnbaker/sketch</a></td>
</tr>
<tr class="even">
<td align="left">sqanti2</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/magdoll/sqanti2">magdoll/sqanti2</a></td>
</tr>
<tr class="odd">
<td align="left">sv-genotyping-paper</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/vgteam/sv-genotyping-paper">vgteam/sv-genotyping-paper</a></td>
</tr>
<tr class="even">
<td align="left">sveval</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/jmonlong/sveval">jmonlong/sveval</a></td>
</tr>
<tr class="odd">
<td align="left">TALON</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dewyman/talon">dewyman/talon</a></td>
</tr>
<tr class="even">
<td align="left">TALON</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dewyman/talon">dewyman/talon</a></td>
</tr>
<tr class="odd">
<td align="left">TOBIAS</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/loosolab/tobias/">loosolab/tobias/</a></td>
</tr>
<tr class="even">
<td align="left">TranscriptClean</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dewyman/transcriptclean">dewyman/transcriptclean</a></td>
</tr>
<tr class="odd">
<td align="left">TranscriptClean</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/dewyman/transcriptclean">dewyman/transcriptclean</a></td>
</tr>
<tr class="even">
<td align="left">UNCALL</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/skovaka/uncall">skovaka/uncall</a></td>
</tr>
<tr class="odd">
<td align="left">VarSight</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/hudsonalpha/varsight">hudsonalpha/varsight</a></td>
</tr>
<tr class="even">
<td align="left">vg</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/vgteam/vg">vgteam/vg</a></td>
</tr>
<tr class="odd">
<td align="left">WHd</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/shilpagarg/whd">shilpagarg/whd</a></td>
</tr>
<tr class="even">
<td align="left">wnv-phylogeography</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/andersen-lab/wnv-phylogeography">andersen-lab/wnv-phylogeography</a></td>
</tr>
<tr class="odd">
<td align="left">XPRESSpipe</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/xpressyourself/xpresspipe">xpressyourself/xpresspipe</a></td>
</tr>
</tbody>
</table>
</div>
<div id="session-info" class="section level1 unnumbered" number="">
<h1 class="unnumbered" number="">Session info</h1>
<p>
<button class="btn btn-primary" data-toggle="collapse" data-target="#SessionInfo">
Session info
</button>
</p>
<div id="SessionInfo" class="collapse">
<pre><code>## ─ Session info ───────────────────────────────────────────────────────────────
##  setting  value                       
##  version  R version 4.0.0 (2020-04-24)
##  os       macOS Catalina 10.15.6      
##  system   x86_64, darwin17.0          
##  ui       X11                         
##  language (EN)                        
##  collate  en_US.UTF-8                 
##  ctype    en_US.UTF-8                 
##  tz       Europe/Berlin               
##  date     2020-09-01                  
## 
## ─ Packages ───────────────────────────────────────────────────────────────────
##  package      * version    date       lib source                          
##  askpass        1.1        2019-01-13 [1] CRAN (R 4.0.0)                  
##  assertthat     0.2.1      2019-03-21 [1] CRAN (R 4.0.0)                  
##  backports      1.1.8      2020-06-17 [1] CRAN (R 4.0.0)                  
##  bitops         1.0-6      2013-08-17 [1] CRAN (R 4.0.0)                  
##  callr          3.4.3      2020-03-28 [1] CRAN (R 4.0.0)                  
##  clamour      * 0.1.0      2020-09-01 [1] Github (lazappi/clamour@c8ea1c7)
##  cli            2.0.2      2020-02-28 [1] CRAN (R 4.0.0)                  
##  colorspace     1.4-1      2019-03-18 [1] CRAN (R 4.0.0)                  
##  crayon         1.3.4      2017-09-16 [1] CRAN (R 4.0.0)                  
##  curl           4.3        2019-12-02 [1] CRAN (R 4.0.0)                  
##  digest         0.6.25     2020-02-23 [1] CRAN (R 4.0.0)                  
##  dplyr        * 1.0.1      2020-07-31 [1] CRAN (R 4.0.2)                  
##  ellipsis       0.3.1      2020-05-15 [1] CRAN (R 4.0.0)                  
##  emo          * 0.0.0.9000 2020-08-17 [1] Github (hadley/emo@3f03b11)     
##  evaluate       0.14       2019-05-28 [1] CRAN (R 4.0.0)                  
##  fansi          0.4.1      2020-01-08 [1] CRAN (R 4.0.0)                  
##  farver         2.0.3      2020-01-16 [1] CRAN (R 4.0.0)                  
##  forcats      * 0.5.0      2020-03-01 [1] CRAN (R 4.0.0)                  
##  fs           * 1.5.0      2020-07-31 [1] CRAN (R 4.0.2)                  
##  generics       0.0.2      2018-11-29 [1] CRAN (R 4.0.0)                  
##  ggforce        0.3.2      2020-06-23 [1] CRAN (R 4.0.2)                  
##  ggplot2      * 3.3.2      2020-06-19 [1] CRAN (R 4.0.2)                  
##  ggraph       * 2.0.3      2020-05-20 [1] CRAN (R 4.0.0)                  
##  ggrepel      * 0.8.2      2020-03-08 [1] CRAN (R 4.0.0)                  
##  ggtext       * 0.1.0      2020-06-04 [1] CRAN (R 4.0.2)                  
##  glue           1.4.1      2020-05-13 [1] CRAN (R 4.0.0)                  
##  graphlayouts   0.7.0      2020-04-25 [1] CRAN (R 4.0.0)                  
##  gridExtra      2.3        2017-09-09 [1] CRAN (R 4.0.0)                  
##  gridtext       0.1.1      2020-02-24 [1] CRAN (R 4.0.2)                  
##  gtable         0.3.0      2019-03-25 [1] CRAN (R 4.0.0)                  
##  here         * 0.1        2017-05-28 [1] CRAN (R 4.0.0)                  
##  highr          0.8        2019-03-20 [1] CRAN (R 4.0.0)                  
##  htmltools      0.5.0      2020-06-16 [1] CRAN (R 4.0.0)                  
##  httr           1.4.2      2020-07-20 [1] CRAN (R 4.0.2)                  
##  igraph       * 1.2.5      2020-03-19 [1] CRAN (R 4.0.0)                  
##  janeaustenr    0.1.5      2017-06-10 [1] CRAN (R 4.0.0)                  
##  jsonlite       1.7.0      2020-06-25 [1] CRAN (R 4.0.0)                  
##  kableExtra   * 1.2.1      2020-08-27 [1] CRAN (R 4.0.2)                  
##  knitr        * 1.29       2020-06-23 [1] CRAN (R 4.0.0)                  
##  labeling       0.3        2014-08-23 [1] CRAN (R 4.0.0)                  
##  lattice        0.20-41    2020-04-02 [1] CRAN (R 4.0.0)                  
##  lifecycle      0.2.0      2020-03-06 [1] CRAN (R 4.0.0)                  
##  lubridate    * 1.7.9      2020-06-08 [1] CRAN (R 4.0.0)                  
##  magick       * 2.4.0      2020-06-23 [1] CRAN (R 4.0.0)                  
##  magrittr       1.5        2014-11-22 [1] CRAN (R 4.0.0)                  
##  markdown       1.1        2019-08-07 [1] CRAN (R 4.0.0)                  
##  MASS           7.3-51.6   2020-04-26 [1] CRAN (R 4.0.0)                  
##  Matrix         1.2-18     2019-11-27 [1] CRAN (R 4.0.0)                  
##  modeltools     0.2-23     2020-03-05 [1] CRAN (R 4.0.0)                  
##  munsell        0.5.0      2018-06-12 [1] CRAN (R 4.0.0)                  
##  NLP            0.2-0      2018-10-18 [1] CRAN (R 4.0.0)                  
##  openssl        1.4.2      2020-06-27 [1] CRAN (R 4.0.0)                  
##  pillar         1.4.6      2020-07-10 [1] CRAN (R 4.0.2)                  
##  pkgconfig      2.0.3      2019-09-22 [1] CRAN (R 4.0.0)                  
##  plyr           1.8.6      2020-03-03 [1] CRAN (R 4.0.0)                  
##  png            0.1-7      2013-12-03 [1] CRAN (R 4.0.0)                  
##  polyclip       1.10-0     2019-03-14 [1] CRAN (R 4.0.0)                  
##  processx       3.4.3      2020-07-05 [1] CRAN (R 4.0.2)                  
##  ps             1.3.3      2020-05-08 [1] CRAN (R 4.0.0)                  
##  purrr        * 0.3.4      2020-04-17 [1] CRAN (R 4.0.0)                  
##  R6             2.4.1      2019-11-12 [1] CRAN (R 4.0.0)                  
##  RColorBrewer * 1.1-2      2014-12-07 [1] CRAN (R 4.0.0)                  
##  Rcpp           1.0.5      2020-07-06 [1] CRAN (R 4.0.0)                  
##  RCurl          1.98-1.2   2020-04-18 [1] CRAN (R 4.0.0)                  
##  reshape2       1.4.4      2020-04-09 [1] CRAN (R 4.0.0)                  
##  rlang          0.4.7      2020-07-09 [1] CRAN (R 4.0.2)                  
##  rmarkdown      2.3        2020-06-18 [1] CRAN (R 4.0.0)                  
##  rprojroot      1.3-2      2018-01-03 [1] CRAN (R 4.0.0)                  
##  rstudioapi     0.11       2020-02-07 [1] CRAN (R 4.0.0)                  
##  rtweet       * 0.7.0      2020-01-08 [1] CRAN (R 4.0.0)                  
##  rvest        * 0.3.6      2020-07-25 [1] CRAN (R 4.0.2)                  
##  scales         1.1.1      2020-05-11 [1] CRAN (R 4.0.0)                  
##  selectr        0.4-2      2019-11-20 [1] CRAN (R 4.0.0)                  
##  sessioninfo    1.1.1      2018-11-05 [1] CRAN (R 4.0.0)                  
##  slam           0.1-47     2019-12-21 [1] CRAN (R 4.0.0)                  
##  SnowballC      0.7.0      2020-04-01 [1] CRAN (R 4.0.0)                  
##  stringi        1.4.6      2020-02-17 [1] CRAN (R 4.0.0)                  
##  stringr      * 1.4.0      2019-02-10 [1] CRAN (R 4.0.0)                  
##  tibble         3.0.3      2020-07-10 [1] CRAN (R 4.0.2)                  
##  tidygraph      1.2.0      2020-05-12 [1] CRAN (R 4.0.0)                  
##  tidyr        * 1.1.1      2020-07-31 [1] CRAN (R 4.0.2)                  
##  tidyselect     1.1.0      2020-05-11 [1] CRAN (R 4.0.0)                  
##  tidytext     * 0.2.5      2020-07-11 [1] CRAN (R 4.0.2)                  
##  tm             0.7-7      2019-12-12 [1] CRAN (R 4.0.0)                  
##  tokenizers     0.2.1      2018-03-29 [1] CRAN (R 4.0.0)                  
##  topicmodels  * 0.2-11     2020-04-19 [1] CRAN (R 4.0.0)                  
##  tweenr         1.0.1      2018-12-14 [1] CRAN (R 4.0.0)                  
##  usethis        1.6.1      2020-04-29 [1] CRAN (R 4.0.0)                  
##  utf8           1.1.4      2018-05-24 [1] CRAN (R 4.0.0)                  
##  vctrs          0.3.2      2020-07-15 [1] CRAN (R 4.0.2)                  
##  viridis      * 0.5.1      2018-03-29 [1] CRAN (R 4.0.0)                  
##  viridisLite  * 0.3.0      2018-02-01 [1] CRAN (R 4.0.0)                  
##  webshot      * 0.5.2      2019-11-22 [1] CRAN (R 4.0.0)                  
##  withr          2.2.0      2020-04-20 [1] CRAN (R 4.0.0)                  
##  wordcloud    * 2.6        2018-08-24 [1] CRAN (R 4.0.0)                  
##  xfun           0.16       2020-07-24 [1] CRAN (R 4.0.2)                  
##  xml2         * 1.3.2      2020-04-23 [1] CRAN (R 4.0.0)                  
##  yaml           2.2.1      2020-02-01 [1] CRAN (R 4.0.0)                  
## 
## [1] /Library/Frameworks/R.framework/Versions/4.0/Resources/library</code></pre>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
