<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>OzSingleCell2019</title>

<script src="site_libs/header-attrs-2.3/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lazappi's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/lazappi/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">OzSingleCell2019</h1>
<h3 class="subtitle">Oz Single Cell conference 2019</h3>
<h4 class="date">Last built: 2020-09-01 18:23:01</h4>

</div>


<p>
<button class="btn btn-primary" data-toggle="collapse" data-target="#Parameters">
Parameters
</button>
</p>
<div id="Parameters" class="collapse">
<table>
<thead>
<tr class="header">
<th align="left">Parameter</th>
<th align="left">Value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">hashtag</td>
<td align="left">#OzSingleCell19</td>
</tr>
<tr class="even">
<td align="left">start_day</td>
<td align="left">2019-07-21</td>
</tr>
<tr class="odd">
<td align="left">end_day</td>
<td align="left">2019-07-23</td>
</tr>
<tr class="even">
<td align="left">timezone</td>
<td align="left">Australia/Melbourne</td>
</tr>
<tr class="odd">
<td align="left">theme</td>
<td align="left">theme_light</td>
</tr>
<tr class="even">
<td align="left">accent</td>
<td align="left">black</td>
</tr>
<tr class="odd">
<td align="left">accent2</td>
<td align="left">#7F7F7F</td>
</tr>
<tr class="even">
<td align="left">kcore</td>
<td align="left">3</td>
</tr>
<tr class="odd">
<td align="left">topics_k</td>
<td align="left">6</td>
</tr>
<tr class="even">
<td align="left">bigram_filter</td>
<td align="left">3</td>
</tr>
<tr class="odd">
<td align="left">fixed</td>
<td align="left">TRUE</td>
</tr>
<tr class="even">
<td align="left">seed</td>
<td align="left">1</td>
</tr>
</tbody>
</table>
</div>
<div id="introduction" class="section level1 unnumbered" number="">
<h1 class="unnumbered" number="">Introduction</h1>
<p>An analysis of tweets from the #OzSingleCell19 hashtag for the <a href="https://www.singlecells.org.au/" title="Oz Single Cell">Oz Single Cell</a> conference 2019. The first day was a workshop and the main conference was day 2 and 3.</p>
<p>A total of 2028 tweets from 450 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1" number="1">
<h1 number="1"><span class="header-section-number">1</span> Timeline</h1>
<div id="tweets-by-day" class="section level2" number="1.1">
<h2 number="1.1"><span class="header-section-number">1.1</span> Tweets by day</h2>
<p><img src="figures/OzSingleCell19/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2" number="1.2">
<h2 number="1.2"><span class="header-section-number">1.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2019-07-21 - 2019-07-23 in the Australia/Melbourne timezone.</p>
<p><img src="figures/OzSingleCell19/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1" number="2">
<h1 number="2"><span class="header-section-number">2</span> Users</h1>
<div id="top-tweeters" class="section level2 tabset tabset-pills" number="2.1">
<h2 class="tabset tabset-pills" number="2.1"><span class="header-section-number">2.1</span> Top tweeters</h2>
<div id="overall" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Overall</h3>
<p><img src="figures/OzSingleCell19/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="original" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Original</h3>
<p><img src="figures/OzSingleCell19/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweets" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Retweets</h3>
<p><img src="figures/OzSingleCell19/tweets-top-users-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweet-proportion" class="section level2" number="2.2">
<h2 number="2.2"><span class="header-section-number">2.2</span> Retweet proportion</h2>
<p><img src="figures/OzSingleCell19/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-timeline" class="section level2" number="2.3">
<h2 number="2.3"><span class="header-section-number">2.3</span> Top tweeters timeline</h2>
<p><img src="figures/OzSingleCell19/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2 tabset tabset-pills" number="2.4">
<h2 class="tabset tabset-pills" number="2.4"><span class="header-section-number">2.4</span> Top tweeters by day</h2>
<div id="overall-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Overall</h3>
<div id="day-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/OzSingleCell19/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/OzSingleCell19/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 3</h4>
<p><img src="figures/OzSingleCell19/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Original</h3>
<div id="day-1-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/OzSingleCell19/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/OzSingleCell19/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 3</h4>
<p><img src="figures/OzSingleCell19/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets-1" class="section level3 unnumbered tabset tabset-fade" number="">
<h3 class="unnumbered tabset tabset-fade" number="">Retweets</h3>
<div id="day-1-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 1</h4>
<p><img src="figures/OzSingleCell19/tweets-top-users-retweets-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 2</h4>
<p><img src="figures/OzSingleCell19/tweets-top-users-retweets-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Day 3</h4>
<p><img src="figures/OzSingleCell19/tweets-top-users-retweets-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
</div>
<div id="sources" class="section level1 tabset tabset-pills" number="3">
<h1 class="tabset tabset-pills" number="3"><span class="header-section-number">3</span> Sources</h1>
<div id="users-1" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Users</h2>
<p><img src="figures/OzSingleCell19/tweets-top-sources-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Tweets</h2>
<p><img src="figures/OzSingleCell19/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="networks" class="section level1" number="4">
<h1 number="4"><span class="header-section-number">4</span> Networks</h1>
<div id="replies" class="section level2" number="4.1">
<h2 number="4.1"><span class="header-section-number">4.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by PageRank.</p>
<p><img src="figures/OzSingleCell19/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2" number="4.2">
<h2 number="4.2"><span class="header-section-number">4.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Filtered for a k-core of 3. Node colour and size adjusted according to PageRank score.</p>
<p><img src="figures/OzSingleCell19/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="tweet-types" class="section level1" number="5">
<h1 number="5"><span class="header-section-number">5</span> Tweet types</h1>
<div id="retweets-2" class="section level2 tabset tabset-pills" number="5.1">
<h2 class="tabset tabset-pills" number="5.1"><span class="header-section-number">5.1</span> Retweets</h2>
<div id="proportion" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/OzSingleCell19/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/OzSingleCell19/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
The slides for my talk on ‘Overview of single cell bioinformatics’ are available from <a href="https://t.co/pXz6ooQRGW" class="uri">https://t.co/pXz6ooQRGW</a> #ozsinglecell19
</td>
<td style="text-align:right;">
117
</td>
</tr>
<tr>
<td style="text-align:left;">
trashystats
</td>
<td style="text-align:left;">
Did you know that your dimension reduction plots might be obscuring information? Not with #rstats package schex (<a href="https://t.co/7WaxV4x4yQ" class="uri">https://t.co/7WaxV4x4yQ</a>) #ozsinglecell19 <a href="https://t.co/5eW9W689tT" class="uri">https://t.co/5eW9W689tT</a>
</td>
<td style="text-align:right;">
68
</td>
</tr>
<tr>
<td style="text-align:left;">
scottgigante
</td>
<td style="text-align:left;">
Survived my 30-minute sprint through “Machine Learning for Biologists” at #ozsinglecell19! Slides and notebook are available at <a href="https://t.co/HsFAYDv4sQ" class="uri">https://t.co/HsFAYDv4sQ</a>
</td>
<td style="text-align:right;">
40
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Come work with me on single cell bioinformatics! We’ll soon be advertising a position at <span class="citation">@WEHI_research</span> to work on this and other exciting projects. Happy to chat if you’re at #ozsinglecell19
</td>
<td style="text-align:right;">
31
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Those interested in the nitty gritty of scRNA-seq normalisation will enjoy reading Aaron Lun’s secret blog <a href="https://t.co/DJ1Wq6ggzv" class="uri">https://t.co/DJ1Wq6ggzv</a> #ozsinglecell19
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
drjosephpowell
</td>
<td style="text-align:left;">
Interested in a postdoc or bioinformatician position and at #ozsinglecell19 ? We have both open. Come find me for a chat or send me a DM. <a href="https://t.co/8ptfdAkpq1" class="uri">https://t.co/8ptfdAkpq1</a> <a href="https://t.co/1v2gsnPDef" class="uri">https://t.co/1v2gsnPDef</a>
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
ozsinglecells
</td>
<td style="text-align:left;">
#ozsinglecell19 is going to be huge! We have 28 talks and 60 posters. You can now upload your posters and talks to our new <span class="citation">@F1000Research</span> channel! <a href="https://t.co/aZQ5KEP2yY" class="uri">https://t.co/aZQ5KEP2yY</a>
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
First #ozsinglecell19 keynote from <span class="citation">@satijalab</span> #sketchnotes <a href="https://t.co/YpOpqWt9kN" class="uri">https://t.co/YpOpqWt9kN</a>
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
shalinhnaik
</td>
<td style="text-align:left;">
It’s happening people #ozsinglecell19 <a href="https://t.co/s0R9OlDcuL" class="uri">https://t.co/s0R9OlDcuL</a>
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
To round out a great session on single cell bioinformatics, <span class="citation">@scottgigante</span> with the unenviable task of teaching us machine learning in 30 mins Check out his teaching materials and slides <a href="https://t.co/wzXZnHlDyu" class="uri">https://t.co/wzXZnHlDyu</a> #ozsinglecell19
</td>
<td style="text-align:right;">
14
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-retweeted" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most retweeted</h3>
<p><img src="figures/OzSingleCell19/most-retweeted-1.png" width="258" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="likes" class="section level2 tabset tabset-pills" number="5.2">
<h2 class="tabset tabset-pills" number="5.2"><span class="header-section-number">5.2</span> Likes</h2>
<div id="proportion-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/OzSingleCell19/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/OzSingleCell19/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-1" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
The slides for my talk on ‘Overview of single cell bioinformatics’ are available from <a href="https://t.co/pXz6ooQRGW" class="uri">https://t.co/pXz6ooQRGW</a> #ozsinglecell19
</td>
<td style="text-align:right;">
370
</td>
</tr>
<tr>
<td style="text-align:left;">
Dr_Heather_Lee
</td>
<td style="text-align:left;">
Very sorry to be missing this year’s meeting #ozsinglecell19 <span class="citation">@ozsinglecells</span>, but my hands are full with a major project at the moment. In the last year I turned one single cell into… <a href="https://t.co/PZkkpftiLP" class="uri">https://t.co/PZkkpftiLP</a>
</td>
<td style="text-align:right;">
201
</td>
</tr>
<tr>
<td style="text-align:left;">
trashystats
</td>
<td style="text-align:left;">
Did you know that your dimension reduction plots might be obscuring information? Not with #rstats package schex (<a href="https://t.co/7WaxV4x4yQ" class="uri">https://t.co/7WaxV4x4yQ</a>) #ozsinglecell19 <a href="https://t.co/5eW9W689tT" class="uri">https://t.co/5eW9W689tT</a>
</td>
<td style="text-align:right;">
194
</td>
</tr>
<tr>
<td style="text-align:left;">
shalinhnaik
</td>
<td style="text-align:left;">
It’s happening people #ozsinglecell19 <a href="https://t.co/s0R9OlDcuL" class="uri">https://t.co/s0R9OlDcuL</a>
</td>
<td style="text-align:right;">
124
</td>
</tr>
<tr>
<td style="text-align:left;">
scottgigante
</td>
<td style="text-align:left;">
Survived my 30-minute sprint through “Machine Learning for Biologists” at #ozsinglecell19! Slides and notebook are available at <a href="https://t.co/HsFAYDv4sQ" class="uri">https://t.co/HsFAYDv4sQ</a>
</td>
<td style="text-align:right;">
109
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Those interested in the nitty gritty of scRNA-seq normalisation will enjoy reading Aaron Lun’s secret blog <a href="https://t.co/DJ1Wq6ggzv" class="uri">https://t.co/DJ1Wq6ggzv</a> #ozsinglecell19
</td>
<td style="text-align:right;">
78
</td>
</tr>
<tr>
<td style="text-align:left;">
shalinhnaik
</td>
<td style="text-align:left;">
Thank you sponsors, speakers, OC, my lab, helpers, and my partners in crime <span class="citation">@AliciaOshlack</span> <span class="citation">@drjosephpowell</span> for a fantastic #ozsinglecell19. Most importantly thank YOU, the participants. Without your passion, intellect and openness, this would not be possible. See you next year!
</td>
<td style="text-align:right;">
51
</td>
</tr>
<tr>
<td style="text-align:left;">
10xGenomics
</td>
<td style="text-align:left;">
10x Co-founder and CSO Ben Hindson, was invited to speak at the #ozSingleCell19 Conference Dinner yesterday evening. A native of Australia, Ben’s inspirational story of how he co-founded 10x Genomics resonated powerfully with the attendees back home. <a href="https://t.co/GBMH2mhkIm" class="uri">https://t.co/GBMH2mhkIm</a>
</td>
<td style="text-align:right;">
48
</td>
</tr>
<tr>
<td style="text-align:left;">
ozsinglecells
</td>
<td style="text-align:left;">
#ozsinglecell19 is going to be huge! We have 28 talks and 60 posters. You can now upload your posters and talks to our new <span class="citation">@F1000Research</span> channel! <a href="https://t.co/aZQ5KEP2yY" class="uri">https://t.co/aZQ5KEP2yY</a>
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Come work with me on single cell bioinformatics! We’ll soon be advertising a position at <span class="citation">@WEHI_research</span> to work on this and other exciting projects. Happy to chat if you’re at #ozsinglecell19
</td>
<td style="text-align:right;">
46
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-likes" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most likes</h3>
<p><img src="figures/OzSingleCell19/most-favorited-1.png" width="258" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="quotes" class="section level2 tabset tabset-pills" number="5.3">
<h2 class="tabset tabset-pills" number="5.3"><span class="header-section-number">5.3</span> Quotes</h2>
<div id="proportion-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Proportion</h3>
<p><img src="figures/OzSingleCell19/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="count-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Count</h3>
<p><img src="figures/OzSingleCell19/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-2" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Top 10</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
ozsinglecells
</td>
<td style="text-align:left;">
It’s almost happening. You’ll know when it starts and #ozsinglecell19 will got off and you’ll have serious FOMO if you are not here <a href="https://t.co/YWhTDrdh99" class="uri">https://t.co/YWhTDrdh99</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
DawnSLin
</td>
<td style="text-align:left;">
Can’t wait! First day back to work and first conference #postPhD. Excellent program, gonna be two days of great science!! #OzSingleCell19 <a href="https://t.co/v2IlcF2IOP" class="uri">https://t.co/v2IlcF2IOP</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
bettyzhang99
</td>
<td style="text-align:left;">
Just wait until you see the #OzSingleCell19 temporary tattoos!!! They are 🔥🔥🔥 Can’t wait to attend the Sunday workshop! 🤗 <a href="https://t.co/vo3aTxOWbC" class="uri">https://t.co/vo3aTxOWbC</a>
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
ozsinglecells
</td>
<td style="text-align:left;">
Let us know when you have submitted your poster or talk to the <span class="citation">@F1000Research</span> Oz Single Cell channel so we can all admire it! #ozsinglecell19 <a href="https://t.co/aZQ5KEP2yY" class="uri">https://t.co/aZQ5KEP2yY</a> <a href="https://t.co/95XfA3wUXV" class="uri">https://t.co/95XfA3wUXV</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
This is the only way to do posters IMO. Poster sessions are for chatting and 🍷 #ozsinglecell19 <a href="https://t.co/sgJr6AUhNz" class="uri">https://t.co/sgJr6AUhNz</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
drjosephpowell
</td>
<td style="text-align:left;">
One of the things I have enjoyed the most since we started oz single cell (and ACPC) is the openness and sharing between often very different disciplines. We all benefit from this! And we should resist possible barriers being created #ozsinglecell19 <a href="https://t.co/vKIrgRyIp8" class="uri">https://t.co/vKIrgRyIp8</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
ScienceJill
</td>
<td style="text-align:left;">
Yes x 1000! Our most successful customers are fluent in both. #ozsinglecell19 <a href="https://t.co/w2jw4d7vJ0" class="uri">https://t.co/w2jw4d7vJ0</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
ozsinglecells
</td>
<td style="text-align:left;">
Heaps of computers for heaps of cells #ozsinglecell19 <a href="https://t.co/v8Ztcy7cSo" class="uri">https://t.co/v8Ztcy7cSo</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
ElinSGrayPhD
</td>
<td style="text-align:left;">
ECR <span class="citation">@weitao_lin</span> attending the #ozsinglecell19 workshop. Awesome initiative! <a href="https://t.co/oqoWcWT95g" class="uri">https://t.co/oqoWcWT95g</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
ozsinglecells
</td>
<td style="text-align:left;">
#ozsinglecell19 <a href="https://t.co/00MH5w2dyE" class="uri">https://t.co/00MH5w2dyE</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
<div id="most-quoted" class="section level3 unnumbered" number="">
<h3 class="unnumbered" number="">Most quoted</h3>
<p><img src="figures/OzSingleCell19/most-quoted-1.png" width="258" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="media" class="section level1 tabset tabset-pills" number="6">
<h1 class="tabset tabset-pills" number="6"><span class="header-section-number">6</span> Media</h1>
<div id="proportion-3" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Proportion</h2>
<p><img src="figures/OzSingleCell19/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-10-3" class="section level2 unnumbered" number="">
<h2 class="unnumbered" number="">Top 10</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Dr_Heather_Lee
</td>
<td style="text-align:left;">
Very sorry to be missing this year’s meeting #ozsinglecell19 <span class="citation">@ozsinglecells</span>, but my hands are full with a major project at the moment. In the last year I turned one single cell into… <a href="https://t.co/PZkkpftiLP" class="uri">https://t.co/PZkkpftiLP</a>
</td>
<td style="text-align:right;">
201
</td>
</tr>
<tr>
<td style="text-align:left;">
trashystats
</td>
<td style="text-align:left;">
Did you know that your dimension reduction plots might be obscuring information? Not with #rstats package schex (<a href="https://t.co/7WaxV4x4yQ" class="uri">https://t.co/7WaxV4x4yQ</a>) #ozsinglecell19 <a href="https://t.co/5eW9W689tT" class="uri">https://t.co/5eW9W689tT</a>
</td>
<td style="text-align:right;">
194
</td>
</tr>
<tr>
<td style="text-align:left;">
shalinhnaik
</td>
<td style="text-align:left;">
It’s happening people #ozsinglecell19 <a href="https://t.co/s0R9OlDcuL" class="uri">https://t.co/s0R9OlDcuL</a>
</td>
<td style="text-align:right;">
124
</td>
</tr>
<tr>
<td style="text-align:left;">
10xGenomics
</td>
<td style="text-align:left;">
10x Co-founder and CSO Ben Hindson, was invited to speak at the #ozSingleCell19 Conference Dinner yesterday evening. A native of Australia, Ben’s inspirational story of how he co-founded 10x Genomics resonated powerfully with the attendees back home. <a href="https://t.co/GBMH2mhkIm" class="uri">https://t.co/GBMH2mhkIm</a>
</td>
<td style="text-align:right;">
48
</td>
</tr>
<tr>
<td style="text-align:left;">
scottgigante
</td>
<td style="text-align:left;">
“Bioinformatics is the Australian platypus” – <span class="citation">@PeteHaitch</span> at #ozsinglecell19 <a href="https://t.co/FInrdm7Lx6" class="uri">https://t.co/FInrdm7Lx6</a>
</td>
<td style="text-align:right;">
43
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
First #ozsinglecell19 keynote from <span class="citation">@satijalab</span> #sketchnotes <a href="https://t.co/YpOpqWt9kN" class="uri">https://t.co/YpOpqWt9kN</a>
</td>
<td style="text-align:right;">
43
</td>
</tr>
<tr>
<td style="text-align:left;">
shalinhnaik
</td>
<td style="text-align:left;">
#ozsinglecell19 <a href="https://t.co/DyaYWNp7qa" class="uri">https://t.co/DyaYWNp7qa</a>
</td>
<td style="text-align:right;">
37
</td>
</tr>
<tr>
<td style="text-align:left;">
ozsinglecells
</td>
<td style="text-align:left;">
Childcare room all ready and nannies waiting for our little attendees #ozsinglecell19 <a href="https://t.co/52aBi65cqa" class="uri">https://t.co/52aBi65cqa</a>
</td>
<td style="text-align:right;">
36
</td>
</tr>
<tr>
<td style="text-align:left;">
shalinhnaik
</td>
<td style="text-align:left;">
friends who science together party together <span class="citation">@jonatcoquet</span> <span class="citation">@MartinGuilliams</span> #ozsinglecell19 <a href="https://t.co/uBco7kwA8O" class="uri">https://t.co/uBco7kwA8O</a>
</td>
<td style="text-align:right;">
35
</td>
</tr>
<tr>
<td style="text-align:left;">
MartinGuilliams
</td>
<td style="text-align:left;">
Read single cell papers on my way to #OZSingleCell19 and I often miss meaningful biology. But not for the LAM paper from <span class="citation">@IdoAmitLab</span> &amp;amp; <span class="citation">@EranElinav</span>. WHAT.A.PAPER! From human to mouse. Disease kinetic, Spatial localization, Ontogeny, role of TREM2, Function in disease. RESPECT! <a href="https://t.co/aaAb5nIc22" class="uri">https://t.co/aaAb5nIc22</a>
</td>
<td style="text-align:right;">
35
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="section" class="section level1 unnumbered toc-ignore" number="">
<h1 class="unnumbered toc-ignore" number=""></h1>
<div id="most-liked-image" class="section level2" number="6.1">
<h2 number="6.1"><span class="header-section-number">6.1</span> Most liked image</h2>
<p><img src="http://pbs.twimg.com/media/EABylo6U8AAGpTl.jpg" /></p>
</div>
</div>
<div id="tweet-text" class="section level1" number="7">
<h1 number="7"><span class="header-section-number">7</span> Tweet text</h1>
<div id="word-cloud" class="section level2" number="7.1">
<h2 number="7.1"><span class="header-section-number">7.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/OzSingleCell19/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="hashtags" class="section level2" number="7.2">
<h2 number="7.2"><span class="header-section-number">7.2</span> Hashtags</h2>
<p>Other hashtags used 5 or more times.</p>
<p><img src="figures/OzSingleCell19/hashtags-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="emojis" class="section level2" number="7.3">
<h2 number="7.3"><span class="header-section-number">7.3</span> Emojis</h2>
<p><img src="figures/OzSingleCell19/emojis-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2" number="7.4">
<h2 number="7.4"><span class="header-section-number">7.4</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 3 times.</p>
<p><img src="figures/OzSingleCell19/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2" number="7.5">
<h2 number="7.5"><span class="header-section-number">7.5</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/OzSingleCell19/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset tabset-fade" number="7.5.1">
<h3 class="tabset tabset-fade" number="7.5.1"><span class="header-section-number">7.5.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 1</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
dralexswarbrick
</td>
<td style="text-align:left;">
Incredible talk from <span class="citation">@Manu___Singh</span>, postdoc at the Garvan Institute, finding the ‘rogue cells’ driving autoimmunity pathology. Single cell DNA&amp;amp;RNA Seq, BCR repertoire analysis and functional in vitro and in vivo assays. 🤯!!! #ozsinglecell19
</td>
<td style="text-align:right;">
0.9930506
</td>
</tr>
<tr>
<td style="text-align:left;">
Tanya_C8
</td>
<td style="text-align:left;">
This was well received at our demo today 😚 <span class="citation">@Partek_Inc</span> table at #ozsinglecell19 , colouring cells with multiple genes on the same plot and selecting them with lasso tool, cell classification process in sc analysis made easier with Partek Flow point and click interface <a href="https://t.co/7Rg4YQpRrO" class="uri">https://t.co/7Rg4YQpRrO</a>
</td>
<td style="text-align:right;">
0.9927375
</td>
</tr>
<tr>
<td style="text-align:left;">
mincle
</td>
<td style="text-align:left;">
The method follows cell division and numbers of divisions with cell transcriptional phenotype - concludes that population heterogeneity is driven by differences in founder cells. For In vitro blood, clonal families populate different mature cell types. #ozsinglecell19
</td>
<td style="text-align:right;">
0.9927375
</td>
</tr>
<tr>
<td style="text-align:left;">
DaneVass
</td>
<td style="text-align:left;">
<span class="citation">@PeteHaitch</span> Great talk today at #ozsinglecell19 ! Do you typically analyse data locally for your single cell projects? With multiple tools now able to speed up analyses post-count table computation, if/when should one move to a server? Is it simply when cell numbers get huge?
</td>
<td style="text-align:right;">
0.9923948
</td>
</tr>
<tr>
<td style="text-align:left;">
Tanya_C8
</td>
<td style="text-align:left;">
Filter out low-quality cells (doublets, cells damaged during cell isolation, or cells with too few reads) is important in single cell data analysis. Take a look on how to use these interactive filters in Partek Flow <span class="citation">@Partek_Inc</span> table at #ozsinglecell19 <a href="https://t.co/sdaWyZmnjR" class="uri">https://t.co/sdaWyZmnjR</a>
</td>
<td style="text-align:right;">
0.9923948
</td>
</tr>
<tr>
<td style="text-align:left;">
TomAsh_1
</td>
<td style="text-align:left;">
<p>Cool idea for #ozsinglecell19</p>
Our (<span class="citation">@SydneyCytometry</span>) workflow for clustering and visualising high-dim cytometry data (<a href="https://t.co/XG4jKLS30B" class="uri">https://t.co/XG4jKLS30B</a>, workflow script, not a package) and our time-series clustering algorithm by Givanna Putri (<a href="https://t.co/DLsSgnVlOx" class="uri">https://t.co/DLsSgnVlOx</a>). <a href="https://t.co/rRykOEZOWx" class="uri">https://t.co/rRykOEZOWx</a>
</td>
<td style="text-align:right;">
0.9923948
</td>
</tr>
<tr>
<td style="text-align:left;">
TomAsh_1
</td>
<td style="text-align:left;">
#ozsinglecell19 do you like pseudo-time analysis? Why not actual-time analysis? Givanna Putri from our group created a time-series clustering tool called ‘ChronoClust’ – GitHub: <a href="https://t.co/sQu9X9sOGp" class="uri">https://t.co/sQu9X9sOGp</a>, link to the paper at <a href="https://t.co/vEYUe8hKcF" class="uri">https://t.co/vEYUe8hKcF</a>.
</td>
<td style="text-align:right;">
0.9920182
</td>
</tr>
<tr>
<td style="text-align:left;">
Tanya_C8
</td>
<td style="text-align:left;">
In time for #ozsinglecell19 too is analysis of Gene Expression with Feature Barcoding (antibody) data , UMAP colored by gene and protein expression data in Partek Flow, <span class="citation">@spatel_partek</span> and me will be hanging out <span class="citation">@Partek_Inc</span> table for demos or just ☕️🍰💻 <a href="https://t.co/qAtV0DoBoz" class="uri">https://t.co/qAtV0DoBoz</a>
</td>
<td style="text-align:right;">
0.9920182
</td>
</tr>
<tr>
<td style="text-align:left;">
mincle
</td>
<td style="text-align:left;">
.@TheEllisPatrick on multiscale measurements of epithelial-virus infection #HIV and the challenges moving from qualitative image analysis to quantitative data analysis #ozsinglecell19
</td>
<td style="text-align:right;">
0.9893915
</td>
</tr>
<tr>
<td style="text-align:left;">
m_mohenska
</td>
<td style="text-align:left;">
Awesome talk by <span class="citation">@EnakshiSinniah</span> from <span class="citation">@n_palpant</span>’s lab about utilising epigenetic information to determine regulatory genes in transcriptional data TRIAGE #ozsinglecell19 <a href="https://t.co/tFFRm9RqAm" class="uri">https://t.co/tFFRm9RqAm</a> <a href="https://t.co/M8faCtsHbP" class="uri">https://t.co/M8faCtsHbP</a>
</td>
<td style="text-align:right;">
0.9893915
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 2</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
.⁦<span class="citation">@LinYingxin</span>⁩ from ⁦<span class="citation">@sydneybioinfo</span>⁩ explains her scClassify method, a multiscale classification method that works with cell type hierarchies. Gives low classification error rates and finer grained cell type identification on #scRNAseq data #ozsinglecell19 <a href="https://t.co/uT9xdavzrl" class="uri">https://t.co/uT9xdavzrl</a>
</td>
<td style="text-align:right;">
0.9930506
</td>
</tr>
<tr>
<td style="text-align:left;">
DrAshHaque
</td>
<td style="text-align:left;">
If you’re interested in using scATACseq and scRNAseq to study T cell differentiation, please chat to Dr. Josh Lee, Dr Jess Engel and 3rd year PhD student Megan Soon at #ozsinglecell19 <span class="citation">@ozsinglecells</span>. They’ll be happy to relate our experiences, and show you our App too! <a href="https://t.co/xQHouNMUaG" class="uri">https://t.co/xQHouNMUaG</a>
</td>
<td style="text-align:right;">
0.9923948
</td>
</tr>
<tr>
<td style="text-align:left;">
combine_au
</td>
<td style="text-align:left;">
Another student talk to finish of today’s #ozsinglecell19 computational session Xiaomei Li “Identifying cell locations from subsets of marker genes - A winning method for the DREAM 2019 Single Cell Transcriptomics Challenge” 🎉👨‍🎓👩‍🎓🎉
</td>
<td style="text-align:right;">
0.9923948
</td>
</tr>
<tr>
<td style="text-align:left;">
DrAshHaque
</td>
<td style="text-align:left;">
If you missed my talk at ICMI2019 but want to talk about modelling Chromium scRNAseq data, please chat with Dr Jess Engel and Dr Josh Lee at #ozsinglecell19 this week. They, and others from my group will be delighted to chat. <a href="https://t.co/gIZrfmyBa0" class="uri">https://t.co/gIZrfmyBa0</a>
</td>
<td style="text-align:right;">
0.9916023
</td>
</tr>
<tr>
<td style="text-align:left;">
trashystats
</td>
<td style="text-align:left;">
Reminder that if you enjoyed the computational session at #ozsinglecell19 this morning, you may want to consider coming to BioCAsia in December in Sydney. We expect there to be much discussion on computational tools for single cell in #rstats. <a href="https://t.co/YImXNpyQJr" class="uri">https://t.co/YImXNpyQJr</a>
</td>
<td style="text-align:right;">
0.9906255
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
To round out a great session on single cell bioinformatics, <span class="citation">@scottgigante</span> with the unenviable task of teaching us machine learning in 30 mins Check out his teaching materials and slides <a href="https://t.co/wzXZnHlDyu" class="uri">https://t.co/wzXZnHlDyu</a> #ozsinglecell19
</td>
<td style="text-align:right;">
0.9900466
</td>
</tr>
<tr>
<td style="text-align:left;">
shalinhnaik
</td>
<td style="text-align:left;">
groups discussing ideas for the #pitchprize at #ozsinglecell19 for a $5000 <span class="citation">@AGRF_genomics</span> prize to use on any service they offer. Single cell, sequencing (short or long read). Looking forward to what zany ideas come out <a href="https://t.co/VBKaXXNSr9" class="uri">https://t.co/VBKaXXNSr9</a>
</td>
<td style="text-align:right;">
0.9900466
</td>
</tr>
<tr>
<td style="text-align:left;">
DrAshHaque
</td>
<td style="text-align:left;">
If you’re interested in studying single-cell parasites by scRNAseq, please chat with PhD student Lianne Lansink from my group in Melbourne this week, who will be happy to chat about all things malaria! <span class="citation">@ozsinglecells</span> #ozsinglecell19 <a href="https://t.co/gi0POxqHqj" class="uri">https://t.co/gi0POxqHqj</a>
</td>
<td style="text-align:right;">
0.9900466
</td>
</tr>
<tr>
<td style="text-align:left;">
bettyzhang99
</td>
<td style="text-align:left;">
Online workflows &amp;amp; tutorials for single cell bioinformatics recommended by <span class="citation">@PeteHaitch</span>: <span class="citation">@Bioconductor</span> <a href="https://t.co/qOid7htWDb" class="uri">https://t.co/qOid7htWDb</a> #Seurat <span class="citation">@satijalab</span> <a href="https://t.co/NGLfTMgtix" class="uri">https://t.co/NGLfTMgtix</a> #scanpy <a href="https://t.co/mGz3qWBZNG" class="uri">https://t.co/mGz3qWBZNG</a> <span class="citation">@ozsinglecells</span> #ozsinglecell19 <a href="https://t.co/MvPLTajIqt" class="uri">https://t.co/MvPLTajIqt</a>
</td>
<td style="text-align:right;">
0.9893915
</td>
</tr>
<tr>
<td style="text-align:left;">
LianneLansink
</td>
<td style="text-align:left;">
Looking forward to be challenged on my bioinformatics knowledge and experimental data during the #ozsinglecell19 poster session! If you want to know more on Plasmodium scRNAseq come to find me at poster 51. There’s a great poster (52) next to mine on T cell memory as well!
</td>
<td style="text-align:right;">
0.9893915
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-3" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 3</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
gmcderm
</td>
<td style="text-align:left;">
<p>Excited about heading to #OzSingleCell19 soon! Hopefully all this time in America hasn’t ruined my accent</p>
“Howdy mam, how many ounces of cells did you process to generate these results?” Whoops, I mean “G’day mate, you must have used a bloody shit ton of cells for this study”
</td>
<td style="text-align:right;">
0.9923948
</td>
</tr>
<tr>
<td style="text-align:left;">
mincle
</td>
<td style="text-align:left;">
.@LongCai_Lab makes important point that transcriptional clusters =/= function and shared transcriptional phenotypes can still have distinct roles if in a distinct tissue context. This needs anatomical resolution to deconvolute properly. #ozsinglecell19
</td>
<td style="text-align:right;">
0.9906255
</td>
</tr>
<tr>
<td style="text-align:left;">
bettyzhang99
</td>
<td style="text-align:left;">
People working in statistics need to be aware of the practical challenges wet lab experiments face and work closely with biologists to mitigate batch effects whenever possible - paraphrase of <span class="citation">@PeteHaitch</span>’s insights into collaboration at #ozsinglecell19
</td>
<td style="text-align:right;">
0.9906255
</td>
</tr>
<tr>
<td style="text-align:left;">
LGMartelotto
</td>
<td style="text-align:left;">
Satellite <span class="citation">@10xGenomics</span> on ATAC-Seq. “When Frankenstein meets Van Helsing: snRNA/ATAC-Seq from same nuclei prep” <span class="citation">@UMCCR</span> <span class="citation">@PeterMacRes</span> <span class="citation">@WEHI_research</span> <span class="citation">@ONJCRI</span> <span class="citation">@mscienceaustnz</span> #OzSingleCell19 <a href="https://t.co/iszT7qMqRA" class="uri">https://t.co/iszT7qMqRA</a>
</td>
<td style="text-align:right;">
0.9900466
</td>
</tr>
<tr>
<td style="text-align:left;">
carolyndg
</td>
<td style="text-align:left;">
<span class="citation">@LongCai_Lab</span> rapid increase of multiplexing of seqFISH over time #ozsinglecell19 also story of job insecurity in science, even when you are doing ridiculously innovative work that is creating a new field 😠 <a href="https://t.co/BszQ8klUQe" class="uri">https://t.co/BszQ8klUQe</a>
</td>
<td style="text-align:right;">
0.9893915
</td>
</tr>
<tr>
<td style="text-align:left;">
ACScytometry
</td>
<td style="text-align:left;">
This is it! Today is the very last chance for early bird and abstract submission!! At #ozsinglecell19 right now? Enjoyed <span class="citation">@TomAsh_1</span> talk? Why not come learn more about the #singlecell world of a #cytometrist here in Melbourne 13-16th October #ACS2019 <span class="citation">@ozsinglecells</span> <a href="https://t.co/bxyh1QDi3a" class="uri">https://t.co/bxyh1QDi3a</a>
</td>
<td style="text-align:right;">
0.9893915
</td>
</tr>
<tr>
<td style="text-align:left;">
gerald_goh
</td>
<td style="text-align:left;">
Comparing regularized negative binomial vs poisson &amp;amp; regular negative binomial models for normalizing scRNA-seq data #ozsinglecell19 <span class="citation">@satijalab</span> <a href="https://t.co/O1Wz0VJHeV" class="uri">https://t.co/O1Wz0VJHeV</a>
</td>
<td style="text-align:right;">
0.9893915
</td>
</tr>
<tr>
<td style="text-align:left;">
gerald_goh
</td>
<td style="text-align:left;">
Sexual dimorphism in gene expression patterns from cardiac hypertrophy mouse models. Fascinating work by Malathi Dona <span class="citation">@BakerResearchAu</span> #ozsinglecell19 <a href="https://t.co/f0YaY6GGsO" class="uri">https://t.co/f0YaY6GGsO</a>
</td>
<td style="text-align:right;">
0.9886440
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
Next up is Siok Tey “Analysing the peripheral blood maturation of natural killer (NK) cells following allogeneic bone marrow transplantation” #OzSingleCell19
</td>
<td style="text-align:right;">
0.9886440
</td>
</tr>
<tr>
<td style="text-align:left;">
paulfharrison
</td>
<td style="text-align:left;">
Melbourne Uni’s famous central carpark under the lawn, like a spatial barcode under a lawn of cells. #ChriSNE #ozsinglecell19 #stretchingAnAnalogy <a href="https://t.co/dY2DZmMBsH" class="uri">https://t.co/dY2DZmMBsH</a>
</td>
<td style="text-align:right;">
0.9877833
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 4</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
MartinGuilliams
</td>
<td style="text-align:left;">
Beautiful use of mathematics to reconstruct cellular development by Leila Perié from <span class="citation">@institut_curie</span> #ozsinglecell19. Index sorted to track surface markers. Cell tracers to track divisions. Correlation between some surface expression (e.g. Sca1) and ancestry. Beautiful! <a href="https://t.co/IJ7SJ0ggYY" class="uri">https://t.co/IJ7SJ0ggYY</a>
</td>
<td style="text-align:right;">
0.9933379
</td>
</tr>
<tr>
<td style="text-align:left;">
LinYingxin
</td>
<td style="text-align:left;">
CITE-seq data computational challenge: RNA data and protein data is like twitter and Facebook networks, integrating them leads to rare cell type identification 👩🏻‍💻 Great team work with Hani <span class="citation">@HaniKim127</span> 😊 #ozsinglecell19 <a href="https://t.co/ojAnePySKM" class="uri">https://t.co/ojAnePySKM</a>
</td>
<td style="text-align:right;">
0.9923948
</td>
</tr>
<tr>
<td style="text-align:left;">
dgallegoortega
</td>
<td style="text-align:left;">
Impressive tour de force from <span class="citation">@MartinGuilliams</span> at #ozsinglecell19 showing the best practices on #genomicscytometry, florescent FACS data, CITE-seq and scRNAseq all integrated to understand biology of #kupffercells. I am a convinced #REAPseq fan <a href="https://t.co/PqfgmBvSGS" class="uri">https://t.co/PqfgmBvSGS</a>
</td>
<td style="text-align:right;">
0.9920182
</td>
</tr>
<tr>
<td style="text-align:left;">
LGMartelotto
</td>
<td style="text-align:left;">
If I could summarise the #OzSingleCell19 Tech workshop with one single word I‘d say BLOODY FRIENDLY, DEFO TECHY, HEAPS of FUN and VERY AUSSIE. Yeah, that’s not one single word but this is my tweet 😁. What word would you use?
</td>
<td style="text-align:right;">
0.9916023
</td>
</tr>
<tr>
<td style="text-align:left;">
LGMartelotto
</td>
<td style="text-align:left;">
Hey sleepy head…wake up wake up wake up…time to head to #OzSingleCell19 workshops! Grab a brew (I mean coffee not beer, that’s for tonight) and come to the Technology Workshop! <a href="https://t.co/dsiOepPl1X" class="uri">https://t.co/dsiOepPl1X</a>
</td>
<td style="text-align:right;">
0.9911408
</td>
</tr>
<tr>
<td style="text-align:left;">
LianneLansink
</td>
<td style="text-align:left;">
Incredible resolution of seqFISH+ spatial single cell transcriptomics by <span class="citation">@LongCai_Lab</span> . Amazing and humbling to be among these scientists daring to create a whole new future of molecular biology at #ozsinglecell19 <a href="https://t.co/Y3htCZ0WDC" class="uri">https://t.co/Y3htCZ0WDC</a> <a href="https://t.co/ibhXIPYT8j" class="uri">https://t.co/ibhXIPYT8j</a>
</td>
<td style="text-align:right;">
0.9911408
</td>
</tr>
<tr>
<td style="text-align:left;">
DaneVass
</td>
<td style="text-align:left;">
Had an amazing time at #ozsinglecell19. So much excellent science, technology and informatics packed into two days. Big thanks to the convenors, organisers, sponsors, commercial partners and of course this incredible single-cell community. Can’t wait for next year! 👏👏🙏🙏
</td>
<td style="text-align:right;">
0.9911408
</td>
</tr>
<tr>
<td style="text-align:left;">
combine_au
</td>
<td style="text-align:left;">
More student talks at #ozsinglecell19 now Masters student Xiao Tan - “Machine learning approaches to integrating multiomics sequencing and spatial tissue imaging data”👨‍🎓
</td>
<td style="text-align:right;">
0.9900466
</td>
</tr>
<tr>
<td style="text-align:left;">
m_mohenska
</td>
<td style="text-align:left;">
<span class="citation">@10xGenomics</span> impresses the audience at #ozsinglecell19, showcasing the future of spatial transcriptomics currently 55um resolution. Their roadmap highlights their amazing plans for the future of #spatialTranscriptomics <a href="https://t.co/OOx0fkfULM" class="uri">https://t.co/OOx0fkfULM</a>
</td>
<td style="text-align:right;">
0.9893915
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
Xiao Tan telling us about using neural networks to combine image and RNA-seq data from spatial transcriptomics and classfy cell types #ozsinglecell19
</td>
<td style="text-align:right;">
0.9893915
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 5</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
10xGenomics
</td>
<td style="text-align:left;">
10x Co-founder and CSO Ben Hindson, was invited to speak at the #ozSingleCell19 Conference Dinner yesterday evening. A native of Australia, Ben’s inspirational story of how he co-founded 10x Genomics resonated powerfully with the attendees back home. <a href="https://t.co/GBMH2mhkIm" class="uri">https://t.co/GBMH2mhkIm</a>
</td>
<td style="text-align:right;">
0.9933379
</td>
</tr>
<tr>
<td style="text-align:left;">
LGMartelotto
</td>
<td style="text-align:left;">
Hexagon shape feature, 65 um feature size (blending only a few cells) and 100 um pitch (Center to Center) in the new #Visium final-ish design from <span class="citation">@10xGenomics</span>! 9 hs workflow! Of note, RIN must be around 7 to qualify for good ST profiling. #OzSingleCell19
</td>
<td style="text-align:right;">
0.9930506
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Plot your single cell data using hexbin plots to avoid overplotting. This is a really neat idea! Check out <span class="citation">@trashystats</span>’s schex package <a href="https://t.co/OajTv4rx5r" class="uri">https://t.co/OajTv4rx5r</a> to do so. Includes bonus vignette on making #shiny dashboards #ozsinglecell19
</td>
<td style="text-align:right;">
0.9920182
</td>
</tr>
<tr>
<td style="text-align:left;">
ivivek87
</td>
<td style="text-align:left;">
<p>This is amazing to share the posters via <span class="citation">@F1000Research</span> #ozsinglecell19 👏🏽</p>
Interesting approach for visual &amp;amp; social medium based engagements. I am hopeful more such initiatives will provide better networking opportunities. #posters #sciencetwitter #Conference #SingleCell <a href="https://t.co/6oaI7UREMF" class="uri">https://t.co/6oaI7UREMF</a>
</td>
<td style="text-align:right;">
0.9916023
</td>
</tr>
<tr>
<td style="text-align:left;">
10xGenomics
</td>
<td style="text-align:left;">
We are excited for the next frontier in our 10x product line - the Visium #Spatial Gene Expression Solution! Here’s 10x Co-founder and CSO Ben Hindson, sharing more about this at this morning’s #ozSingleCell19 conference. <a href="https://t.co/A6khveIly8" class="uri">https://t.co/A6khveIly8</a>
</td>
<td style="text-align:right;">
0.9916023
</td>
</tr>
<tr>
<td style="text-align:left;">
10xGenomics
</td>
<td style="text-align:left;">
We are here at #ozSingleCell19 in Melbourne, Australia. Come visit our booth, learn about how 10x solutions can help your single cell research, and meet our 10x Co-founder and CSO Ben Hindson. <a href="https://t.co/cqrIdgdUlL" class="uri">https://t.co/cqrIdgdUlL</a>
</td>
<td style="text-align:right;">
0.9906255
</td>
</tr>
<tr>
<td style="text-align:left;">
gerald_goh
</td>
<td style="text-align:left;">
.@KrishnaswamyLab on manifold learning: single cell data seem high dimensional due to ambient measurements; however, gene states are constrained and contain redundant information, and are therefore low dimension #ozsinglecell19 <span class="citation">@YaleCancer</span>
</td>
<td style="text-align:right;">
0.9900466
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>FOMO strong today with the start of #ozsinglecell19</p>
Also: your annual reminder that Tweetdeck lets you use AND/OR to grab all the official (and unofficial/typo) tags, gotta catch ’em all! <a href="https://t.co/jDbYlFXBoH" class="uri">https://t.co/jDbYlFXBoH</a>
</td>
<td style="text-align:right;">
0.9893915
</td>
</tr>
<tr>
<td style="text-align:left;">
money_gupta
</td>
<td style="text-align:left;">
<ol start="2" style="list-style-type: decimal">
<li>You are again traveling to that place with a different airline and ur flight gets canceled an hour before departure <span class="citation">@Qantas</span> and they put you in a flight on the same day (some good luck 🙃) Lesson: Just go wth the flow, control nothing works better sometime #ozsinglecell19 2/2
</td>
<td style="text-align:right;">
0.9893915
</td>
</tr>
<tr>
<td style="text-align:left;">
gerald_goh
</td>
<td style="text-align:left;">
Cross-platform cell type mapping combined with a hidden Markov random field model <a href="https://t.co/WXebuNSUqJ" class="uri">https://t.co/WXebuNSUqJ</a> #ozsinglecell19 <a href="https://t.co/21INKVL1vm" class="uri">https://t.co/21INKVL1vm</a> <a href="https://t.co/s5XHjzgEbX" class="uri">https://t.co/s5XHjzgEbX</a>
</td>
<td style="text-align:right;">
0.9886440
</td>
</tr>
</tbody>
</table></li>
</ol>
</div>
<div id="topic-6" class="section level4 unnumbered" number="">
<h4 class="unnumbered" number="">Topic 6</h4>
<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
MGI_BGI
</td>
<td style="text-align:left;">
MGI, a subsidiary of BGI Group, will showcase our innovative single cell RNA sequencing solution at the #OzSingleCell19 conference July 22-23 in Melbourne. Don’t miss the chance to meet us at the BGI/MGI exhibition area. #MGI #BGI #singlecell #sequencing #RNAseq <a href="https://t.co/kC8maZBjrJ" class="uri">https://t.co/kC8maZBjrJ</a>
</td>
<td style="text-align:right;">
0.9936023
</td>
</tr>
<tr>
<td style="text-align:left;">
10xGenomics
</td>
<td style="text-align:left;">
10x Staff Scientist Geoffrey McDermott presented a dataset using almost 200,000 CD8+ T cells &amp;amp; 44 specific antigens, demonstrating the immense and scalable power of the 10x #SingleCell Immune Profiling Solution. This paves a new way for exploring immunity. #ozSingleCell19 <a href="https://t.co/rhVlwZX7z4" class="uri">https://t.co/rhVlwZX7z4</a>
</td>
<td style="text-align:right;">
0.9933379
</td>
</tr>
<tr>
<td style="text-align:left;">
MartinGuilliams
</td>
<td style="text-align:left;">
Read single cell papers on my way to #OZSingleCell19 and I often miss meaningful biology. But not for the LAM paper from <span class="citation">@IdoAmitLab</span> &amp;amp; <span class="citation">@EranElinav</span>. WHAT.A.PAPER! From human to mouse. Disease kinetic, Spatial localization, Ontogeny, role of TREM2, Function in disease. RESPECT! <a href="https://t.co/aaAb5nIc22" class="uri">https://t.co/aaAb5nIc22</a>
</td>
<td style="text-align:right;">
0.9933379
</td>
</tr>
<tr>
<td style="text-align:left;">
ClarksysCorner
</td>
<td style="text-align:left;">
Tom Weber introducing cellular barcoding with Loxcodes. An array of lox sites with intervening DNA being barcodes. CRE can randomise array and generate a very large number of cellular barcodes through mix of inversions and excisions. #ozsinglecell19
</td>
<td style="text-align:right;">
0.9923948
</td>
</tr>
<tr>
<td style="text-align:left;">
LGMartelotto
</td>
<td style="text-align:left;">
What a wonderful single-cell 10x-perience at #OzSingleCell19!! Great talks, amazingly beautiful, smart and inspiring people…mixed feelings…I’m knackered but somehow I am already counting the days until #OzSingleCell20. Cheers 🍻 to an amazing rest of 2019! See you in 2020!
</td>
<td style="text-align:right;">
0.9923948
</td>
</tr>
<tr>
<td style="text-align:left;">
GammyAL
</td>
<td style="text-align:left;">
Want to perform scRNA Nanopore seq to link any gene genotype/isoforms with your usual 10X 3’ or 5’ scRNA data (retrospective captures too!)? All lead authors of RAGE-seq <a href="https://t.co/FirUsKAzxL" class="uri">https://t.co/FirUsKAzxL</a> are at #ozsinglecell19 Come see one of us! <span class="citation">@Manu___Singh</span> <span class="citation">@carsweshau</span> <span class="citation">@dralexswarbrick</span>
</td>
<td style="text-align:right;">
0.9916023
</td>
</tr>
<tr>
<td style="text-align:left;">
bettyzhang99
</td>
<td style="text-align:left;">
Great beginner-friendly &amp;amp; comprehensive presentation from <span class="citation">@TomAsh_1</span> on single-cell technologies incl. flow cytometry, mass cytometry, genomic cytometry, tissue mass cytometry and more! #ozsinglecell19 <a href="https://t.co/1bSwZHa1xE" class="uri">https://t.co/1bSwZHa1xE</a>
</td>
<td style="text-align:right;">
0.9911408
</td>
</tr>
<tr>
<td style="text-align:left;">
bettyzhang99
</td>
<td style="text-align:left;">
How do we combine all these single cell technologies? <span class="citation">@TomAsh_1</span> provides some insights on combination workflows &amp;amp; data analysis approaches. It is important to learn from each other’s lessons and share knowledge across fields. #OzSingleCell19 <a href="https://t.co/p5ufArsqQV" class="uri">https://t.co/p5ufArsqQV</a>
</td>
<td style="text-align:right;">
0.9906255
</td>
</tr>
<tr>
<td style="text-align:left;">
gerald_goh
</td>
<td style="text-align:left;">
Nice data from <span class="citation">@sunnyz_wu</span> from <span class="citation">@dralexswarbrick</span> lab showing scRNA-seq from cryopreserved tissue generates highly robust data. PhD students at #ozsinglecell19 all kicking ass 👏<span class="citation">@GarvanInstitute</span> <a href="https://t.co/43us4lvFMe" class="uri">https://t.co/43us4lvFMe</a>
</td>
<td style="text-align:right;">
0.9900466
</td>
</tr>
<tr>
<td style="text-align:left;">
gerald_goh
</td>
<td style="text-align:left;">
Imaging-FACS-RNAseq integrated pipeline for studying intratumor heterogeneity. Amazing work by Jane Visvader &amp;amp; Anne Rios <span class="citation">@WEHI_research</span> paper here: <a href="https://t.co/Oxwh6XRuZ9" class="uri">https://t.co/Oxwh6XRuZ9</a> #ozsinglecell19 <a href="https://t.co/oY3mTsRFYX" class="uri">https://t.co/oY3mTsRFYX</a>
</td>
<td style="text-align:right;">
0.9900466
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="links" class="section level1" number="8">
<h1 number="8"><span class="header-section-number">8</span> Links</h1>
<p>Links to GitHub, GitLab, BitBucket, Bioconductor or CRAN mentioned in Tweets.</p>
<table>
<colgroup>
<col width="13%" />
<col width="3%" />
<col width="4%" />
<col width="4%" />
<col width="73%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Name</th>
<th align="right">Tweets</th>
<th align="right">Retweets</th>
<th align="left">Type</th>
<th align="left">Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">scPred</td>
<td align="right">3</td>
<td align="right">7</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/imb-computational-genomics-lab/scpred">imb-computational-genomics-lab/scpred</a></td>
</tr>
<tr class="even">
<td align="left">OzSingleCells_Challenge_2019</td>
<td align="right">1</td>
<td align="right">7</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/luyitian/ozsinglecells_challenge_2019">luyitian/ozsinglecells_challenge_2019</a></td>
</tr>
<tr class="odd">
<td align="left">GPfates</td>
<td align="right">1</td>
<td align="right">6</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/teichlab/gpfates">teichlab/gpfates</a></td>
</tr>
<tr class="even">
<td align="left">giotto-viewer</td>
<td align="right">1</td>
<td align="right">4</td>
<td align="left">BitBucket</td>
<td align="left"><a href="https://bitbucket.org/qzhudfci/giotto-viewer/src/default/">qzhudfci/giotto-viewer/src/default/</a></td>
</tr>
<tr class="odd">
<td align="left">nichenetr</td>
<td align="right">1</td>
<td align="right">4</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/saeyslab/nichenetr">saeyslab/nichenetr</a></td>
</tr>
<tr class="even">
<td align="left">vdjpuzzle2</td>
<td align="right">1</td>
<td align="right">4</td>
<td align="left">BitBucket</td>
<td align="left"><a href="https://bitbucket.org/kirbyvisp/vdjpuzzle2">kirbyvisp/vdjpuzzle2</a></td>
</tr>
<tr class="odd">
<td align="left">scdney</td>
<td align="right">2</td>
<td align="right">2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sydneybiox/scdney">sydneybiox/scdney</a></td>
</tr>
<tr class="even">
<td align="left">SCTCwhatateam</td>
<td align="right">2</td>
<td align="right">2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/thanhbuu04/sctcwhatateam">thanhbuu04/sctcwhatateam</a></td>
</tr>
<tr class="odd">
<td align="left">schex</td>
<td align="right">3</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/saskiafreytag/schex">saskiafreytag/schex</a></td>
</tr>
<tr class="even">
<td align="left">constclust</td>
<td align="right">1</td>
<td align="right">2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/ivirshup/constclust">ivirshup/constclust</a></td>
</tr>
<tr class="odd">
<td align="left">scd</td>
<td align="right">1</td>
<td align="right">2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sydneybiox/scd">sydneybiox/scd</a></td>
</tr>
<tr class="even">
<td align="left">machine-learning-tutorial</td>
<td align="right">2</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/scottgigante/machine-learning-tutorial">scottgigante/machine-learning-tutorial</a></td>
</tr>
<tr class="odd">
<td align="left">MAGIC</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/krishnaswamylab/magic">krishnaswamylab/magic</a></td>
</tr>
<tr class="even">
<td align="left">weitrix</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/pfh/weitrix">pfh/weitrix</a></td>
</tr>
<tr class="odd">
<td align="left">CAPX</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sydneycytometry/capx">sydneycytometry/capx</a></td>
</tr>
<tr class="even">
<td align="left">celaref</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/monashbioinformaticsplatform/celaref">monashbioinformaticsplatform/celaref</a></td>
</tr>
<tr class="odd">
<td align="left">celaref.html</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">Bioc</td>
<td align="left"><a href="https://bioconductor.org/packages/celaref.html">bioconductor.org/packages/celaref.html</a></td>
</tr>
<tr class="even">
<td align="left">CellBench</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/shians/cellbench">shians/cellbench</a></td>
</tr>
<tr class="odd">
<td align="left">cellranger_convert</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/tomkellygenetics/cellranger_convert">tomkellygenetics/cellranger_convert</a></td>
</tr>
<tr class="even">
<td align="left">connectome</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/forrest-lab/connectome">forrest-lab/connectome</a></td>
</tr>
<tr class="odd">
<td align="left">CoreNET</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/biomedicalmachinelearning/corenet">biomedicalmachinelearning/corenet</a></td>
</tr>
<tr class="even">
<td align="left">DoubletDetection</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/jonathanshor/doubletdetection">jonathanshor/doubletdetection</a></td>
</tr>
<tr class="odd">
<td align="left">DoubletDetection</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/tomkellygenetics/doubletdetection">tomkellygenetics/doubletdetection</a></td>
</tr>
<tr class="even">
<td align="left">leiden</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/tomkellygenetics/leiden">tomkellygenetics/leiden</a></td>
</tr>
<tr class="odd">
<td align="left">ozSingleCell19</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/joseah/ozsinglecell19">joseah/ozsinglecell19</a></td>
</tr>
<tr class="even">
<td align="left">polyApipe</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/swbioinf/polyapipe">swbioinf/polyapipe</a></td>
</tr>
<tr class="odd">
<td align="left">scdney</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sydneybiox/scdney/commit/dc0317801ba814f66caf1da8b805b71c8a34e5ee">sydneybiox/scdney/commit/dc0317801ba814f66caf1da8b805b71c8a34e5ee</a></td>
</tr>
<tr class="even">
<td align="left">scSplit</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/jon-xu/scsplit">jon-xu/scsplit</a></td>
</tr>
<tr class="odd">
<td align="left">simpleSingleCell</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">Bioc</td>
<td align="left"><a href="https://bioconductor.org/packages/simpleSingleCell">bioconductor.org/packages/simpleSingleCell</a></td>
</tr>
<tr class="even">
<td align="left">splatter.html</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">Bioc</td>
<td align="left"><a href="https://bioconductor.org/packages/splatter.html">bioconductor.org/packages/splatter.html</a></td>
</tr>
<tr class="odd">
<td align="left">tenx.html</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">Bioc</td>
<td align="left"><a href="https://bioconductor.org/packages/tenx.html">bioconductor.org/packages/tenx.html</a></td>
</tr>
<tr class="even">
<td align="left">TRIAGE</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/woojunshim/triage">woojunshim/triage</a></td>
</tr>
<tr class="odd">
<td align="left">umapr</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/ropenscilabs/umapr">ropenscilabs/umapr</a></td>
</tr>
<tr class="even">
<td align="left">workflow</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/saskiafreytag/workflow">saskiafreytag/workflow</a></td>
</tr>
</tbody>
</table>
</div>
<div id="session-info" class="section level1 unnumbered" number="">
<h1 class="unnumbered" number="">Session info</h1>
<p>
<button class="btn btn-primary" data-toggle="collapse" data-target="#SessionInfo">
Session info
</button>
</p>
<div id="SessionInfo" class="collapse">
<pre><code>## ─ Session info ───────────────────────────────────────────────────────────────
##  setting  value                       
##  version  R version 4.0.0 (2020-04-24)
##  os       macOS Catalina 10.15.6      
##  system   x86_64, darwin17.0          
##  ui       X11                         
##  language (EN)                        
##  collate  en_US.UTF-8                 
##  ctype    en_US.UTF-8                 
##  tz       Europe/Berlin               
##  date     2020-09-01                  
## 
## ─ Packages ───────────────────────────────────────────────────────────────────
##  package      * version    date       lib source                          
##  askpass        1.1        2019-01-13 [1] CRAN (R 4.0.0)                  
##  assertthat     0.2.1      2019-03-21 [1] CRAN (R 4.0.0)                  
##  backports      1.1.8      2020-06-17 [1] CRAN (R 4.0.0)                  
##  bitops         1.0-6      2013-08-17 [1] CRAN (R 4.0.0)                  
##  callr          3.4.3      2020-03-28 [1] CRAN (R 4.0.0)                  
##  clamour      * 0.1.0      2020-09-01 [1] Github (lazappi/clamour@c8ea1c7)
##  cli            2.0.2      2020-02-28 [1] CRAN (R 4.0.0)                  
##  colorspace     1.4-1      2019-03-18 [1] CRAN (R 4.0.0)                  
##  crayon         1.3.4      2017-09-16 [1] CRAN (R 4.0.0)                  
##  curl           4.3        2019-12-02 [1] CRAN (R 4.0.0)                  
##  digest         0.6.25     2020-02-23 [1] CRAN (R 4.0.0)                  
##  dplyr        * 1.0.1      2020-07-31 [1] CRAN (R 4.0.2)                  
##  ellipsis       0.3.1      2020-05-15 [1] CRAN (R 4.0.0)                  
##  emo          * 0.0.0.9000 2020-08-17 [1] Github (hadley/emo@3f03b11)     
##  evaluate       0.14       2019-05-28 [1] CRAN (R 4.0.0)                  
##  fansi          0.4.1      2020-01-08 [1] CRAN (R 4.0.0)                  
##  farver         2.0.3      2020-01-16 [1] CRAN (R 4.0.0)                  
##  forcats      * 0.5.0      2020-03-01 [1] CRAN (R 4.0.0)                  
##  fs           * 1.5.0      2020-07-31 [1] CRAN (R 4.0.2)                  
##  generics       0.0.2      2018-11-29 [1] CRAN (R 4.0.0)                  
##  ggforce        0.3.2      2020-06-23 [1] CRAN (R 4.0.2)                  
##  ggplot2      * 3.3.2      2020-06-19 [1] CRAN (R 4.0.2)                  
##  ggraph       * 2.0.3      2020-05-20 [1] CRAN (R 4.0.0)                  
##  ggrepel      * 0.8.2      2020-03-08 [1] CRAN (R 4.0.0)                  
##  ggtext       * 0.1.0      2020-06-04 [1] CRAN (R 4.0.2)                  
##  glue           1.4.1      2020-05-13 [1] CRAN (R 4.0.0)                  
##  graphlayouts   0.7.0      2020-04-25 [1] CRAN (R 4.0.0)                  
##  gridExtra      2.3        2017-09-09 [1] CRAN (R 4.0.0)                  
##  gridtext       0.1.1      2020-02-24 [1] CRAN (R 4.0.2)                  
##  gtable         0.3.0      2019-03-25 [1] CRAN (R 4.0.0)                  
##  here         * 0.1        2017-05-28 [1] CRAN (R 4.0.0)                  
##  highr          0.8        2019-03-20 [1] CRAN (R 4.0.0)                  
##  htmltools      0.5.0      2020-06-16 [1] CRAN (R 4.0.0)                  
##  httr           1.4.2      2020-07-20 [1] CRAN (R 4.0.2)                  
##  igraph       * 1.2.5      2020-03-19 [1] CRAN (R 4.0.0)                  
##  janeaustenr    0.1.5      2017-06-10 [1] CRAN (R 4.0.0)                  
##  jsonlite       1.7.0      2020-06-25 [1] CRAN (R 4.0.0)                  
##  kableExtra   * 1.2.1      2020-08-27 [1] CRAN (R 4.0.2)                  
##  knitr        * 1.29       2020-06-23 [1] CRAN (R 4.0.0)                  
##  labeling       0.3        2014-08-23 [1] CRAN (R 4.0.0)                  
##  lattice        0.20-41    2020-04-02 [1] CRAN (R 4.0.0)                  
##  lifecycle      0.2.0      2020-03-06 [1] CRAN (R 4.0.0)                  
##  lubridate    * 1.7.9      2020-06-08 [1] CRAN (R 4.0.0)                  
##  magick       * 2.4.0      2020-06-23 [1] CRAN (R 4.0.0)                  
##  magrittr       1.5        2014-11-22 [1] CRAN (R 4.0.0)                  
##  markdown       1.1        2019-08-07 [1] CRAN (R 4.0.0)                  
##  MASS           7.3-51.6   2020-04-26 [1] CRAN (R 4.0.0)                  
##  Matrix         1.2-18     2019-11-27 [1] CRAN (R 4.0.0)                  
##  modeltools     0.2-23     2020-03-05 [1] CRAN (R 4.0.0)                  
##  munsell        0.5.0      2018-06-12 [1] CRAN (R 4.0.0)                  
##  NLP            0.2-0      2018-10-18 [1] CRAN (R 4.0.0)                  
##  openssl        1.4.2      2020-06-27 [1] CRAN (R 4.0.0)                  
##  pillar         1.4.6      2020-07-10 [1] CRAN (R 4.0.2)                  
##  pkgconfig      2.0.3      2019-09-22 [1] CRAN (R 4.0.0)                  
##  plyr           1.8.6      2020-03-03 [1] CRAN (R 4.0.0)                  
##  png            0.1-7      2013-12-03 [1] CRAN (R 4.0.0)                  
##  polyclip       1.10-0     2019-03-14 [1] CRAN (R 4.0.0)                  
##  processx       3.4.3      2020-07-05 [1] CRAN (R 4.0.2)                  
##  ps             1.3.3      2020-05-08 [1] CRAN (R 4.0.0)                  
##  purrr        * 0.3.4      2020-04-17 [1] CRAN (R 4.0.0)                  
##  R6             2.4.1      2019-11-12 [1] CRAN (R 4.0.0)                  
##  RColorBrewer * 1.1-2      2014-12-07 [1] CRAN (R 4.0.0)                  
##  Rcpp           1.0.5      2020-07-06 [1] CRAN (R 4.0.0)                  
##  RCurl          1.98-1.2   2020-04-18 [1] CRAN (R 4.0.0)                  
##  reshape2       1.4.4      2020-04-09 [1] CRAN (R 4.0.0)                  
##  rlang          0.4.7      2020-07-09 [1] CRAN (R 4.0.2)                  
##  rmarkdown      2.3        2020-06-18 [1] CRAN (R 4.0.0)                  
##  rprojroot      1.3-2      2018-01-03 [1] CRAN (R 4.0.0)                  
##  rstudioapi     0.11       2020-02-07 [1] CRAN (R 4.0.0)                  
##  rtweet       * 0.7.0      2020-01-08 [1] CRAN (R 4.0.0)                  
##  rvest        * 0.3.6      2020-07-25 [1] CRAN (R 4.0.2)                  
##  scales         1.1.1      2020-05-11 [1] CRAN (R 4.0.0)                  
##  selectr        0.4-2      2019-11-20 [1] CRAN (R 4.0.0)                  
##  sessioninfo    1.1.1      2018-11-05 [1] CRAN (R 4.0.0)                  
##  slam           0.1-47     2019-12-21 [1] CRAN (R 4.0.0)                  
##  SnowballC      0.7.0      2020-04-01 [1] CRAN (R 4.0.0)                  
##  stringi        1.4.6      2020-02-17 [1] CRAN (R 4.0.0)                  
##  stringr      * 1.4.0      2019-02-10 [1] CRAN (R 4.0.0)                  
##  tibble         3.0.3      2020-07-10 [1] CRAN (R 4.0.2)                  
##  tidygraph      1.2.0      2020-05-12 [1] CRAN (R 4.0.0)                  
##  tidyr        * 1.1.1      2020-07-31 [1] CRAN (R 4.0.2)                  
##  tidyselect     1.1.0      2020-05-11 [1] CRAN (R 4.0.0)                  
##  tidytext     * 0.2.5      2020-07-11 [1] CRAN (R 4.0.2)                  
##  tm             0.7-7      2019-12-12 [1] CRAN (R 4.0.0)                  
##  tokenizers     0.2.1      2018-03-29 [1] CRAN (R 4.0.0)                  
##  topicmodels  * 0.2-11     2020-04-19 [1] CRAN (R 4.0.0)                  
##  tweenr         1.0.1      2018-12-14 [1] CRAN (R 4.0.0)                  
##  usethis        1.6.1      2020-04-29 [1] CRAN (R 4.0.0)                  
##  utf8           1.1.4      2018-05-24 [1] CRAN (R 4.0.0)                  
##  vctrs          0.3.2      2020-07-15 [1] CRAN (R 4.0.2)                  
##  viridis      * 0.5.1      2018-03-29 [1] CRAN (R 4.0.0)                  
##  viridisLite  * 0.3.0      2018-02-01 [1] CRAN (R 4.0.0)                  
##  webshot      * 0.5.2      2019-11-22 [1] CRAN (R 4.0.0)                  
##  withr          2.2.0      2020-04-20 [1] CRAN (R 4.0.0)                  
##  wordcloud    * 2.6        2018-08-24 [1] CRAN (R 4.0.0)                  
##  xfun           0.16       2020-07-24 [1] CRAN (R 4.0.2)                  
##  xml2         * 1.3.2      2020-04-23 [1] CRAN (R 4.0.0)                  
##  yaml           2.2.1      2020-02-01 [1] CRAN (R 4.0.0)                  
## 
## [1] /Library/Frameworks/R.framework/Versions/4.0/Resources/library</code></pre>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
